var store = [{
        "title": "Aggregates",
        "excerpt":"Aggregates, clumps of smaller soil particles glued together by biological processes, are key components for e.g. soil hydraulic properties, nutrient recycling and pool sizes. Stable soil aggregates also contribute to resistance towards wind and water erosion, and are linked to improved water capture, infiltration and storage.   The smartphone apps Slakes and Moulder (a renamed version of Slakes) uses image analysis of dried, pea sized soil aggregates before and after soaking. Aggregates that resist dispersion after rewetting and slake (or mould, i.e. disintegrate) less, are more stable compared to those that break apart. A higher resistance indicate a healthier soil.   Both apps are available on both Apple’s App store and Google play for Android.   Short method description   From a dried soil sample three pea sized aggregates are selected and put in a shallow dish. A smartphone with the app installed is mounted above the dish and an image taken of the dish and the three aggregates. The dish is then gently filled with water, enough to just cover the aggregates. The user can adjust the app’s identification of the aggregates, and another images is captured. After 10 minutes a third image is taken and the app calculates an aggregate stability index (table 1).   Table 1. Aggregate stability recorded with each observation with the Slakes/Moulder apps.                  Property       Indicator       AI4SH extended naming       Unit                       Aggregate stability       aggregate-stability-index       slakes_aggregate-stability-index       unitless                 Aggregate stability       aggregate-stability-index       moulder_aggregate-stability-index       unitless           Data format   The AI4SH observed aggregate stability data are available in 2 differently formatted json objects, a flat version more suitable for direct accessibility and a nested version intended for users who wants to convert the data to a relational sql database. The flat version does not include any arrays whereas the nested version is structured with nested arrays.   Flat json object   Below is an example of a flat json object for aggregates stability observations. Object keys with a double underscore denote a link to other entries in the database. For example “person__email” is a key for the person responsible, with more detailed contact information in a separate json document. The hashtag comments are not allowed when operating a json file, they are added here for explaining each json object.   {   \"data_source\": \"ai4sh_se-loennstorp\", # 'project'+'_'+'country'+'-'+'pilot site'   \"sampling_log\": { # Assembly of samples collected in the same day at a particular pilot site       \"name\": \"se-loennstorp-20240815\", # 'country'+'-'+'pilot site'+'-'+'sampling date'     \"date_stamp\": \"20240815\", # Date when the sample was collected in the field     \"person__email\": \"sampling@gmail.com\" # Email of person responsible for the sampling   },   \"sample\": \"se-loennstorp-20240815_55-c_0-20\", # 'country'+'-'+'pilot site'+'-'+'sampling date'+'_'+'sample point id'+'_'+'min depth''+'-'+'max depth'   \"locus\": { # Assembly of the point location, setting and depth     \"position__name\": \"se-loennstorp_55-c\", # 'country'+'-'+'pilot site'+'-'+'sample point id' used for checking internal consistency     \"point\": \"55-c\", # sample point id     \"setting\": \"uniform\", # The spatial setting in the landscape (uniform, alley, plantline, edge)     \"latitude\": 55.6671111111111, # Latitude in WGS84     \"longitude\": 13.1156944444444, # Longitude in WGS84     \"min_depth\": 0, # Sample upper limit in cm in soil profile     \"max_depth\": 20 # Sample lower limit in cm in soil profile   },   \"observation\": { # Assembly of metadata and data related to the observation     \"sample_preparation__name\": \"dried-aggregate-select+soaked\", # Explanatory name of soil treatment before observation     \"person__email\": \"analysis@gmail.com\", # Email of person responsible for the observation     \"subsample\": \"a\", # For registering if physically different soil subsamples were analysed separately, default for a single analysis = a     \"replicate\": 0, # If the analysis was replicated on the same subsample more than once, the first test is always set to 0     \"n_repeats\": 1, # The number of repetitions done by any instrument on a single subsample and as part of a single replicate, with results reported as mean or mean and standard deviation     \"date_stamp\": \"20241110\", # Date when the analysis was done     \"logistic\": { # Assembly of logistical metadata on the handling of the sample       \"sample_preservation__name\": null, # Explanatory name of soil preservation       \"sample_transport__name\": null, # Explanatory name of soil transportation if sensitive for analysis results       \"transport_duration_h\": 0, # duration of any sensitive transportation in hours       \"sample_storage__name\": null, # Explanatory name of soil storage if sensitive for analysis results       \"storage_duration_h\": 2008, # Storage duration in hours       \"person__email\": \"logistic@gmail.com\" # Email of person responsible for the logistics     },     \"analysis\": { # Assembly of analysis results, procedures and instruments       \"slakes_aggregate-stability-index\": { # Soil property reported as AI4SH extended naming (part after underscore = core indicator)         \"value\": 0.69, # Recorded value (single value or mean of repeated observations)         \"unit__name\": \"index\", # Recorded unit         \"indicator__name\": \"slakes_aggregate-stability-index\", # AI4SH extended name of the soil indicator (part after underscore = core indicator)         \"procedure\": \"slakes\", # Procedure for the analysis (not always similar to method)         \"analysis_method__name\": \"slakes\", # Name of the method         \"instrument_brand__name\": \"smartphone\", # Instrument type or brand         \"instrument_model__name\": \"slakes+samsung-a26\", # Model or version of the instrument         \"instrument_id\": \"0\" # Any individual instrument id, the default for unknown id is \"0\"       }     }   } }   Nested json object   The nested json object for aggregate stability observations contains the same object keys and values as the flat version, but hierarchically organised with nested arrays. For explanations of the object keys please see the flat json object above.   {   \"data_source\": [     {       \"name\": \"ai4sh_se-loennstorp\",       \"site\": [         {           \"name\": \"se-loennstorp\",           \"point\": [             {               \"name\": \"55-c\",               \"latitude\": 55.6671111111111,               \"longitude\": 13.1156944444444,               \"setting\": \"uniform\",               \"sampling_log\": [                 {                   \"name\": \"se-loennstorp-20240815\",                   \"date_stamp\": \"20240815\",                   \"person__email\": \"sampling@gmail.com\",                   \"sample\": [                     {                       \"name\": \"se-loennstorp-20240815_55-c_0-20\",                       \"min_depth\": 0,                       \"max_depth\": 20,                       \"observation\": [                         {                           \"sample_preparation__name\": \"dried-aggregate-select+soaked\",                           \"person__email\": \"analysis@gmail.com\",                           \"subsample\": \"a\",                           \"replicate\": 0,                           \"n_repeats\": 1,                           \"date_stamp\": \"20241110\",                           \"logistic\": {                             \"sample_preservation__name\": \"dried\",                             \"sample_transport__name\": null,                             \"transport_duration_h\": 0,                             \"sample_storage__name\": null,                             \"storage_duration_h\": 2088,                             \"person__email\": \"logistic@gmail.com\"                           },                           \"analysis_method\": {                             \"slakes\": [                               {                                 \"slakes app\": [                                   {                                     \"value\": 0.69,                                     \"unit__name\": \"index\",                                     \"indicator__name\": \"slakes_aggregate-stability-index\",                                     \"analysis_method__name\": \"slakes\",                                     \"instrument_brand__name\": \"smartphone\",                                     \"instrument_model__name\": \"slakes+samsung-a26\",                                     \"instrument_id\": \"0\"                                   }                                 ]                               }                             ]                           }                         }                       ]                     }                   ]                 }               ]             }           ]         }       ]     }   ] }   Data access and file naming   The data is provided under the following licenses:      Data License: Creative Commons Attribution license (CC-BY)   Code License: Massachusetts Institute of Technology License (MIT License)   Recorded data from the aggregate stability analysis are available at:      Flat json format https://github.com/AI4SH/in-situ_data/tree/main/flat/aggregates   Nested json format: https://github.com/AI4SH/in-situ_data/tree/main/nested/aggregates   The file naming is defined to separate all eventual renewed sampling or analysis efforts and is built up from the following metadata:     country   pilot site   sampling date   point id   min depth   max depth   subsample   replicate   setting   sample_preservation   procedure   instrument   analysis date   For the examples above, the file name thus have the following component values:      se   loennstorp   20240815   55-c   0   20   a   0   uniform   dry-pick-soak   slakes+samsung-a26   0   20241110   And the final file name becomes:   se-loennstorp-20240815_55-c_0-20_a_0_uniform_dry-pick-soak_slakes+samsung-a26_0_20241110.json   Acknowledgments and Funding   This work is part of the AI4SoilHealth project, funded by the European Union’s Horizon Europe Research and Innovation Programme under Grant Agreement No. 101086179.   Funded by the European Union. The views expressed are those of the authors and do not necessarily reflect those of the European Union or the European Research Executive Agency.   ","categories": [],
        "tags": [],
        "url": "/in-situ_data_docs/docs/in-situ_methods/aggregates/",
        "teaser": null
      },{
        "title": "Bulk density and volumetric soil moisture content",
        "excerpt":"Soil compaction is recognised as one of the major threats to soil health. Compaction affects the infiltration, soil water holding capacity and other hydraulic properties. It also affects the macrofauna and indirectly also the microbiological community.   The bulk density of the soil upper layers is influenced by the soil mother material, soil natural processes and human soil management. To record the bulk density the classical method of sinking and withdrawing a soil cylinder into and out of underdisturbed soil is the superior method.   Short method description   A sturdy soil cylinder, usually with one end sharpened and with a precisely determined volumes is sunk (i.e. hammered) into the soil and taken out with soil extending out from both sides. A knife or similar tool is used to cut off any soil sticking out from the cylinder and the ends are sealed, e.g. with a plastic cap. The cylinder with its wet soil is then weighed, dried at 105o C and weighed again. Together with the weight of the cylinder (and caps if needed), the weight and volume of the wet and dry cylinder content are calculated. From this data the soil bulk density and volumetric soil moisture content are calculated.   Table 1. Soil properties recored with each observation using a soil cylinder and subsequent weighing and drying.                  Property       Indicator       AI4SH extended naming       Unit                       Soil moisture       soil-moisture-volumetric-content       soil-cylinder-drying@105c_soil-moisture-volumetric-content       vol*vol^-1                 Bulk density       bulk-density       soil-cylinder-drying@105c_bulk-density       g*cm^-3           Data format   The AI4SH observed bulk density and volumetric soil moisture content data are available in 2 differently formatted json objects, a flat version more suitable for direct accessibility and a nested version intended for users who wants to convert the data to a relational sql database. The flat version does not include any arrays whereas the nested version is structured with nested arrays.   Flat json object   Below is an example of a flat json object for bulk density and volumetric soil moisture content observations. Object keys with a double underscore denote a link to other entries in the database. For example “person__email” is a key for the person responsible, with more detailed contact information in a separate json document. The hashtag comments are not allowed when operating a json file, they are added here for explaining each json object.   {   \"data_source\": \"ai4sh_se-loennstorp\", # 'project'+'_'+'country'+'-'+'pilot site'   \"sampling_log\": { # Assembly of samples collected in the same day at a particular pilot site       \"name\": \"se-loennstorp-20240815\", # 'country'+'-'+'pilot site'+'-'+'sampling date'     \"date_stamp\": \"20240815\", # Date when the sample was collected in the field     \"person__email\": \"sampling@gmail.com\" # Email of person responsible for the sampling   },   \"sample\": \"se-loennstorp-20240815_55-c_0-20\", # 'country'+'-'+'pilot site'+'-'+'sampling date'+'_'+'sample point id'+'_'+'min depth''+'-'+'max depth'   \"locus\": { # Assembly of the point location, setting and depth     \"position__name\": \"se-loennstorp_55-c\", # 'country'+'-'+'pilot site'+'-'+'sample point id' used for checking internal consistency     \"point\": \"55-c\", # sample point id     \"setting\": \"uniform\", # The spatial setting in the landscape (uniform, alley, plantline, edge)     \"latitude\": 55.6671111111111, # Latitude in WGS84     \"longitude\": 13.1156944444444, # Longitude in WGS84     \"min_depth\": 0, # Sample upper limit in cm in soil profile     \"max_depth\": 20 # Sample lower limit in cm in soil profile   },   \"observation\": { # Assembly of metadata and data related to the observation     sample_preparation__name\": \"soil-undisturbed-in-situ\", # Explanatory name of soil treatment before observation     \"person__email\": \"analysis@gmail.com\", # Email of person responsible for the observation     \"subsample\": \"a\", # For registering if physically different soil subsamples were analysed separately, default for a single analysis = a     \"replicate\": 0, # If the analysis was replicated on the same subsample more than once, the first test is always set to 0     \"n_repeats\": 1, # The number of repetitions done by any instrument on a single subsample and as part of a single replicate, with results reported as mean or mean and standard deviation     \"date_stamp\": \"20240815\", # Date when the analysis was done     \"logistic\": { # Assembly of logistical metadata on the handling of the sample       \"sample_preservation__name\": null, # Explanatory name of soil preservation       \"sample_transport__name\": null, # Explanatory name of soil transportation if sensitive for analysis results       \"transport_duration_h\": 0, # duration of any sensitive transportation in hours       \"sample_storage__name\": null, # Explanatory name of soil storage if sensitive for analysis results       \"storage_duration_h\": 0, # Storage duration in hours       \"person__email\": \"logistic@gmail.com\" # Email of person responsible for the logistics     },     \"analysis\": { # Assembly of analysis results, procedures and instruments       \"soil-cylinder-drying@105c_soil-moisture-volumetric-content\": { # Soil property reported as AI4SH extended naming (part after underscore = core indicator)         \"value\": 0.162717138949006, # Recorded value (single value or mean of repeated observations)         \"unit__name\": \"vol*vol^-1\", # Recorded unit         \"indicator__name\": \"soil-cylinder-drying@105c_soil-moisture-volumetric-content\", # AI4SH extended name of the soil indicator (part after underscore = core indicator)         \"procedure\": \"soil-cylinder-drying@105c\", # Procedure for the analysis (not always similar to method)         \"analysis_method__name\": \"soil-cylinder-drying@105c\", # Name of the method         \"instrument_brand__name\": \"general-soil-cylinder\", # Instrument type or brand         \"instrument_model__name\": \"0\", # Model or version of the instrument         \"instrument_id\": \"0\" # Any individual instrument id, the default for unknown id is \"0\"           }       \"soil-cylinder-drying@105c_bulk-density\": {         \"value\": 1.28338286105099,         \"unit__name\": \"g*cm^-3\",         \"indicator__name\": \"soil-cylinder-drying@105c_bulk-density\",         \"procedure\": \"soil-cylinder-drying@105c\",         \"analysis_method__name\": \"soil-cylinder-drying@105c\",         \"instrument_brand__name\": \"general-soil-cylinder\",         \"instrument_model__name\": \"0\",         \"instrument_id\": \"0\"       }     }   } }   Nested json object   The nested json object for soil bulk density and volumetric soil moisture content observations contains the same object keys and values as the flat version, but hierarchically organised with nested arrays. For explanations of the object keys please see the flat json object above.   {   \"data_source\": [     {       \"name\": \"ai4sh_se-loennstorp\",       \"site\": [         {           \"name\": \"se-loennstorp\",           \"point\": [             {               \"name\": \"55-c\",               \"latitude\": 55.6671111111111,               \"longitude\": 13.1156944444444,               \"setting\": \"uniform\",               \"sampling_log\": [                 {                   \"name\": \"se-loennstorp-20240815\",                   \"date_stamp\": \"20240815\",                   \"person__email\": \"sampling@gmail.com\",                   \"sample\": [                     {                       \"name\": \"se-loennstorp-20240815_55-c_0-20\",                       \"min_depth\": 0,                       \"max_depth\": 20,                       \"observation\": [                         {                           \"sample_preparation__name\": \"soil-undisturbed-in-situ\",                           \"person__email\": \"analysis@gmail.com\",                           \"subsample\": \"a\",                           \"replicate\": 0,                           \"n_repeats\": 1,                           \"date_stamp\": \"20240815\",                           \"logistic\": {                             \"sample_preservation__name\": null,                             \"sample_transport__name\": null,                             \"transport_duration_h\": 0,                             \"sample_storage__name\": null,                             \"storage_duration_h\": 0,                             \"person__email\": \"logistic@gmail.com\"                           },                           \"analysis_method\": {                             \"soil-cylinder-drying@105c\": [                               {                                 \"soil-cylinder\": [                                   {                                     \"value\": 0.162717138949006,                                     \"unit__name\": \"vol*vol^-1\",                                     \"indicator__name\": \"soil-cylinder-drying@105c_soil-moisture-volumetric-content\",                                     \"analysis_method__name\": \"soil-cylinder-drying@105c\",                                     \"instrument_brand__name\": \"general-soil-cylinder\",                                     \"instrument_model__name\": \"0\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 1.28338286105099,                                     \"unit__name\": \"g*cm^-3\",                                     \"indicator__name\": \"soil-cylinder-drying@105c_bulk-density\",                                     \"procedure\": \"soil-cylinder-drying@105c\",                                     \"analysis_method__name\": \"soil-cylinder-drying@105c\",                                     \"instrument_brand__name\": \"general-soil-cylinder\",                                     \"instrument_model__name\": \"0\",                                     \"instrument_id\": \"0\"                                   }                                 ]                               }                             ]                           }                         }                       ]                     }                   ]                 }               ]             }           ]         }       ]     }   ] }   Data access and file naming   The data is provided under the following licenses:      Data License: Creative Commons Attribution license (CC-BY)   Code License: Massachusetts Institute of Technology License (MIT License)   Recorded data from the bulk density and volumetric soil moisture content analysis are available at:      Flat json format https://github.com/AI4SH/in-situ_data/tree/main/flat/bulk_density   Nested json format: https://github.com/AI4SH/in-situ_data/tree/main/nested/bulk_density   The file naming is defined to separate all eventual renewed sampling or analysis efforts and is built up from the following metadata:     country   pilot site   sampling date   point id   min depth   max depth   subsample   replicate   setting   sample_preservation   procedure   instrument   analysis date   For the examples above, the file name thus have the following component values:      se   loennstorp   20240815   55-c   0   20   a   0   uniform   no-prep   soil-cylinder-drying@105c   0   20240815   And the final file name becomes:   se-loennstorp-20240815_55-c_0-20_a_0_uniform_no-prep_soil-cylinder-drying@105c_0_20240815.json   Acknowledgments and Funding   This work is part of the AI4SoilHealth project, funded by the European Union’s Horizon Europe Research and Innovation Programme under Grant Agreement No. 101086179.   Funded by the European Union. The views expressed are those of the authors and do not necessarily reflect those of the European Union or the European Research Executive Agency.   ","categories": [],
        "tags": [],
        "url": "/in-situ_data_docs/docs/in-situ_methods/bulkdensity/",
        "teaser": null
      },{
        "title": "Enzymatic activity",
        "excerpt":"All biological processes and functions are governed by enzymes and their activities. Assessing the activity rates of soil extracellular enzymes has traditionally been an expensive analysis restricted to the laboratory and specialist personnel.   The Swiss start-up Digit Soil, a partner of the AI4SH (AI4SoilHealth) project, has developed a rapid method for quantifying five key soil enzymatic activities. The Soil Enzymatic Activity Reader (SEAR) and reaction plates developed by Digit Soil form a stand-alone system that can be used in an ordinary home or office (laboratory-independent) and is applicable for use by, for instance, citizen scientists. This move makes the complex measurement of soil functional diversity and activity simple and inexpensive.   Short method description   The Soil Enzymatic Activity Reader (SEAR), developed by Digit Soil, uses a combination of reaction plates and fluorescence spectroscopy Fetzer et al., 2025. The method requires fresh samples, which must be either analyzed directly from the field or stored and transported at a low temperature (a few degrees Celsius) for no more than a few days before analysis.Reaction plates, designed with 5 x 5 wells (which include three replicates for each enzyme plus internal calibration wells), are pressed onto the soil contained in a soil tray. The tray is then inserted into a spectrometer that uses targeted UV light to excite the reactants in the reaction plate. A digital camera registers the resulting signal, which is later computed to determine the enzymatic activity rate over a 40-minute period. The enzymatic activity rates of five key enzymes are then reported. The enzymatic activities measured are summarised in table 1.   Table 1. Enzymatic activities recorded with each observation with the Soil Enzymatic Activity Reader (SEAR).                  Property       Indicator       AI4SH extended naming       Unit                       Chitinase/β-glucosaminidase       GLA       digit-soil-sear_GLA       pmol*min^-1                 β-Glucosidase       GLS       digit-soil-sear_GLS       pmol*min^-1                 phosphatases (Phosphomonesterases)       PHO       digit-soil-sear_PHO       pmol*min^-1                 β-Xylosidase       XYL       digit-soil-sear_XYL       pmol*min^-1                 Leucine-aminopeptidase       LEU       digit-soil-sear_LEU       pmol*min^-1           Data format   Each SEAR measurement is based on 3 technical replications of each enzymatic activity and the reported data represents the mean value. The observed enzymatic activity data are available in 2 differently formatted json objects, a flat version more suitable for direct accessibility and a nested version intended for users who want to convert the data to a relational sql database. The flat version does not include any arrays whereas the nested version is structured with nested arrays.   Flat json object   Below is an example of a flat json object for enzymatic observations. Object keys with a double underscore denote a link to other entries in the database. For example “person__email” is a key for the person responsible, with more detailed contact information in a separate json document. The hashtag comments are not allowed when operating a json file, they are added here for explaining each json object.   {   \"data_source\": \"ai4sh_se-loennstorp\", # 'project'+'_'+'country'+'-'+'pilot site'   \"sampling_log\": { # Assembly of samples collected in the same day at a particular pilot site       \"name\": \"se-loennstorp-20240815\", # 'country'+'-'+'pilot site'+'-'+'sampling date'     \"date_stamp\": \"20240815\", # Date when the sample was collected in the field     \"person__email\": \"sampling@gmail.com\" # Email of person responsible for the sampling   },   \"sample\": \"se-loennstorp-20240815_55-c_0-20\", # 'country'+'-'+'pilot site'+'-'+'sampling date'+'_'+'sample point id'+'_'+'min depth''+'-'+'max depth'   \"locus\": { # Assembly of the point location, setting and depth     \"position__name\": \"se-loennstorp_55-c\", # 'country'+'-'+'pilot site'+'-'+'sample point id' used for checking internal consistency     \"point\": \"55-c\", # sample point id     \"setting\": \"uniform\", # The spatial setting in the landscape (uniform, alley, plantline, edge)     \"latitude\": 55.6671111111111, # Latitude in WGS84     \"longitude\": 13.1156944444444, # Longitude in WGS84     \"min_depth\": 0, # Sample upper limit in cm in soil profile     \"max_depth\": 20 # Sample lower limit in cm in soil profile   },   \"observation\": { # Assembly of metadata and data related to the observation     \"sample_preparation__name\": \"mixed-untreated-soil-in-lab\", # Explanatory name of soil treatment before observation     \"person__email\": \"analysis@gmail.com\", # Email of person responsible for the observation     \"subsample\": \"a\", # For registering if physically different soil subsamples were analysed separately, default for a single analysis = a     \"replicate\": 0, # If the analysis was replicated on the same subsample more than once, the first test is always set to 0     \"n_repeats\": 5, # The number of repetitions done by any instrument on a single subsample and as part of a single replicate, with results reported as mean or mean and standard deviation     \"date_stamp\": \"20240815\", # Date when the analysis was done     \"logistic\": { # Assembly of logistical metadata on the handling of the sample       \"sample_preservation__name\": null, # Explanatory name of soil preservation       \"sample_transport__name\": null, # Explanatory name of soil transportation if sensitive for analysis results       \"transport_duration_h\": 0, # duration of any sensitive transportation in hours       \"sample_storage__name\": null, # Explanatory name of soil storage if sensitive for analysis results       \"storage_duration_h\": 0, # Storage duration in hours       \"person__email\": \"logistic@gmail.com\" # Email of person responsible for the logistics     },     \"analysis\": { # Assembly of analysis results, procedures and instruments       \"digit-soil-sear_LEU\": { # Soil property reported as AI4SH extended naming (part after underscore = core indicator)         \"value\": 3.5, # Recorded value (single value or mean of repeated observations)         \"unit__name\": \"pmol*min^-1\", # Recorded unit         \"indicator__name\": \"digit-soil-sear_LEU\", # AI4SH extended name of the soil indicator (part after underscore = core indicator)         \"procedure\": \"digit-soil-sear\", # Procedure for the analysis (not always similar to method)         \"analysis_method__name\": \"digit-soil-sear\", # Name of the method         \"instrument_brand__name\": \"digit-soil\", # Instrument type or brand         \"instrument_model__name\": \"bob\", # Model or version of the instrument         \"instrument_id\": \"0\" # Any individual instrument id, the default for unknown id is \"0\"       },       \"digit-soil-sear_GLA\": {         \"value\": 8.2905,         \"unit__name\": \"pmol*min^-1\",         \"indicator__name\": \"digit-soil-sear_GLA\",         \"procedure\": \"digit-soil-sear\",         \"analysis_method__name\": \"digit-soil-sear\",         \"instrument_brand__name\": \"digit-soil\",         \"instrument_model__name\": \"bob\",         \"instrument_id\": \"0\"       },       \"digit-soil-sear_GLS\": {         \"value\": 25.273,         \"unit__name\": \"pmol*min^-1\",         \"indicator__name\": \"digit-soil-sear_GLS\",         \"procedure\": \"digit-soil-sear\",         \"analysis_method__name\": \"digit-soil-sear\",         \"instrument_brand__name\": \"digit-soil\",         \"instrument_model__name\": \"bob\",         \"instrument_id\": \"0\"       },       \"digit-soil-sear_PHO\": {         \"value\": 60.2077,         \"unit__name\": \"pmol*min^-1\",         \"indicator__name\": \"digit-soil-sear_PHO\",         \"procedure\": \"digit-soil-sear\",         \"analysis_method__name\": \"digit-soil-sear\",         \"instrument_brand__name\": \"digit-soil\",         \"instrument_model__name\": \"bob\",         \"instrument_id\": \"0\"       },       \"digit-soil-sear_XYL\": {         \"value\": 8.1644,         \"unit__name\": \"pmol*min^-1\",         \"indicator__name\": \"digit-soil-sear_XYL\",         \"procedure\": \"digit-soil-sear\",         \"analysis_method__name\": \"digit-soil-sear\",         \"instrument_brand__name\": \"digit-soil\",         \"instrument_model__name\": \"bob\",         \"instrument_id\": \"0\"       }     }   } }   Nested json object   The nested json object for soil enzymatic activity observations contains the same object keys and values as the flat version, but hierarchically organised with nested arrays. For explanations of the object keys please see the flat json object above.   {   \"data_source\": [     {       \"name\": \"ai4sh_se-loennstorp\",       \"site\": [         {           \"name\": \"se-loennstorp\",           \"point\": [             {               \"name\": \"55-c\",               \"latitude\": 55.6671111111111,               \"longitude\": 13.1156944444444,               \"setting\": \"uniform\",               \"sampling_log\": [                 {                   \"name\": \"se-loennstorp-20240815\",                   \"date_stamp\": \"20240815\",                   \"person__email\": \"sampling@gmail.com\",                   \"sample\": [                     {                       \"name\": \"se-loennstorp-20240815_55-c_0-20\",                       \"min_depth\": 0,                       \"max_depth\": 20,                       \"observation\": [                         {                           \"sample_preparation__name\": \"mixed-untreated-soil-in-lab\",                           \"person__email\": \"analysis@gmail.com\",                           \"subsample\": \"a\",                           \"replicate\": 0,                           \"n_repeats\": 1,                           \"date_stamp\": \"20240815\",                           \"logistic\": {                             \"sample_preservation__name\": null,                             \"sample_transport__name\": null,                             \"transport_duration_h\": 0,                             \"sample_storage__name\": null,                             \"storage_duration_h\": 0,                             \"person__email\": \"logistic@gmail.com\"                           },                           \"analysis_method\": {                             \"digit-soil-sear\": [                               {                                 \"digitsoil\": [                                   {                                     \"value\": 3.5,                                     \"unit__name\": \"pmol*min^-1\",                                     \"indicator__name\": \"digit-soil-sear_LEU\",                                     \"analysis_method__name\": \"digit-soil-sear\",                                     \"instrument_brand__name\": \"digit-soil\",                                     \"instrument_model__name\": \"bob\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 3.8288,                                     \"unit__name\": \"pmol*min^-1\",                                     \"indicator__name\": \"digit-soil-sear_GLA\",                                     \"procedure\": \"digit-soil-sear\",                                     \"analysis_method__name\": \"digit-soil-sear\",                                     \"instrument_brand__name\": \"digit-soil\",                                     \"instrument_model__name\": \"bob\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 6.5383,                                     \"unit__name\": \"pmol*min^-1\",                                     \"indicator__name\": \"digit-soil-sear_GLS\",                                     \"procedure\": \"digit-soil-sear\",                                     \"analysis_method__name\": \"digit-soil-sear\",                                     \"instrument_brand__name\": \"digit-soil\",                                     \"instrument_model__name\": \"bob\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 7.9978,                                     \"unit__name\": \"pmol*min^-1\",                                     \"indicator__name\": \"digit-soil-sear_PHO\",                                     \"procedure\": \"digit-soil-sear\",                                     \"analysis_method__name\": \"digit-soil-sear\",                                     \"instrument_brand__name\": \"digit-soil\",                                     \"instrument_model__name\": \"bob\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.5,                                     \"unit__name\": \"pmol*min^-1\",                                     \"indicator__name\": \"digit-soil-sear_XYL\",                                     \"procedure\": \"digit-soil-sear\",                                     \"analysis_method__name\": \"digit-soil-sear\",                                     \"instrument_brand__name\": \"digit-soil\",                                     \"instrument_model__name\": \"bob\",                                     \"instrument_id\": \"0\"                                   }                                 ]                               }                             ]                           }                         }                       ]                     }                   ]                 }               ]             }           ]         }       ]     }   ] }    Data access and file naming   The data is provided under the following licenses:      Data License: Creative Commons Attribution license (CC-BY)   Code License: Massachusetts Institute of Technology License (MIT License)   Recorded data from the enzymatic activity analysis are available at:      Flat json format https://github.com/AI4SH/in-situ_data/tree/main/flat/digit-soil_sear   Nested json format: https://github.com/AI4SH/in-situ_data/tree/main/nested/digit-soil_sear   The file naming is defined to separate all eventual renewed sampling or analysis efforts and is built up from the following metadata:     country   pilot site   sampling date   point id   min depth   max depth   subsample   replicate   setting   sample_preservation   procedure   instrument   analysis date   For the examples above, the file name thus have the following component values:      se   loennstorp   20240815   55-c   0   20   a   0   uniform   mix-wet   bob   0   20240815   And the final file name becomes:   se-loennstorp-20240815_55-c_0-20_a_0_uniform_mix-wet_bob_0_20240815.json   Acknowledgments and Funding   This work is part of the AI4SoilHealth project, funded by the European Union’s Horizon Europe Research and Innovation Programme under Grant Agreement No. 101086179.   Funded by the European Union. The views expressed are those of the authors and do not necessarily reflect those of the European Union or the European Research Executive Agency.   References   Fetzer, J., Meller, S., Iven, H., Baur, D., García Rivera, P., Meller, A. &amp; Luster, J. (2025) ‘Novel, laboratory-independent device to measure extracellular enzymatic activity in soils’, Frontiers in Environmental Science, 13, p. 1663635. doi: 10.3389/fenvs.2025.1663635   ","categories": [],
        "tags": [],
        "url": "/in-situ_data_docs/docs/in-situ_methods/digitsoil/",
        "teaser": null
      },{
        "title": "Enzymatic activity",
        "excerpt":"All biological processes and functions are governed by enzymes and their activities. Assessing the activity rates of soil extracellular enzymes has traditionally been an expensive analysis restricted to the laboratory and specialist personnel.   The Swiss start-up Digit Soil, a partner of the AI4SH (AI4SoilHealth) project, has developed a rapid method for quantifying five key soil enzymatic activities. The Soil Enzymatic Activity Reader (SEAR) and reaction plates developed by Digit Soil form a stand-alone system that can be used in an ordinary home or office (laboratory-independent) and is applicable for use by, for instance, citizen scientists. This move makes the complex measurement of soil functional diversity and activity simple and inexpensive.   Short method description   The Soil Enzymatic Activity Reader (SEAR), developed by Digit Soil, uses a combination of reaction plates and fluorescence spectroscopy. The method requires fresh samples, which must be either analyzed directly from the field or stored and transported at a low temperature (a few degrees Celsius) for no more than a few days before analysis.   Reaction plates, designed with 5 x 5 wells (which include three replicates for each enzyme plus internal calibration wells), are pressed onto the soil contained in a soil tray. The tray is then inserted into a spectrometer that uses UV light to excite the reactants in the reaction plate. A digital camera registers the resulting signal, which is later computed to determine the enzymatic activity rate over a 40-minute period. The enzymatic activity rates of five key enzymes are then reported. The enzymatic activities measured are summarised in table 1   Table 1. Enzymatic activities recorded with each observation with the Soil Enzymatic Activity Reader (SEAR).                  Property       Indicator       AI4SH extended naming       Unit                       Chitinase/β-glucosaminidase       GLA       digit-soil-sear_GLA       pmol*min^-1                 β-Glucosidase       GLS       digit-soil-sear_GLS       pmol*min^-1                 phosphatases (Phosphomonesterases)       PHO       digit-soil-sear_PHO       pmol*min^-1                 β-Xylosidase       XYL       digit-soil-sear_XYL       pmol*min^-1                 Leucine-aminopeptidase       LEU       digit-soil-sear_LEU       pmol*min^-1           Data format   Each SEAR measurement is based on 3 technical replications of each enzymatic activity and the reported data represents the mean value. The observed enzymatic activity data are available in 2 differently formatted json objects, a flat version more suitable for direct accessibility and a nested version intended for users who want to convert the data to a relational sql database. The flat version does not include any arrays whereas the nested version is structured with nested arrays.   Flat json object   Below is an example of a flat json object for enzymatic observations. Object keys with a double underscore denote a link to other entries in the database. For example “person__email” is a key for the person responsible, with more detailed contact information in a separate json document. The hashtag comments are not allowed when operating a json file, they are added here for explaining each json object.   {   \"data_source\": \"ai4sh_se-loennstorp\", # 'project'+'_'+'country'+'-'+'pilot site'   \"sampling_log\": { # Assembly of samples collected in the same day at a particular pilot site       \"name\": \"se-loennstorp-20240815\", # 'country'+'-'+'pilot site'+'-'+'sampling date'     \"date_stamp\": \"20240815\", # Date when the sample was collected in the field     \"person__email\": \"sampling@gmail.com\" # Email of person responsible for the sampling   },   \"sample\": \"se-loennstorp-20240815_55-c_0-20\", # 'country'+'-'+'pilot site'+'-'+'sampling date'+'_'+'sample point id'+'_'+'min depth''+'-'+'max depth'   \"locus\": { # Assembly of the point location, setting and depth     \"position__name\": \"se-loennstorp_55-c\", # 'country'+'-'+'pilot site'+'-'+'sample point id' used for checking internal consistency     \"point\": \"55-c\", # sample point id     \"setting\": \"uniform\", # The spatial setting in the landscape (uniform, alley, plantline, edge)     \"latitude\": 55.6671111111111, # Latitude in WGS84     \"longitude\": 13.1156944444444, # Longitude in WGS84     \"min_depth\": 0, # Sample upper limit in cm in soil profile     \"max_depth\": 20 # Sample lower limit in cm in soil profile   },   \"observation\": { # Assembly of metadata and data related to the observation     \"sample_preparation__name\": \"mixed-untreated-soil-in-lab\", # Explanatory name of soil treatment before observation     \"person__email\": \"analysis@gmail.com\", # Email of person responsible for the observation     \"subsample\": \"a\", # For registering if physically different soil subsamples were analysed separately, default for a single analysis = a     \"replicate\": 0, # If the analysis was replicated on the same subsample more than once, the first test is always set to 0     \"n_repeats\": 5, # The number of repetitions done by any instrument on a single subsample and as part of a single replicate, with results reported as mean or mean and standard deviation     \"date_stamp\": \"20240815\", # Date when the analysis was done     \"logistic\": { # Assembly of logistical metadata on the handling of the sample       \"sample_preservation__name\": null, # Explanatory name of soil preservation       \"sample_transport__name\": null, # Explanatory name of soil transportation if sensitive for analysis results       \"transport_duration_h\": 0, # duration of any sensitive transportation in hours       \"sample_storage__name\": null, # Explanatory name of soil storage if sensitive for analysis results       \"storage_duration_h\": 0, # Storage duration in hours       \"person__email\": \"logistic@gmail.com\" # Email of person responsible for the logistics     },     \"analysis\": { # Assembly of analysis results, procedures and instruments       \"digit-soil-sear_LEU\": { # Soil property reported as AI4SH extended naming (part after underscore = core indicator)         \"value\": 3.5, # Recorded value (single value or mean of repeated observations)         \"unit__name\": \"pmol*min^-1\", # Recorded unit         \"indicator__name\": \"digit-soil-sear_LEU\", # AI4SH extended name of the soil indicator (part after underscore = core indicator)         \"procedure\": \"digit-soil-sear\", # Procedure for the analysis (not always similar to method)         \"analysis_method__name\": \"digit-soil-sear\", # Name of the method         \"instrument_brand__name\": \"digit-soil\", # Instrument type or brand         \"instrument_model__name\": \"bob\", # Model or version of the instrument         \"instrument_id\": \"0\" # Any individual instrument id, the default for unknown id is \"0\"       },       \"digit-soil-sear_GLA\": {         \"value\": 8.2905,         \"unit__name\": \"pmol*min^-1\",         \"indicator__name\": \"digit-soil-sear_GLA\",         \"procedure\": \"digit-soil-sear\",         \"analysis_method__name\": \"digit-soil-sear\",         \"instrument_brand__name\": \"digit-soil\",         \"instrument_model__name\": \"bob\",         \"instrument_id\": \"0\"       },       \"digit-soil-sear_GLS\": {         \"value\": 25.273,         \"unit__name\": \"pmol*min^-1\",         \"indicator__name\": \"digit-soil-sear_GLS\",         \"procedure\": \"digit-soil-sear\",         \"analysis_method__name\": \"digit-soil-sear\",         \"instrument_brand__name\": \"digit-soil\",         \"instrument_model__name\": \"bob\",         \"instrument_id\": \"0\"       },       \"digit-soil-sear_PHO\": {         \"value\": 60.2077,         \"unit__name\": \"pmol*min^-1\",         \"indicator__name\": \"digit-soil-sear_PHO\",         \"procedure\": \"digit-soil-sear\",         \"analysis_method__name\": \"digit-soil-sear\",         \"instrument_brand__name\": \"digit-soil\",         \"instrument_model__name\": \"bob\",         \"instrument_id\": \"0\"       },       \"digit-soil-sear_XYL\": {         \"value\": 8.1644,         \"unit__name\": \"pmol*min^-1\",         \"indicator__name\": \"digit-soil-sear_XYL\",         \"procedure\": \"digit-soil-sear\",         \"analysis_method__name\": \"digit-soil-sear\",         \"instrument_brand__name\": \"digit-soil\",         \"instrument_model__name\": \"bob\",         \"instrument_id\": \"0\"       }     }   } }   Nested json object   The nested json object for soil enzymatic activity observations contains the same object keys and values as the flat version, but hierarchically organised with nested arrays. For explanations of the object keys please see the flat json object above.   {   \"data_source\": [     {       \"name\": \"ai4sh_se-loennstorp\",       \"site\": [         {           \"name\": \"se-loennstorp\",           \"point\": [             {               \"name\": \"55-c\",               \"latitude\": 55.6671111111111,               \"longitude\": 13.1156944444444,               \"setting\": \"uniform\",               \"sampling_log\": [                 {                   \"name\": \"se-loennstorp-20240815\",                   \"date_stamp\": \"20240815\",                   \"person__email\": \"sampling@gmail.com\",                   \"sample\": [                     {                       \"name\": \"se-loennstorp-20240815_55-c_0-20\",                       \"min_depth\": 0,                       \"max_depth\": 20,                       \"observation\": [                         {                           \"sample_preparation__name\": \"mixed-untreated-soil-in-lab\",                           \"person__email\": \"analysis@gmail.com\",                           \"subsample\": \"a\",                           \"replicate\": 0,                           \"n_repeats\": 1,                           \"date_stamp\": \"20240815\",                           \"logistic\": {                             \"sample_preservation__name\": null,                             \"sample_transport__name\": null,                             \"transport_duration_h\": 0,                             \"sample_storage__name\": null,                             \"storage_duration_h\": 0,                             \"person__email\": \"logistic@gmail.com\"                           },                           \"analysis_method\": {                             \"digit-soil-sear\": [                               {                                 \"digitsoil\": [                                   {                                     \"value\": 3.5,                                     \"unit__name\": \"pmol*min^-1\",                                     \"indicator__name\": \"digit-soil-sear_LEU\",                                     \"analysis_method__name\": \"digit-soil-sear\",                                     \"instrument_brand__name\": \"digit-soil\",                                     \"instrument_model__name\": \"bob\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 3.8288,                                     \"unit__name\": \"pmol*min^-1\",                                     \"indicator__name\": \"digit-soil-sear_GLA\",                                     \"procedure\": \"digit-soil-sear\",                                     \"analysis_method__name\": \"digit-soil-sear\",                                     \"instrument_brand__name\": \"digit-soil\",                                     \"instrument_model__name\": \"bob\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 6.5383,                                     \"unit__name\": \"pmol*min^-1\",                                     \"indicator__name\": \"digit-soil-sear_GLS\",                                     \"procedure\": \"digit-soil-sear\",                                     \"analysis_method__name\": \"digit-soil-sear\",                                     \"instrument_brand__name\": \"digit-soil\",                                     \"instrument_model__name\": \"bob\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 7.9978,                                     \"unit__name\": \"pmol*min^-1\",                                     \"indicator__name\": \"digit-soil-sear_PHO\",                                     \"procedure\": \"digit-soil-sear\",                                     \"analysis_method__name\": \"digit-soil-sear\",                                     \"instrument_brand__name\": \"digit-soil\",                                     \"instrument_model__name\": \"bob\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.5,                                     \"unit__name\": \"pmol*min^-1\",                                     \"indicator__name\": \"digit-soil-sear_XYL\",                                     \"procedure\": \"digit-soil-sear\",                                     \"analysis_method__name\": \"digit-soil-sear\",                                     \"instrument_brand__name\": \"digit-soil\",                                     \"instrument_model__name\": \"bob\",                                     \"instrument_id\": \"0\"                                   }                                 ]                               }                             ]                           }                         }                       ]                     }                   ]                 }               ]             }           ]         }       ]     }   ] }    Data access and file naming   The data is provided under the following licenses:      Data License: Creative Commons Attribution license (CC-BY)   Code License: Massachusetts Institute of Technology License (MIT License)   Recorded data from the enzymatic activity analysis are available at:      Flat json format https://github.com/AI4SH/in-situ_data/tree/main/flat/digit-soil_sear   Nested json format: https://github.com/AI4SH/in-situ_data/tree/main/nested/digit-soil_sear   The file naming is defined to separate all eventual renewed sampling or analysis efforts and is built up from the following metadata:     country   pilot site   sampling date   point id   min depth   max depth   subsample   replicate   setting   sample_preservation   procedure   instrument   analysis date   For the examples above, the file name thus have the following component values:      se   loennstorp   20240815   55-c   0   20   a   0   uniform   mix-wet   bob   0   20240815   And the final file name becomes:   se-loennstorp-20240815_55-c_0-20_a_0_uniform_mix-wet_bob_0_20240815.json   Acknowledgments and Funding   This work is part of the AI4SoilHealth project, funded by the European Union’s Horizon Europe Research and Innovation Programme under Grant Agreement No. 101086179.   Funded by the European Union. The views expressed are those of the authors and do not necessarily reflect those of the European Union or the European Research Executive Agency.   ","categories": [],
        "tags": [],
        "url": "/in-situ_data_docs/docs/in-situ_methods/digitsoil-new/",
        "teaser": null
      },{
        "title": "eDNA",
        "excerpt":"environmental DNA (eDNA) analysis is a complex laboratory bound chain of processes called metabarcoding. Also the sampling in the field requires special equipment and the sample has to be preserved with a shielding liquid preserving the DNA during storage and transport. The raw eDNA result is a dataset of nucleotid sequences, that is then compared to database libraries (bioinformatic treatment) for determining both the organisms (or orgamism groups) and the processes that occur in the soil.   With growing DNA sequence libraries and reduced costs for the metabarcoding, progressively more and more information can be obtained from eDNA analysis. eDNA metabarcoding is thus increasingly becoming an important method for soil health characterisation.   Short method description   The field sampling requires care to avoid DNA contamination from e.g. human DNA. Once extracted only a very small sample (milligram) is required. The sample must be preserved to prevent any deterioration or metabolic changes in the DNA nuceoltides and that requires a DNA sheild to be added to the sample. Once the DNA shield is added, the sample can be stored and transported.   The pipeline of processes that compose eDNA metabarcoding in the laboratry include:     extraction,   amplification,   purification,   sequencing, and   bioinformatic treatment (database library mining).   The eDNA analysed as part of AI4SH focused on three taxonomic domains:     Bacteria,   Archaea, and   Eukarya.   For the Eukarya the analyses were restricted to fungi.   For each group the richness (total number of species - recorded as alpha and chao1 indexes), diversity (recorded as alpha Shannon and alpha Simpson indexes) and evenness (pielo index) are recorded. Group related soil functions included for example chemoheterotrophy, human pathogens, plant pathogens and nitrogen fixation. Table 1 is a complete list of eDNA derived organism groups and functions.   Table 1. eDNA derived organism groups and functions recorded with each observation with the applied metabarcoding pipelines.                  Property       Indicator       AI4SH extended naming       Unit                       Prokaryotes                                         alpha observed richness       Prokaryotes alpha richness       metabarcoding_Prokaryotes-alpha-observed-richness       unitless                 alpha chao1 estimated richness       Prokaryotes chao1 richness       metabarcoding_Prokaryotes-alpha-chao1-estimated-richness       unitless                 alpha dominance       Prokaryotes alpha dominance       metabarcoding_Prokaryotes-alpha-dominance       unitless                 alpha pielou e       Prokaryotes Pielou evenness       metabarcoding_Prokaryotes-alpha-pielou-e       unitless                 alpha shannon       Prokaryotes alpha Shannon diversity       metabarcoding_Prokaryotes-alpha-shannon       unitless                 alpha simpson       Prokaryotes alpha Simpson diversity       metabarcoding_Prokaryotes-alpha-simpson       unitless                 functional prediction chemoheterotrophy       Prokaryotes chemoheterotrophy       metabarcoding_Prokaryotes-functional-prediction-chemoheterotrophy       unitless                 functional prediction human pathogens all       human pathogens       metabarcoding_Prokaryotes-functional-prediction-human-pathogens-all       unitless                 functional prediction nitrogen fixation       Nitrogen fixation       metabarcoding_Prokaryotes-functional-prediction-nitrogen-fixation       unitless                 Fungi                                         alpha observed richness       Fungi alpha richness       metabarcoding_Fungi-alpha-observed-richness       unitless                 alpha chao1 estimated richness       Fungi chao1 richness       metabarcoding_Fungi-alpha-chao1-estimated-richness       unitless                 alpha dominance       Fungi alpha dominance       metabarcoding_Fungi-alpha-dominance       uitless                 alpha pielou e       Fungi Pielou evenness       metabarcoding_Fungi-alpha-pielou-e       unitless                 alpha shannon       Fungi alpha shannon diversity       metabarcoding_Fungi-alpha-shannon       unitless                 alpha simpson       Fungi alpha simpson diversity       metabarcoding_Fungi-alpha-simpson       unitless                 funtional prediction Ectomycorrhizal fungi       Ectomycorrhizal fungi function       metabarcoding_Fungi-funtional-prediction-Ectomycorrhizal-fungi       unitless                 funtional prediction Arbuscular mycorrhizal fungi       Arbuscular mycorrhizal fungi function       metabarcoding_Fungi-funtional-prediction-Arbuscular-mycorrhizal-fungi       unitless                 funtional prediction fungal saprotrophs       Saprotrophic fungi function       metabarcoding_Fungi-funtional-prediction-fungal-saprotrophs       unitless                 funtional prediction fungal plant pathogens       Plant pathogenic fungi function       metabarcoding_Fungi-funtional-prediction-fungal-plant-pathogens       unitless           Detailed method description   The metabarcoding of the eDNA samples were all done by Novogene; all pipeline processing is outlined in table 2.   Table 2. Metabarcoding pipeline processes applied for the eDNA analysis in AI4SH.                  pipeline stage       descritpion/reference                               Prokarypties                                 extraction       ZymoBIOMICS DNA Miniprep Kit                         amplification - forward priming       515f Modified GTGYCAGCMGCCGCGGTAA                         amplification - Reverse priming       806r Modified GGACTACNVGGGTWTCTAAT                         purification       Purify the PCR products using magnetic bead purification. Mix samples in equidensity ratios based on the concentration of PCR products                         sequencing       platform: Illumina; strategy: 2x250; data requirement: 30K tags                         reference library       Silva                         bioinformatics       Microbiome Marker Gene Analysis with Qiime2-DADA2: Assign paired-end reads to samples based on their unique barcode and truncate by cutting off the barcode and primer sequences. Merge paired-end reads using FLASH (Magoč and Salzberg, 2011. Quality filter the raw tags using the fastp software to obtain high-quality Clean Tags (Bokulich et al., 2013). Compare the tags with the reference database to detect chimera sequences, then obtain the Effective Tags by removing the chimera sequences with the vsearch package (Edgar et al., 2011). For the Effective Tags obtained previously, denoise with DADA2 module in the QIIME2 software to obtain initial ASVs (Amplicon Sequence Variants). Perform species annotation using QIIME2 software.                         functional prediction       Faprotax                         nucleotide sequence library       Sequence Read Archive                         Fungi                                 extraction       ZymoBIOMICS DNA Miniprep Kit                         amplification - forward priming       gITS7 GTGARTCATCGARTCTTTG                         amplification - reverse priming       ITS4 TCCTCCGCTTATTGATATGC                         purification       Purify the PCR products using magnetic bead purification. Mix samples in equidensity ratios based on the concentration of PCR products                         sequencing       platform: Illumina; strategy: 2x250; data requirement: 30K tags                         reference library       Unite                         bioinformatics       Microbiome Marker Gene Analysis with Qiime2-DADA2: Assign paired-end reads to samples based on their unique barcode and truncate by cutting off the barcode and primer sequences. Merge paired-end reads using FLASH (Magoč and Salzberg, 2011). Quality filter the raw tags using the fastp software to obtain high-quality Clean Tags (Bokulich et al., 2013). Compare the tags with the reference database to detect chimera sequences, then obtain the Effective Tags by removing the chimera sequences with the vsearch package (Edgar et al., 2011). For the Effective Tags obtained previously, denoise with DADA2 module in the QIIME2 software to obtain initial ASVs (Amplicon Sequence Variants). Perform species annotation using QIIME2 software.                         functional prediction       Fungild                         nucleotide sequence library       Sequence Read Archive                   Data format   The AI4SH observed eDNA derived organism groups and functions data are available in 2 differently formatted json objects, a flat version more suitable for direct accessibility and a nested version intended for users who wants to convert the data to a relational sql database. The flat version does not include any arrays whereas the nested version is structured with nested arrays.   Flat json object   Below is an example of a flat json object for eDNA derived organism groups and functions observations. Object keys with a double underscore denote a link to other entries in the database. For example “person__email” is a key for the person responsible, with more detailed contact information in a separate json document. The hashtag comments are not allowed when operating a json file, they are added here for explaining each json object.   {   \"data_source\": \"ai4sh_se-loennstorp\", # 'project'+'_'+'country'+'-'+'pilot site'   \"sampling_log\": { # Assembly of samples collected in the same day at a particular pilot site       \"name\": \"se-loennstorp-20240815\", # 'country'+'-'+'pilot site'+'-'+'sampling date'     \"date_stamp\": \"20240815\", # Date when the sample was collected in the field     \"person__email\": \"sampling@gmail.com\" # Email of person responsible for the sampling   },   \"sample\": \"se-loennstorp-20240815_55-c_0-20\", # 'country'+'-'+'pilot site'+'-'+'sampling date'+'_'+'sample point id'+'_'+'min depth''+'-'+'max depth'   \"locus\": { # Assembly of the point location, setting and depth     \"position__name\": \"se-loennstorp_55-c\", # 'country'+'-'+'pilot site'+'-'+'sample point id' used for checking internal consistency     \"point\": \"55-c\", # sample point id     \"setting\": \"uniform\", # The spatial setting in the landscape (uniform, alley, plantline, edge)     \"latitude\": 55.6671111111111, # Latitude in WGS84     \"longitude\": 13.1156944444444, # Longitude in WGS84     \"min_depth\": 0, # Sample upper limit in cm in soil profile     \"max_depth\": 20 # Sample lower limit in cm in soil profile   },   \"observation\": { # Assembly of metadata and data related to the observation     sample_preparation__name\": \"mixed-untreated-soil-in-lab\", # Explanatory name of soil treatment before observation     \"person__email\": \"analysis@gmail.com\", # Email of person responsible for the observation     \"subsample\": \"a\", # For registering if physically different soil subsamples were analysed separately, default for a single analysis = a     \"replicate\": 0, # If the analysis was replicated on the same subsample more than once, the first test is always set to 0     \"n_repeats\": 1, # The number of repetitions done by any instrument on a single subsample and as part of a single replicate, with results reported as mean or mean and standard deviation     \"date_stamp\": \"20241110\", # Date when the analysis was done     \"logistic\": { # Assembly of logistical metadata on the handling of the sample       \"sample_preservation__name\": \"dna shield\", # Explanatory name of soil preservation       \"sample_transport__name\": \"cold\", # Explanatory name of soil transportation if sensitive for analysis results       \"transport_duration_h\": 14, # duration of any sensitive transportation in hours       \"sample_storage__name\": \"cold\",  # Explanatory name of soil storage if sensitive for analysis results       \"storage_duration_h\": 2088, # Storage duration in hours       \"person__email\": \"logistic@gmail.com\" # Email of person responsible for the logistics     },     \"analysis\": { # Assembly of analysis results, procedures and instruments       \"metabarcoding_Prokaryotes-alpha-observed-richness\": { # Soil property reported as AI4SH extended naming (part after underscore = core indicator)         \"value\": 1219.0, # Recorded value (single value or mean of repeated observations)         \"unit__name\": \"unitless\", # Recorded unit         \"indicator__name\": \"metabarcoding_Prokaryotes-alpha-observed-richness\", # AI4SH extended name of the soil indicator (part after underscore = core indicator)         \"procedure\": \"metabarcoding\", # Procedure for the analysis         \"analysis_method__name\": \"edna-prokaryotes-alpha-observed-richness\", # Name of the method         \"instrument_brand__name\": \"edna-prokaryotes-alpha-observed-richness\", # Instrument type or brand         \"instrument_model__name\": \"metabarcoding-chain\",  # Model or version of the instrument         \"instrument_id\": \"0\" # Any individual instrument id, the default for unknown id is       },       \"metabarcoding_Prokaryotes-alpha-chao1-estimated-richness\": {         \"value\": 1247.879,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Prokaryotes-alpha-chao1-estimated-richness\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-prokaryotes-alpha-chao1-estimated-richness\",         \"instrument_brand__name\": \"edna-prokaryotes-alpha-chao1-estimated-richness\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Prokaryotes-alpha-dominance\": {         \"value\": 0.004,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Prokaryotes-alpha-dominance\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-prokaryotes-alpha-dominance\",         \"instrument_brand__name\": \"edna-prokaryotes-alpha-dominance\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Prokaryotes-alpha-pielou-e\": {         \"value\": 0.887,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Prokaryotes-alpha-pielou-e\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-prokaryotes-alpha-pielou-e\",         \"instrument_brand__name\": \"edna-prokaryotes-alpha-pielou-e\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Prokaryotes-alpha-shannon\": {         \"value\": 9.094,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Prokaryotes-alpha-shannon\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-prokaryotes-alpha-shannon\",         \"instrument_brand__name\": \"edna-prokaryotes-alpha-shannon\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Prokaryotes-alpha-simpson\": {         \"value\": 0.996,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Prokaryotes-alpha-simpson\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-prokaryotes-alpha-simpson\",         \"instrument_brand__name\": \"edna-prokaryotes-alpha-simpson\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Prokaryotes-functional-prediction-chemoheterotrophy\": {         \"value\": 0.110063849194,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Prokaryotes-functional-prediction-chemoheterotrophy\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-prokaryotes-functional-prediction-chemoheterotrophy\",         \"instrument_brand__name\": \"edna-prokaryotes-functional-prediction-chemoheterotrophy\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Prokaryotes-functional-prediction-human-pathogens-all\": {         \"value\": 0.000337826424783,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Prokaryotes-functional-prediction-human-pathogens-all\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-prokaryotes-functional-prediction-human-pathogens-all\",         \"instrument_brand__name\": \"edna-prokaryotes-functional-prediction-human-pathogens-all\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Prokaryotes-functional-prediction-nitrogen-fixation\": {         \"value\": 0.0,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Prokaryotes-functional-prediction-nitrogen-fixation\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-prokaryotes-functional-prediction-nitrogen-fixation\",         \"instrument_brand__name\": \"edna-prokaryotes-functional-prediction-nitrogen-fixation\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Fungi-alpha-observed-richness\": {         \"value\": 537.0,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Fungi-alpha-observed-richness\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-fungi-alpha-observed-richness\",         \"instrument_brand__name\": \"edna-fungi-alpha-observed-richness\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Fungi-alpha-chao1-estimated-richness\": {         \"value\": 543.6,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Fungi-alpha-chao1-estimated-richness\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-fungi-alpha-chao1-estimated-richness\",         \"instrument_brand__name\": \"edna-fungi-alpha-chao1-estimated-richness\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Fungi-alpha-dominance\": {         \"value\": 0.044,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Fungi-alpha-dominance\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-fungi-alpha-dominance\",         \"instrument_brand__name\": \"edna-fungi-alpha-dominance\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Fungi-alpha-pielou-e\": {         \"value\": 0.671,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Fungi-alpha-pielou-e\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-fungi-alpha-pielou-e\",         \"instrument_brand__name\": \"edna-fungi-alpha-pielou-e\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Fungi-alpha-shannon\": {         \"value\": 6.082,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Fungi-alpha-shannon\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-fungi-alpha-shannon\",         \"instrument_brand__name\": \"edna-fungi-alpha-shannon\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Fungi-alpha-simpson\": {         \"value\": 0.956,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Fungi-alpha-simpson\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-fungi-alpha-simpson\",         \"instrument_brand__name\": \"edna-fungi-alpha-simpson\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Fungi-funtional-prediction-Ectomycorrhizal-fungi\": {         \"value\": 0.00284074439098,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Fungi-funtional-prediction-Ectomycorrhizal-fungi\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-fungi-funtional-prediction-ectomycorrhizal-fungi\",         \"instrument_brand__name\": \"edna-fungi-funtional-prediction-ectomycorrhizal-fungi\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Fungi-funtional-prediction-Arbuscular-mycorrhizal-fungi\": {         \"value\": 0.000135273542428,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Fungi-funtional-prediction-Arbuscular-mycorrhizal-fungi\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-fungi-funtional-prediction-arbuscular-mycorrhizal-fungi\",         \"instrument_brand__name\": \"edna-fungi-funtional-prediction-arbuscular-mycorrhizal-fungi\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Fungi-funtional-prediction-fungal-saprotrophs\": {         \"value\": 0.381471389645888,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Fungi-funtional-prediction-fungal-saprotrophs\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-fungi-funtional-prediction-fungal-saprotrophs\",         \"instrument_brand__name\": \"edna-fungi-funtional-prediction-fungal-saprotrophs\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Fungi-funtional-prediction-fungal-plant-pathogens\": {         \"value\": 0.395404564515375,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Fungi-funtional-prediction-fungal-plant-pathogens\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-fungi-funtional-prediction-fungal-plant-pathogens\",         \"instrument_brand__name\": \"edna-fungi-funtional-prediction-fungal-plant-pathogens\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       }     }   } }   Nested json object   The nested json object for the eDNA derived organism groups and functions observations contains the same object keys and values as the flat version, but hierarchically organised with nested arrays. For explanations of the object keys please see the flat json object above.   {   \"data_source\": [     {       \"name\": \"ai4sh_se-loennstorp\",       \"site\": [         {           \"name\": \"se-loennstorp\",           \"point\": [             {               \"name\": \"55-c\",               \"latitude\": 55.6671111111111,               \"longitude\": 13.1156944444444,               \"setting\": \"uniform\",               \"sampling_log\": [                 {                   \"name\": \"se-loennstorp-20240815\",                   \"date_stamp\": \"20240815\",                   \"person__email\": \"sampling@gmail.com\",                   \"sample\": [                     {                       \"name\": \"se-loennstorp-20240815_55-c_0-20\",                       \"min_depth\": 0,                       \"max_depth\": 20,                       \"observation\": [                         {                           \"sample_preparation__name\": \"mixed-untreated-soil-in-lab\",                           \"person__email\": \"analysis@gmail.com\",                           \"subsample\": \"a\",                           \"replicate\": 0,                           \"n_repeats\": 1,                           \"date_stamp\": \"20241110\",                           \"logistic\": {                             \"sample_preservation__name\": \"dna shield\",                             \"sample_transport__name\": \"cold\",                             \"transport_duration_h\": 14,                             \"sample_storage__name\": \"cold\",                             \"storage_duration_h\": 2088,                             \"person__email\": \"logistic@gmail.com\"                           },                           \"analysis_method\": {                             \"metabarcoding\": [                               {                                 \"edna\": [                                   {                                     \"value\": 1219.0,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Prokaryotes-alpha-observed-richness\",                                     \"analysis_method__name\": \"edna-prokaryotes-alpha-observed-richness\",                                     \"instrument_brand__name\": \"edna-prokaryotes-alpha-observed-richness\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 1247.879,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Prokaryotes-alpha-chao1-estimated-richness\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-prokaryotes-alpha-chao1-estimated-richness\",                                     \"instrument_brand__name\": \"edna-prokaryotes-alpha-chao1-estimated-richness\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.004,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Prokaryotes-alpha-dominance\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-prokaryotes-alpha-dominance\",                                     \"instrument_brand__name\": \"edna-prokaryotes-alpha-dominance\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.887,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Prokaryotes-alpha-pielou-e\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-prokaryotes-alpha-pielou-e\",                                     \"instrument_brand__name\": \"edna-prokaryotes-alpha-pielou-e\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 9.094,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Prokaryotes-alpha-shannon\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-prokaryotes-alpha-shannon\",                                     \"instrument_brand__name\": \"edna-prokaryotes-alpha-shannon\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.996,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Prokaryotes-alpha-simpson\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-prokaryotes-alpha-simpson\",                                     \"instrument_brand__name\": \"edna-prokaryotes-alpha-simpson\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.110063849194,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Prokaryotes-functional-prediction-chemoheterotrophy\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-prokaryotes-functional-prediction-chemoheterotrophy\",                                     \"instrument_brand__name\": \"edna-prokaryotes-functional-prediction-chemoheterotrophy\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.000337826424783,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Prokaryotes-functional-prediction-human-pathogens-all\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-prokaryotes-functional-prediction-human-pathogens-all\",                                     \"instrument_brand__name\": \"edna-prokaryotes-functional-prediction-human-pathogens-all\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.0,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Prokaryotes-functional-prediction-nitrogen-fixation\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-prokaryotes-functional-prediction-nitrogen-fixation\",                                     \"instrument_brand__name\": \"edna-prokaryotes-functional-prediction-nitrogen-fixation\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 537.0,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Fungi-alpha-observed-richness\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-fungi-alpha-observed-richness\",                                     \"instrument_brand__name\": \"edna-fungi-alpha-observed-richness\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 543.6,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Fungi-alpha-chao1-estimated-richness\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-fungi-alpha-chao1-estimated-richness\",                                     \"instrument_brand__name\": \"edna-fungi-alpha-chao1-estimated-richness\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.044,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Fungi-alpha-dominance\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-fungi-alpha-dominance\",                                     \"instrument_brand__name\": \"edna-fungi-alpha-dominance\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.671,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Fungi-alpha-pielou-e\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-fungi-alpha-pielou-e\",                                     \"instrument_brand__name\": \"edna-fungi-alpha-pielou-e\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 6.082,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Fungi-alpha-shannon\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-fungi-alpha-shannon\",                                     \"instrument_brand__name\": \"edna-fungi-alpha-shannon\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.956,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Fungi-alpha-simpson\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-fungi-alpha-simpson\",                                     \"instrument_brand__name\": \"edna-fungi-alpha-simpson\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.00284074439098,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Fungi-funtional-prediction-Ectomycorrhizal-fungi\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-fungi-funtional-prediction-ectomycorrhizal-fungi\",                                     \"instrument_brand__name\": \"edna-fungi-funtional-prediction-ectomycorrhizal-fungi\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.000135273542428,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Fungi-funtional-prediction-Arbuscular-mycorrhizal-fungi\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-fungi-funtional-prediction-arbuscular-mycorrhizal-fungi\",                                     \"instrument_brand__name\": \"edna-fungi-funtional-prediction-arbuscular-mycorrhizal-fungi\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.381471389645888,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Fungi-funtional-prediction-fungal-saprotrophs\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-fungi-funtional-prediction-fungal-saprotrophs\",                                     \"instrument_brand__name\": \"edna-fungi-funtional-prediction-fungal-saprotrophs\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.395404564515375,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Fungi-funtional-prediction-fungal-plant-pathogens\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-fungi-funtional-prediction-fungal-plant-pathogens\",                                     \"instrument_brand__name\": \"edna-fungi-funtional-prediction-fungal-plant-pathogens\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   }                                 ]                               }                             ]                           }                         }                       ]                     }                   ]                 }               ]             }           ]         }       ]     }   ] }   Data access and file naming   The data is provided under the following licenses:      Data License: Creative Commons Attribution license (CC-BY)   Code License: Massachusetts Institute of Technology License (MIT License)   Recorded data from the eDNA metabarcoding analysis are available at:      Flat json format https://github.com/AI4SH/in-situ_data/tree/main/flat/edna   Nested json format: https://github.com/AI4SH/in-situ_data/tree/main/nested/edna   The file naming is defined to separate all eventual renewed sampling or analysis efforts and is built up from the following metadata:     country   pilot site   sampling date   point id   min depth   max depth   subsample   replicate   setting   sample_preservation   procedure   instrument   analysis date   For the examples above, the file name thus have the following component values:      se   loennstorp   20240815   55-c   0   20   a   0   uniform   mix-wet   metabarcoding-chain   0   20241110   And the final file name becomes:   se-loennstorp-20240815_55-c_0-20_a_0_uniform_mix-wet_metabarcoding-chain_0_20241110.json   Acknowledgments and Funding   This work is part of the AI4SoilHealth project, funded by the European Union’s Horizon Europe Research and Innovation Programme under Grant Agreement No. 101086179.   Funded by the European Union. The views expressed are those of the authors and do not necessarily reflect those of the European Union or the European Research Executive Agency.   ","categories": [],
        "tags": [],
        "url": "/in-situ_data_docs/docs/in-situ_methods/edna/",
        "teaser": null
      },{
        "title": "Infiltration",
        "excerpt":"Infiltration, the capacity of soil to swallow water, is a prerequisite for soils and their ecosystems to function. Reduced infiltration capacity, due e.g. to compaction or water repellant conditions arising from drought, makes soils more prone to erosion and less resilient to droughts and storms. Low infiltration capacity also exacerbate flash floods and downstream flooding. As part of AI4SH a fairly simple singe-ring infiltration method (denoted as Beerkan method) is used that allows to link the infiltration rates to various soil hydraulic properties   Short method description   Single ring infiltration is a published scientific method for estimating soil hydraulic properties (Lassabatere et al., 2006). The ring can easily be constructed from a food tin can and inserted ~1 cm into the soil. Small, fixed volumes of water (typically 50 to 100 ml) are sequentially poured into the ring and the time it takes for the water to infiltrate is recorded. The pouring of water continuous until the time it takes for it to infiltrate is stable. To calculate the hydraulic properties (table 1), data on bulk density and soil moisture before and after the infiltration test are needed - e.g. obtained using the soil cylinder bulk density and volumetric soil moisture content method.   Table 1. Hydarulic soil properties recorded with each single ring infiltration observation.                  Property       Indicator       AI4SH extended naming       Unit                       infiltration rate       infiltration       single-ring-infiltration_final-infiltration-rate       m*day^-1                 porosity       porosity       single-ring-infiltration_porosity       vol*vol^-1                 sorptivity       sorptivity       single-ring-infiltration_sorptivity       m*day^-1/2                 saturated hydraulic conductivity       hydraulic conductivity       single-ring-infiltration_saturated-hydraulic-conductivity       m*day^-1                 field capacity       field capacity       single-ring-infiltration_field-capacity       vol*vol^-1                 plant avaiable water       plant available water       single-ring-infiltration_plant-available-water       vol*vol^-1                 saturated water content       saturated water content       single-ring-infiltration_saturated-water-content       vol*vol^-1                 hg water pressure head scale parameter       hg water pressure head       single-ring-infiltration_hg-water-pressure-head-scale-parameter       m                 n shape parameter of retention curve       retention curve n       single-ring-infiltration_n-shape-parameter-of-retention-curve       unitless                 m shape parameter of retention curve       retention curve m       single-ring-infiltration_n-shape-parameter-of-retention-curve       unitless           Data format   The AI4SH observed infiltration derived data are available in 2 differently formatted json objects, a flat version more suitable for direct accessibility and a nested version intended for users who wants to convert the data to a relational sql database. The flat version does not include any arrays whereas the nested version is structured with nested arrays.   Flat json object   Below is an example of a flat json object for infiltration derived observations. Object keys with a double underscore denote a link to other entries in the database. For example “person__email” is a key for the person responsible, with more detailed contact information in a separate json document. The hashtag comments are not allowed when operating a json file, they are added here for explaining each json object.   {   \"data_source\": \"ai4sh_se-loennstorp\", # 'project'+'_'+'country'+'-'+'pilot site'   \"sampling_log\": { # Assembly of samples collected in the same day at a particular pilot site       \"name\": \"se-loennstorp-20240815\", # 'country'+'-'+'pilot site'+'-'+'sampling date'     \"date_stamp\": \"20240815\", # Date when the sample was collected in the field     \"person__email\": \"sampling@gmail.com\" # Email of person responsible for the sampling   },   \"sample\": \"se-loennstorp-20240815_55-c_0-20\", # 'country'+'-'+'pilot site'+'-'+'sampling date'+'_'+'sample point id'+'_'+'min depth''+'-'+'max depth'   \"locus\": { # Assembly of the point location, setting and depth     \"position__name\": \"se-loennstorp_55-c\", # 'country'+'-'+'pilot site'+'-'+'sample point id' used for checking internal consistency     \"point\": \"55-c\", # sample point id     \"setting\": \"uniform\", # The spatial setting in the landscape (uniform, alley, plantline, edge)     \"latitude\": 55.6671111111111, # Latitude in WGS84     \"longitude\": 13.1156944444444, # Longitude in WGS84     \"min_depth\": 0, # Sample upper limit in cm in soil profile     \"max_depth\": 20 # Sample lower limit in cm in soil profile   },   \"observation\": { # Assembly of metadata and data related to the observation     sample_preparation__name\": \"soil-undisturbed-in-situ\", # Explanatory name of soil treatment before observation     \"person__email\": \"analysis@gmail.com\", # Email of person responsible for the observation     \"subsample\": \"a\", # For registering if physically different soil subsamples were analysed separately, default for a single analysis = a     \"replicate\": 0, # If the analysis was replicated on the same subsample more than once, the first test is always set to 0     \"n_repeats\": 1, # The number of repetitions done by any instrument on a single subsample and as part of a single replicate, with results reported as mean or mean and standard deviation     \"date_stamp\": \"20240815\", # Date when the analysis was done     \"logistic\": { # Assembly of logistical metadata on the handling of the sample       \"sample_preservation__name\": null, # Explanatory name of soil preservation       \"sample_transport__name\": null, # Explanatory name of soil transportation if sensitive for analysis results       \"transport_duration_h\": 0, # duration of any sensitive transportation in hours       \"sample_storage__name\": null, # Explanatory name of soil storage if sensitive for analysis results       \"storage_duration_h\": 0, # Storage duration in hours       \"person__email\": \"logistic@gmail.com\" # Email of person responsible for the logistics     },     \"analysis\": { # Assembly of analysis results, procedures and instruments       \"single-ring-infiltration_porosity\": { # Soil property reported as AI4SH extended naming (part after underscore = core indicator)         \"value\": 0.53,  # Recorded value (single value or mean of repeated observations)         \"unit__name\": \"vol*vol^-1\",  # Recorded unit         \"indicator__name\": \"single-ring-infiltration_porosity\",  # AI4SH extended name of the soil indicator (part after underscore = core indicator)         \"procedure\": \"single-ring-infiltration\", # Procedure for the analysis (not always similar to method)         \"analysis_method__name\": \"single-ring-infiltration\", # Name of the method         \"instrument_brand__name\": \"single-ring\", # Instrument type or brand         \"instrument_model__name\": \"tin-can\", # Model or version of the instrument         \"instrument_id\": \"0\" # Any individual instrument id, the default for unknown id is \"0\"       },       \"single-ring-infiltration_final-infiltration-rate\": {         \"value\": 10.8524,         \"unit__name\": \"m*day^-1\",         \"indicator__name\": \"single-ring-infiltration_final-infiltration-rate\",         \"procedure\": \"single-ring-infiltration\",         \"analysis_method__name\": \"single-ring-infiltration\",         \"instrument_brand__name\": \"single-ring\",         \"instrument_model__name\": \"tin-can\",         \"instrument_id\": \"0\"       },       \"single-ring-infiltration_sorptivity\": {         \"value\": 0.291019,         \"unit__name\": \"m*day^-1/2\",         \"indicator__name\": \"single-ring-infiltration_sorptivity\",         \"procedure\": \"single-ring-infiltration\",         \"analysis_method__name\": \"single-ring-infiltration\",         \"instrument_brand__name\": \"single-ring\",         \"instrument_model__name\": \"tin-can\",         \"instrument_id\": \"0\"       },       \"single-ring-infiltration_saturated-hydraulic-conductivity\": {         \"value\": 7.61166,         \"unit__name\": \"m*day^-1\",         \"indicator__name\": \"single-ring-infiltration_saturated-hydraulic-conductivity\",         \"procedure\": \"single-ring-infiltration\",         \"analysis_method__name\": \"single-ring-infiltration\",         \"instrument_brand__name\": \"single-ring\",         \"instrument_model__name\": \"tin-can\",         \"instrument_id\": \"0\"       },       \"single-ring-infiltration_field-capacity\": {         \"value\": 0.206606,         \"unit__name\": \"vol*vol^-1\",         \"indicator__name\": \"single-ring-infiltration_field-capacity\",         \"procedure\": \"single-ring-infiltration\",         \"analysis_method__name\": \"single-ring-infiltration\",         \"instrument_brand__name\": \"single-ring\",         \"instrument_model__name\": \"tin-can\",         \"instrument_id\": \"0\"       },       \"single-ring-infiltration_plant-available-water\": {         \"value\": 0.0981831,         \"unit__name\": \"vol*vol^-1\",         \"indicator__name\": \"single-ring-infiltration_plant-available-water\",         \"procedure\": \"single-ring-infiltration\",         \"analysis_method__name\": \"single-ring-infiltration\",         \"instrument_brand__name\": \"single-ring\",         \"instrument_model__name\": \"tin-can\",         \"instrument_id\": \"0\"       },       \"single-ring-infiltration_saturated-water-content\": {         \"value\": 0.53,         \"unit__name\": \"vol*vol^-1\",         \"indicator__name\": \"single-ring-infiltration_saturated-water-content\",         \"procedure\": \"single-ring-infiltration\",         \"analysis_method__name\": \"single-ring-infiltration\",         \"instrument_brand__name\": \"single-ring\",         \"instrument_model__name\": \"tin-can\",         \"instrument_id\": \"0\"       },       \"single-ring-infiltration_hg-water-pressure-head-scale-parameter\": {         \"value\": 0.0124932,         \"unit__name\": \"mm\",         \"indicator__name\": \"single-ring-infiltration_hg-water-pressure-head-scale-parameter\",         \"procedure\": \"single-ring-infiltration\",         \"analysis_method__name\": \"single-ring-infiltration\",         \"instrument_brand__name\": \"single-ring\",         \"instrument_model__name\": \"tin-can\",         \"instrument_id\": \"0\"       },       \"single-ring-infiltration_n-shape-parameter-of-retention-curve\": {         \"value\": 2.16893,         \"unit__name\": \"unitless\",         \"indicator__name\": \"single-ring-infiltration_n-shape-parameter-of-retention-curve\",         \"procedure\": \"single-ring-infiltration\",         \"analysis_method__name\": \"single-ring-infiltration\",         \"instrument_brand__name\": \"single-ring\",         \"instrument_model__name\": \"tin-can\",         \"instrument_id\": \"0\"       },       \"single-ring-infiltration_m-shape-parameter-of-retention-curve\": {         \"value\": 0.0778884,         \"unit__name\": \"unitless\",         \"indicator__name\": \"single-ring-infiltration_m-shape-parameter-of-retention-curve\",         \"procedure\": \"single-ring-infiltration\",         \"analysis_method__name\": \"single-ring-infiltration\",         \"instrument_brand__name\": \"single-ring\",         \"instrument_model__name\": \"tin-can\",         \"instrument_id\": \"0\"       }     }   } }   Nested json object   The nested json object for soil infiltration derived observations contains the same object keys and values as the flat version, but hierarchically organised with nested arrays. For explanations of the object keys please see the flat json object above.   {   \"data_source\": [     {       \"name\": \"ai4sh_se-loennstorp\",       \"site\": [         {           \"name\": \"se-loennstorp\",           \"point\": [             {               \"name\": \"55-c\",               \"latitude\": 55.6671111111111,               \"longitude\": 13.1156944444444,               \"setting\": \"uniform\",               \"sampling_log\": [                 {                   \"name\": \"se-loennstorp-20240815\",                   \"date_stamp\": \"20240815\",                   \"person__email\": \"sampling@gmail.com\",                   \"sample\": [                     {                       \"name\": \"se-loennstorp-20240815_55-c_0-20\",                       \"min_depth\": 0,                       \"max_depth\": 20,                       \"observation\": [                         {                           \"sample_preparation__name\": \"soil-undisturbed-in-situ\",                           \"person__email\": \"analysis@gmail.com\",                           \"subsample\": \"a\",                           \"replicate\": 0,                           \"n_repeats\": 1,                           \"date_stamp\": \"20240815\",                           \"logistic\": {                             \"sample_preservation__name\": null,                             \"sample_transport__name\": null,                             \"transport_duration_h\": 0,                             \"sample_storage__name\": null,                             \"storage_duration_h\": 0,                             \"person__email\": \"logistic@gmail.com\"                           },                           \"analysis_method\": {                             \"single-ring-infiltration\": [                               {                                 \"single-ring infiltration\": [                                   {                                     \"value\": 0.53,                                     \"unit__name\": \"vol*vol^-1\",                                     \"indicator__name\": \"single-ring-infiltration_porosity\",                                     \"analysis_method__name\": \"single-ring-infiltration\",                                     \"instrument_brand__name\": \"single-ring\",                                     \"instrument_model__name\": \"tin-can\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 10.8524,                                     \"unit__name\": \"m*day^-1\",                                     \"indicator__name\": \"single-ring-infiltration_final-infiltration-rate\",                                     \"procedure\": \"single-ring-infiltration\",                                     \"analysis_method__name\": \"single-ring-infiltration\",                                     \"instrument_brand__name\": \"single-ring\",                                     \"instrument_model__name\": \"tin-can\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.291019,                                     \"unit__name\": \"m*day^-1/2\",                                     \"indicator__name\": \"single-ring-infiltration_sorptivity\",                                     \"procedure\": \"single-ring-infiltration\",                                     \"analysis_method__name\": \"single-ring-infiltration\",                                     \"instrument_brand__name\": \"single-ring\",                                     \"instrument_model__name\": \"tin-can\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 7.61166,                                     \"unit__name\": \"m*day^-1\",                                     \"indicator__name\": \"single-ring-infiltration_saturated-hydraulic-conductivity\",                                     \"procedure\": \"single-ring-infiltration\",                                     \"analysis_method__name\": \"single-ring-infiltration\",                                     \"instrument_brand__name\": \"single-ring\",                                     \"instrument_model__name\": \"tin-can\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.206606,                                     \"unit__name\": \"vol*vol^-1\",                                     \"indicator__name\": \"single-ring-infiltration_field-capacity\",                                     \"procedure\": \"single-ring-infiltration\",                                     \"analysis_method__name\": \"single-ring-infiltration\",                                     \"instrument_brand__name\": \"single-ring\",                                     \"instrument_model__name\": \"tin-can\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.0981831,                                     \"unit__name\": \"vol*vol^-1\",                                     \"indicator__name\": \"single-ring-infiltration_plant-available-water\",                                     \"procedure\": \"single-ring-infiltration\",                                     \"analysis_method__name\": \"single-ring-infiltration\",                                     \"instrument_brand__name\": \"single-ring\",                                     \"instrument_model__name\": \"tin-can\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.53,                                     \"unit__name\": \"vol*vol^-1\",                                     \"indicator__name\": \"single-ring-infiltration_saturated-water-content\",                                     \"procedure\": \"single-ring-infiltration\",                                     \"analysis_method__name\": \"single-ring-infiltration\",                                     \"instrument_brand__name\": \"single-ring\",                                     \"instrument_model__name\": \"tin-can\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.0124932,                                     \"unit__name\": \"mm\",                                     \"indicator__name\": \"single-ring-infiltration_hg-water-pressure-head-scale-parameter\",                                     \"procedure\": \"single-ring-infiltration\",                                     \"analysis_method__name\": \"single-ring-infiltration\",                                     \"instrument_brand__name\": \"single-ring\",                                     \"instrument_model__name\": \"tin-can\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 2.16893,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"single-ring-infiltration_n-shape-parameter-of-retention-curve\",                                     \"procedure\": \"single-ring-infiltration\",                                     \"analysis_method__name\": \"single-ring-infiltration\",                                     \"instrument_brand__name\": \"single-ring\",                                     \"instrument_model__name\": \"tin-can\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.0778884,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"single-ring-infiltration_m-shape-parameter-of-retention-curve\",                                     \"procedure\": \"single-ring-infiltration\",                                     \"analysis_method__name\": \"single-ring-infiltration\",                                     \"instrument_brand__name\": \"single-ring\",                                     \"instrument_model__name\": \"tin-can\",                                     \"instrument_id\": \"0\"                                   }                                 ]                               }                             ]                           }                         }                       ]                     }                   ]                 }               ]             }           ]         }       ]     }   ] }   Data access and file naming   The data is provided under the following licenses:      Data License: Creative Commons Attribution license (CC-BY)   Code License: Massachusetts Institute of Technology License (MIT License)   Recorded data from the infiltration derived analysis are available at:      Flat json format https://github.com/AI4SH/in-situ_data/tree/main/flat/infiltration   Nested json format: https://github.com/AI4SH/in-situ_data/tree/main/nested/infiltration   The file naming is defined to separate all eventual renewed sampling or analysis efforts and is built up from the following metadata:     country   pilot site   sampling date   point id   min depth   max depth   subsample   replicate   setting   sample_preservation   procedure   instrument   analysis date   For the examples above, the file name thus have the following component values:      se   loennstorp   20240815   55-c   0   20   a   0   uniform   no-prep   tin-can   0   20240815   And the final file name becomes:   se-loennstorp-20240815_55-c_0-20_a_0_uniform_no-prep_tin-can_0_20240815.json   Acknowledgments and Funding   This work is part of the AI4SoilHealth project, funded by the European Union’s Horizon Europe Research and Innovation Programme under Grant Agreement No. 101086179.   Funded by the European Union. The views expressed are those of the authors and do not necessarily reflect those of the European Union or the European Research Executive Agency.   ","categories": [],
        "tags": [],
        "url": "/in-situ_data_docs/docs/in-situ_methods/infiltration/",
        "teaser": null
      },{
        "title": "Ion Selective Electrode",
        "excerpt":"The ionic composition, together with water, to a large extent defines the non-biological living conditions in a soil ecosystem. Key ions include hydrogen (H+, usually expressed as pH), sodium (Na+), chloride (Cl-), potassium (K+), calcium (Ca2+), magnesium (Mg2+) and different nitrogen species like nitrate (NO3-) and ammonium (NH4+). These ions can all be determined using designated Ion Selective Electrodes (ISEs). ISEs measure the activity of specific ions, usually in a solution, by converting the electrical potential across a membrane that is only permeable to a specific ion. The concentration difference between a reference solution inside the ISE and the concentration in the solution causes a potential that is null (0) when the concentration across the membrane is equal. The larger the concentration difference the larger the absolute value of the potential. An ISE measures this potential, and from 2 or more solutions with known concentrations of the target ion, can be used to estimate the concentration of a particular ion in a solution with unknown concentrations of this particular ion.   Short method description   ISEs are sensitive instruments and require carefulness when handled and calibrated. The most widely used ISE type is for measuring pH, and pH electrodes are also cheaper and more stable compared to other ISEs. In AI4SH, pH ISEs for both direct observation in the field, and for soil diluted in distilled water (at a ratio 1:5) were tested (table 1).   Table 1. pH recorded Ion Selective Electrode (ISE) observations.                  Property       Indicator       AI4SH extended naming       Unit                       Hydrogen ion concentration       pH(soil)       xspectre-ise-ph-solid_ph(soil)       pH                 Hydrogen ion concentration       pH(water)       xspectre-ise-ph-liquid_ph(water)       pH           The ISE for direct observation in the field has a more robust tip that can be inserted directly into soft soils, or after pricking a small hole with a pencil or similarly tipped tool in more sturdy soils. It is despite this a sensitive and breakable tool and was only tested at a single pilot site. Ordinary ISEs are built for sampling solutions, and was tested at several pilot sites as part of the AI4SH in-situ sampling. Following the standard procedure when measuring pH, the AI4SH solution applied method diluted soil samples in distilled water at a ratio 1:5 and then measured the pH with a standard pH ISE. For calibration, pH buffers of pH=4.01 and pH=10.01 were used. The microcontroller unit for running and recording the pH ISE observations in AI4SH is constructed by the Swedish startup Xspectre.   Data format   Each pH ISE measurement, both in soil and soil:water solutions, is based on 6 replications and the recorded data report the mean value and the standard deviation. The observed pH data are available in 2 differently formatted json objects, a flat version more suitable for direct accessibility and a nested version intended for users who wants to convert the data to a relational sql database. The flat version does not include any arrays whereas the nested version is structured with nested arrays.   Flat json object   Below is an example of a flat json object for pH observation. Object keys with a double underscore denote a link to other entries in the database. For example “person__email” is a key for the person responsible, with more detailed contact information in a separate json document. The hashtag comments are not allowed when operating a json file, they are added here for explaining each json object.   {   \"campaign\": \"ai4sh_fi-jokioinen\", # 'project'+'_'+'country'+'-'+'pilot site'   \"sampling_log\": { # Assembly of samples collected in the same day at a particular pilot site     \"name\": \"fi-jokioinen-20241008\", # 'country'+'-'+'pilot site'+'-'+'sampling date'     \"date_stamp\": \"20241008\", # Date when the sample was collected in the field     \"person__email\": \"thomasg@xspectre.com\" # Email of person responsible for the sampling   },   \"sample\": \"fi-jokioinen-20241008_7-r_0-20\", # 'country'+'-'+'pilot site'+'-'+'sampling date'+'_'+'sample point id'+'_'+'min depth''+'-'+'max depth'   \"locus\": { # Assembly of the point location, setting and depth     \"position__name\": \"fi-jokioinen_7-r\", # 'country'+'-'+'pilot site'+'-'+'sample point id' used for checking internal consistency     \"point\": \"7-r\", # sample point id     \"setting\": \"uniform\", # The spatial setting in the landscape (uniform, alley, plantline, edge)     \"latitude\": 60.817951078, # Latitude in WGS84     \"longitude\": 23.493872667, # Longitude in WGS84     \"min_depth\": 0, # Sample upper limit in cm in soil profile     \"max_depth\": 20 # Sample lower limit in cm in soil profile   },   \"observation\": { # Assembly of metadata and data related to the observation     \"sample_preparation__name\": \"soil-undisturbed-in-situ\", # Explanatory name of soil treatment before observation     \"person__email\": \"thomasg@xspectre.com\", # Email of person responsible for the observation     \"subsample\": \"a\", # For registering if physically different soil subsamples were analysed separately, default for a single analysis = a     \"replicate\": 0, # If the analysis was replicated on the same subsample more than once, the first test is always set to 0     \"n_repeats\": 6, # The number of repetitions done by any instrument on a single subsample and as part of a single replicate, with results reported as mean or mean and standard deviation     \"date_stamp\": \"20241011\", # Date when the analysis was done     \"logistic\": { # Assembly of logistical metadata on the handling of the sample       \"sample_preservation__name\": null, # Explanatory name of soil preservation       \"sample_transport__name\": null, # Explanatory name of soil transportation if sensitive for analysis results       \"transport_duration_h\": 0, # duration of any sensitive transportation in hours       \"sample_storage__name\": null, # Explanatory name of soil storage if sensitive for analysis results       \"storage_duration_h\": 72,       \"person__email\": \"thomasg@xspectre.com\" # Email of person responsible for the logistics     },     \"analysis\": {  # Assembly of analysis results, procedures and instruments       \"xspectre-ise-ph-solid_ph(soil)\": {  # Soil property reported as AI4SH extended naming (part after underscore = core indicator)         \"value\": 6.810666666666665, # Recorded value (single value or mean of repeated observations)         \"standard_deviation\": 0.05562573345334154, # Recorded standard deviation of repeated measurements         \"unit__name\": \"pH\", # Recorded unit         \"indicator__name\": \"xspectre-ise-ph-solid_ph(soil)\", # AI4SH extended name of the soil indicator (part after underscore = core indicator)         \"procedure\": \"xspectre-ise-ph-solid\", # Procedure for the analysis         \"analysis_method__name\": \"ise-ph-soil\", # Name of the method         \"instrument_brand__name\": \"soil-ise-ph\", # Instrument type or brand         \"instrument_model__name\": \"ise-ph-soil\", # Model or version of the instrument         \"instrument_id\": \"0\"  # Any individual instrument id, the default for unknown id is \"0\"       }     }   } }   Nested json object   The nested json object pH ISE observations contains the same object keys and values as the flat version, but hierarchically organised with nested arrays. For explanations of the object keys please see the flat json object above.   {   \"data_source\": [     {       \"name\": \"ai4sh_fi-jokioinen\",       \"site\": [         {           \"name\": \"fi-jokioinen\",           \"point\": [             {               \"name\": \"7-r\",               \"latitude\": 60.817951078,               \"longitude\": 23.493872667,               \"setting\": \"uniform\",               \"sampling_log\": [                 {                   \"name\": \"fi-jokioinen-20241008\",                   \"date_stamp\": \"20241008\",                   \"person__email\": \"thomasg@xspectre.com\",                   \"sample\": [                     {                       \"name\": \"fi-jokioinen-20241008_7-r_0-20\",                       \"min_depth\": 0,                       \"max_depth\": 20,                       \"observation\": [                         {                           \"sample_preparation__name\": \"soil-undisturbed-in-situ\",                           \"person__email\": \"thomasg@xspectre.com\",                           \"subsample\": \"a\",                           \"replicate\": 0,                           \"n_repeats\": 6,                           \"date_stamp\": \"20241011\",                           \"logistic\": {                             \"sample_preservation__name\": null,                             \"sample_transport__name\": null,                             \"transport_duration_h\": 0,                             \"sample_storage__name\": null,                             \"storage_duration_h\": 72,                             \"person__email\": \"thomasg@xspectre.com\"                           },                           \"analysis_method\": {                             \"xspectre-ise-ph-solid\": [                               {                                 \"value\": 6.810666666666665,                                 \"standard_deviation\": 0.05562573345334154,                                 \"unit__name\": \"pH\",                                 \"indicator__name\": \"xspectre-ise-ph-solid_ph(soil)\",                                 \"analysis_method__name\": \"ise-ph-soil\",                                 \"instrument_brand__name\": \"soil-ise-ph\",                                 \"instrument_model__name\": \"ise-ph-soil\",                                 \"instrument_id\": \"0\"                               }                             ]                           }                         }                       ]                     }                   ]                 }               ]             }           ]         }       ]     }   ] }   Data access and file naming   The data is provided under the following licenses:      Data License: Creative Commons Attribution license (CC-BY)   Code License: Massachusetts Institute of Technology License (MIT License)   Recorded data from the pH ISE analysis are available at:      Flat json format https://github.com/AI4SH/in-situ_data/tree/main/flat/ise-ph   Nested json format: https://github.com/AI4SH/in-situ_data/tree/main/nested/ise-ph   The file naming is defined to separate all eventual renewed sampling or analysis efforts and is built up from the following metadata:     country   pilot site   sampling date   point id   min depth   max depth   subsample   replicate   setting   sample_preservation   procedure   instrument   analysis date   For the examples above, the file name thus have the following component values:      fi   jokioinen   20241008   7-r   0   20   a   0   uniform   no-prep   ise-ph-soil   0   20241011   And the final file name becomes:   fi-jokioinen-20241008_7-r_0-20_a_0_uniform_no-prep_ise-ph-soil_0_20241011.json   Acknowledgments and Funding   This work is part of the AI4SoilHealth project, funded by the European Union’s Horizon Europe Research and Innovation Programme under Grant Agreement No. 101086179.   Funded by the European Union. The views expressed are those of the authors and do not necessarily reflect those of the European Union or the European Research Executive Agency.   ","categories": [],
        "tags": [],
        "url": "/in-situ_data_docs/docs/in-situ_methods/ise/",
        "teaser": null
      },{
        "title": "Microbiometer",
        "excerpt":"Microbiometer is a commercial kit for quick estimation of soil microbial carbon biomass and the ratio between bacteria and fungi.   Short method description   The commercial Microbiometer test mixes a small (1 ml) soil sample with a reagent solution whereafter the mixture is rested for 20 minutes while the reaction is completed and the solution settles. Three drops of the supernatant are extracted using a provided pipette and applied to a likewise provided test card. The test card is then scanned with a smartphone using the Microbiometer app. If the colour change of the drop area on the test card is too weak, additional three drops are added and the test card re-scanned. From the colour intensity of the drop area, the app reports the soil’s microbial biomass carbon content and fungal-to-bacterial ratio (table 1). The test takes approximately 30 minutes in total. The app and the test card were updated during 2025 and thus the AI4SH reported results relate to both the classic and pro versions of the Microbiometer.   Table 1. Microbiometer recorded observations with each observation.                  Property       Indicator       AI4SH extended naming       Unit                       Microbial biomass       microbial-C       microbiometer_Microbial-C       ug*g^-1                 Fungi fraction       fungi-fraction       microbiometer_fungi-fraction       %                 Baceteria fraction       bacteria-fraction       microbiometer_bacteria-fraction       %           Data format   The AI4SH observed microbiometer data are available in 2 differently formatted json objects, a flat version more suitable for direct accessibility and a nested version intended for users who wants to convert the data to a relational sql database. The flat version does not include any arrays whereas the nested version is structured with nested arrays.   Flat json object   Below is an example of a flat json object for Microbiometer observations. Object keys with a double underscore denote a link to other entries in the database. For example “person__email” is a key for the person responsible, with more detailed contact information in a separate json document. The hashtag comments are not allowed when operating a json file, they are added here for explaining each json object.   {   \"data_source\": \"ai4sh_se-loennstorp\", # 'project'+'_'+'country'+'-'+'pilot site'   \"sampling_log\": { # Assembly of samples collected in the same day at a particular pilot site       \"name\": \"se-loennstorp-20240815\", # 'country'+'-'+'pilot site'+'-'+'sampling date'     \"date_stamp\": \"20240815\", # Date when the sample was collected in the field     \"person__email\": \"sampling@gmail.com\" # Email of person responsible for the sampling   },   \"sample\": \"se-loennstorp-20240815_55-c_0-20\", # 'country'+'-'+'pilot site'+'-'+'sampling date'+'_'+'sample point id'+'_'+'min depth''+'-'+'max depth'   \"locus\": { # Assembly of the point location, setting and depth     \"position__name\": \"se-loennstorp_55-c\", # 'country'+'-'+'pilot site'+'-'+'sample point id' used for checking internal consistency     \"point\": \"55-c\", # sample point id     \"setting\": \"uniform\", # The spatial setting in the landscape (uniform, alley, plantline, edge)     \"latitude\": 55.6671111111111, # Latitude in WGS84     \"longitude\": 13.1156944444444, # Longitude in WGS84     \"min_depth\": 0, # Sample upper limit in cm in soil profile     \"max_depth\": 20 # Sample lower limit in cm in soil profile   },   \"observation\": { # Assembly of metadata and data related to the observation     \"sample_preparation__name\": \"mixed-untreated-soil-in-lab\", # Explanatory name of soil treatment before observation     \"person__email\": \"analysis@gmail.com\", # Email of person responsible for the observation     \"subsample\": \"a\", # For registering if physically different soil subsamples were analysed separately, default for a single analysis = a     \"replicate\": 0, # If the analysis was replicated on the same subsample more than once, the first test is always set to 0     \"n_repeats\": 1, # The number of repetitions done by any instrument on a single subsample and as part of a single replicate, with results reported as mean or mean and standard deviation     \"date_stamp\": \"20240815\", # Date when the analysis was done     \"logistic\": { # Assembly of logistical metadata on the handling of the sample       \"sample_preservation__name\": null, # Explanatory name of soil preservation       \"sample_transport__name\": null, # Explanatory name of soil transportation if sensitive for analysis results       \"transport_duration_h\": 0, # duration of any sensitive transportation in hours       \"sample_storage__name\": null, # Explanatory name of soil storage if sensitive for analysis results       \"storage_duration_h\": 0, # Storage duration in hours       \"person__email\": \"logistic@gmail.com\" # Email of person responsible for the logistics     },     \"analysis\": { # Assembly of analysis results, procedures and instruments       \"microbiometer_Microbial-C\": { # Soil property reported as AI4SH extended naming (part after underscore = core indicator)         \"value\": 385.0, # Recorded value (single value or mean of repeated observations)         \"unit__name\": \"ug*g^-1\", # Recorded unit         \"indicator__name\": \"microbiometer_Microbial-C\", # AI4SH extended name of the soil indicator (part after underscore = core indicator)         \"procedure\": \"microbiometer\", # Procedure for the analysis (not always similar to method)         \"analysis_method__name\": \"microbiometer\", # Name of the method         \"instrument_brand__name\": \"microbiometer\", # Instrument type or brand         \"instrument_model__name\": \"classic\", # Model or version of the instrument         \"instrument_id\": \"0\" # Any individual instrument id, the default for unknown id is \"0\"       },       \"microbiometer_fungi-fraction\": {         \"value\": 42.0,         \"unit__name\": \"%\",         \"indicator__name\": \"microbiometer_fungi-fraction\",         \"procedure\": \"microbiometer\",         \"analysis_method__name\": \"microbiometer\",         \"instrument_brand__name\": \"microbiometer\",         \"instrument_model__name\": \"classic\",         \"instrument_id\": \"0\"       },       \"microbiometer_bacteria-fraction\": {         \"value\": 58.0,         \"unit__name\": \"%\",         \"indicator__name\": \"microbiometer_bacteria-fraction\",         \"procedure\": \"microbiometer\",         \"analysis_method__name\": \"microbiometer\",         \"instrument_brand__name\": \"microbiometer\",         \"instrument_model__name\": \"classic\",         \"instrument_id\": \"0\"       }     }   } }   Nested json object   The nested json object for Microbiometer observations contains the same object keys and values as the flat version, but hierarchically organised with nested arrays. For explanations of the object keys please see the flat json object above.   {   \"data_source\": [     {       \"name\": \"ai4sh_se-loennstorp\",       \"site\": [         {           \"name\": \"se-loennstorp\",           \"point\": [             {               \"name\": \"55-c\",               \"latitude\": 55.6671111111111,               \"longitude\": 13.1156944444444,               \"setting\": \"uniform\",               \"sampling_log\": [                 {                   \"name\": \"se-loennstorp-20240815\",                   \"date_stamp\": \"20240815\",                   \"person__email\": \"sampling@gmail.com\",                   \"sample\": [                     {                       \"name\": \"se-loennstorp-20240815_55-c_0-20\",                       \"min_depth\": 0,                       \"max_depth\": 20,                       \"observation\": [                         {                           \"sample_preparation__name\": \"mixed-untreated-soil-in-lab\",                           \"person__email\": \"analysis@gmail.com\",                           \"subsample\": \"a\",                           \"replicate\": 0,                           \"n_repeats\": 1,                           \"date_stamp\": \"20240815\",                           \"logistic\": {                             \"sample_preservation__name\": null,                             \"sample_transport__name\": null,                             \"transport_duration_h\": 0,                             \"sample_storage__name\": null,                             \"storage_duration_h\": 0,                             \"person__email\": \"logistic@gmail.com\"                           },                           \"analysis_method\": {                             \"microbiometer\": [                               {                                 \"microbiometer\": [                                   {                                     \"value\": 385.0,                                     \"unit__name\": \"ug*g^-1\",                                     \"indicator__name\": \"tot-c\",                                     \"analysis_method__name\": \"microbiometer-tot-c\",                                     \"instrument-brand__name\": \"microbiometer-tot-c\",                                     \"instrument-model__name\": \"classic\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 42.0,                                     \"unit__name\": \"%\",                                     \"indicator__name\": \"fungi\",                                     \"procedure\": \"microbiometer-fungi\",                                     \"analysis_method__name\": \"microbiometer-fungi\",                                     \"instrument-brand__name\": \"microbiometer-fungi\",                                     \"instrument-model__name\": \"classic\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 58.0,                                     \"unit__name\": \"%\",                                     \"indicator__name\": \"bacteria\",                                     \"procedure\": \"microbiometer-bacteria\",                                     \"analysis_method__name\": \"microbiometer-bacteria\",                                     \"instrument-brand__name\": \"microbiometer-bacteria\",                                     \"instrument-model__name\": \"classic\",                                     \"instrument_id\": \"0\"                                   }                                 ]                               }                             ]                           }                         }                       ]                     }                   ]                 }               ]             }           ]         }       ]     }   ] }   Data access and file naming   The data is provided under the following licenses:      Data License: Creative Commons Attribution license (CC-BY)   Code License: Massachusetts Institute of Technology License (MIT License)   Recorded data from the Microbiometer analysis are available at:      Flat json format https://github.com/AI4SH/in-situ_data/tree/main/flat/microbiometer   Nested json format: https://github.com/AI4SH/in-situ_data/tree/main/nested/microbiometer   The file naming is defined to separate all eventual renewed sampling or analysis efforts and is built up from the following metadata:     country   pilot site   sampling date   point id   min depth   max depth   subsample   replicate   setting   sample_preservation   procedure   instrument   analysis date   For the examples above, the file name thus have the following component values:      se   loennstorp   20240815   55-c   0   20   a   0   uniform   mix-wet   classic   0   20240815   And the final file name becomes:   se-loennstorp-20240815_55-c_0-20_a_0_uniform_mix-wet_classic_0_20240815.json   Acknowledgments and Funding   This work is part of the AI4SoilHealth project, funded by the European Union’s Horizon Europe Research and Innovation Programme under Grant Agreement No. 101086179.   Funded by the European Union. The views expressed are those of the authors and do not necessarily reflect those of the European Union or the European Research Executive Agency.   ","categories": [],
        "tags": [],
        "url": "/in-situ_data_docs/docs/in-situ_methods/microbiometer/",
        "teaser": null
      },{
        "title": "Penetrometer",
        "excerpt":"Physically robust and microchip controlled pentrometers for analysing physicochemical soil properties directly in-situ has become available over the past few years. Using multiple steel pins, developments in microelectronics and signal interpretation these simple instrumetns can now separate water and salt content and also detect specific ions, like hydrogen (pH) and those derived from nitrogen, phosphorus and potassium (NPK) – the key soil macronutrients. Soil penetrometers that can operate at the power levels of mobile phones have emerged only recently and are as yet scientifically unproven. The AI4SH project tested the 5-pin penetrometer model NPKPHCTH-S from ComWinTop store on AliExpress operating at 5 volt and mounted with a microcontroller and usb/bluetooth connection, developed by the startup Xsepctre.   Note the following issues related to the penetrometer data when applying it for further analysis:     the penetrometer data is only available for a subset of pilot sites,   in most cases each sample was anaysed using more than 1 subsample (thus there are more than one record per sample), and   for the Finnish site (Jokionen, used as examples below), 3 different copies of the same penetrometer model where used to allow evaluating bias and consistency.   Difference between subsample, replicate and repetition   The difference between a subsample and a replicate is that if the same physical volume of soil is used for the analysis it is a replicate, but if the analysis is applied to a separate physical volume it is a subsample. Thus all destructive analysis methods can per definition not be replicates. For the soil penetrometers to apply a replicate would mean letting it stay in the exact same postion for two sequantial observaion. As each penetrometer observation is already based on 6 repetion and the recorded data is stated as mean value and standard deviation (see example below), one more observation in the exact same position renders no new information. Thus, for the penetrometer analysis done as part of AI4SH, the position was typically shifted to different sides in the dug central pit, and each observation recorded as a subsample.   Short method description   The penetrometer is simply pushed into the soil, either from the top or vertically in a pit. The observation is started from the device (computer or smartphone) connected to the microcontroller unit. In the setup used in AI4SH, each observation is repeated 6 times and the user must approve of the results displayed as mean and standard deviation before the observation is recorded. The penetrometer used in the AI4SH project simultaneously records 9 soil parameters with each observation, table 1.   Table 1. Soil properties recorded with each observation with the ComWinTop NPKPHCTH-S 5 steel pin penetrometer.                  Property       Indicator       AI4SH extended naming       Unit                       temperature       temperature       xspectre-penetrometer_temperature       C                 Soil moisture       soil-moisture-volumetric-content       xspectre-penetrometer_soil-moisture-volumetric-content       vol*vol^-1                 pH       pH(soil)       xspectre-penetrometer_ph(soil)       pH                 Nitrogen       N       xspectre-penetrometer_nitrogen       mg*l-1                 Phosphorus       P       xspectre-penetrometer_phosphorus       mg*l-1                 Potassium       K       xspectre-penetrometer_potassium       mg*l-1                 Electric conductivity       electrical-conductivity       xspectre-penetrometer_electrical-conductivity       us*cm-1                 Salinity       salinity       xspectre-penetrometer_salinity       mg*l-1                 Total Dissolved Solid       total-dissolved-solids       xspectre-penetrometer_total-dissolved-solids       mg*l-1           Data format   Each measurement with the NPKPHCTH-S penetrometer is based on 6 repetitions and the recorded data contain the mean value and the standard deviation of these 6 repetitions. The observed penetrometer data are available in 2 differently formatted json objects, a flat version more suitable for direct accessibility and a nested version intended for users who wants to convert the data to a relational sql database. The flat version does not include any arrays whereas the nested version is structured with nested arrays.   Flat json object   Below is an example of a flat json object for penetrometer observations. Object keys with a double underscore denote a link to other entries in the database. For example “person__email” is a key for the person responsible, with more detailed contact information in a separate json document. The hashtag comments are not allowed when operating a json file, they are added here for explaining each json object.   {   \"campaign\": \"ai4sh_fi-jokioinen\", # 'project'+'_'+'country'+'-'+'pilot site'   \"sampling_log\": { # Assembly of samples collected in the same day at a particular pilot site     \"name\": \"fi-jokioinen-20241008\", # 'country'+'-'+'pilot site'+'-'+'sampling date'     \"date_stamp\": \"20241008\", # Date when the sample was collected in the field     \"person__email\": \"thomasg@xspectre.com\" # Email of person responsible for the sampling   },   \"sample\": \"fi-jokioinen-20241008_22-r_0-20\", # 'country'+'-'+'pilot site'+'-'+'sampling date'+'_'+'sample   \"locus\": { # Assembly of the point location, setting and depth     \"position__name\": \"fi-jokioinen_22-r\",  # 'country'+'-'+'pilot site'+'-'+'sample point id' used for checking internal consistency     \"point\": \"22-r\", # sample point id     \"setting\": \"uniform\", # The spatial setting in the landscape (uniform, alley, plantline, edge)     \"latitude\": 60.817773051, # Latitude in WGS84     \"longitude\": 23.479190111, # Longitude in WGS84     \"min_depth\": 0, # Sample upper limit in cm in soil profile     \"max_depth\": 20 # Sample lower limit in cm in soil profile   },   \"observation\": { # Assembly of metadata and data related to the observation     \"sample_preparation__name\": \"mixed-untreated-soil-in-lab\", # Explanatory name of soil treatment before observation     \"person__email\": \"thomasg@xspectre.com\", # Email of person responsible for the observation     \"subsample\": \"b\", # For registering if physically different soil subsamples were analysed separately, default for a single analysis = a     \"replicate\": 0, # If the analysis was replicated on the same subsample more than once, the first test is always set to 0     \"n_repeats\": 6, # The number of repetitions done by any instrument on a single subsample and as part of a single replicate, with results reported as mean or mean and standard deviation     \"date_stamp\": \"20241008\", # Date when the analysis was done     \"logistic\": { # Assembly of logistical metadata on the handling of the sample       \"sample_preservation__name\": null, # Explanatory name of soil preservation       \"sample_transport__name\": null, # Explanatory name of soil transportation if sensitive for analysis results       \"transport_duration_h\": 0, # duration of any sensitive transportation in hours       \"sample_storage__name\": null, # Explanatory name of soil storage if sensitive for analysis results       \"storage_duration_h\": 0, # Storage duration in hours       \"person__email\": \"thomasg@gxspectre.com\" # Email of person responsible for the logistics     },     \"analysis\": { # Assembly of analysis results, procedures and instruments       \"temperature\": { # Soil property reported from the analysis         \"value\": 16.45, # The recorded value (single value or mean or repeated observations)         \"standard_deviation\": 0.05000000000000071, # The recorded satandard deviation, if applicable         \"unit__name\": \"C\", # the recorded unit         \"indicator__name\": \"temperature\", # The name of the soil indicator         \"procedure\": \"xspectre-penetrometer\", # The procedure used for the analysis         \"analysis_method__name\": \"penetrometer\", # The instrument type or brand used for the analysis         \"instrument-brand__name\": \"comwintop\", # # Name of the method used for the analysis         \"instrument-model__name\": \"npkphcth-s\", # The instrument type or brand used for the analysis         \"instrument_id\": \"0003\" # Any individual instrument id, the default for unknown id is \"0\"       },       \"sm\": { # Each soil property reported with the same instrument are reported as individual objects         \"value\": 22.183333333333334,         \"standard_deviation\": 0.24094720491334964,         \"unit__name\": \"vol*vol-1\",         \"indicator__name\": \"sm\",         \"procedure\": \"xspectre-penetrometer\",         \"analysis_method__name\": \"penetrometer\",         \"instrument-brand__name\": \"comwintop\",         \"instrument-model__name\": \"npkphcth-s\",         \"instrument_id\": \"0003\"       },       \"ec\": { # Each soil property reported with the same instrument are reported as individual objects         \"value\": 28.0,         \"standard_deviation\": 0.0,         \"unit__name\": \"us*cm-1\",         \"indicator__name\": \"ec\",         \"procedure\": \"xspectre-penetrometer\",         \"analysis_method__name\": \"penetrometer\",         \"instrument-brand__name\": \"comwintop\",         \"instrument-model__name\": \"npkphcth-s\",         \"instrument_id\": \"0003\"       },       \"ph\": { # Each soil property reported with the same instrument are reported as individual objects         \"value\": 6.366666666666667,         \"standard_deviation\": 0.18856180831641262,         \"unit__name\": \"pH\",         \"indicator__name\": \"ph\",         \"procedure\": \"xspectre-penetrometer\",         \"analysis_method__name\": \"penetrometer\",         \"instrument-brand__name\": \"comwintop\",         \"instrument-model__name\": \"npkphcth-s\",         \"instrument_id\": \"0003\"       },       \"nitrogen\": { # Each soil property reported with the same instrument are reported as individual objects         \"value\": 88.0,         \"standard_deviation\": 0.0,         \"unit__name\": \"mg*l-1\",         \"indicator__name\": \"nitrogen\",         \"procedure\": \"xspectre-penetrometer\",         \"analysis_method__name\": \"penetrometer\",         \"instrument-brand__name\": \"comwintop\",         \"instrument-model__name\": \"npkphcth-s\",         \"instrument_id\": \"0003\"       },       \"phosphorus\": { # Each soil property reported with the same instrument are reported as individual objects         \"value\": 21.0,         \"standard_deviation\": 0.0,         \"unit__name\": \"mg*l-1\",         \"indicator__name\": \"phosphorus\",         \"procedure\": \"xspectre-penetrometer\",         \"analysis_method__name\": \"penetrometer\",         \"instrument-brand__name\": \"comwintop\",         \"instrument-model__name\": \"npkphcth-s\",         \"instrument_id\": \"0003\"       },       \"potassium\": { # Each soil property reported with the same instrument are reported as individual objects         \"value\": 96.0,         \"standard_deviation\": 0.0,         \"unit__name\": \"mg*l-1\",         \"indicator__name\": \"potassium\",         \"procedure\": \"xspectre-penetrometer\",         \"analysis_method__name\": \"penetrometer\",         \"instrument-brand__name\": \"comwintop\",         \"instrument-model__name\": \"npkphcth-s\",         \"instrument_id\": \"0003\"       },       \"salinity\": { # Each soil property reported with the same instrument are reported as individual objects         \"value\": 158.0,         \"standard_deviation\": 0.0,         \"unit__name\": \"mg*l-1\",         \"indicator__name\": \"salinity\",         \"procedure\": \"xspectre-penetrometer\",         \"analysis_method__name\": \"penetrometer\",         \"instrument-brand__name\": \"comwintop\",         \"instrument-model__name\": \"npkphcth-s\",         \"instrument_id\": \"0003\"       },       \"tds\": { # Each soil property reported with the same instrument are reported as individual objects         \"value\": 144.0,         \"standard_deviation\": 0.0,         \"unit__name\": \"mg*l-1\",         \"indicator__name\": \"tds\",         \"procedure\": \"xspectre-penetrometer\",         \"analysis_method__name\": \"penetrometer\",         \"instrument-brand__name\": \"comwintop\",         \"instrument-model__name\": \"npkphcth-s\",         \"instrument_id\": \"0003\"       }     }   } }   Nested json object   The nested json object for penetrometer observations contains the same object keys and values as the flat version, but hierarchically organised with nested arrays. For explanations of the object keys please see the flat json object above.   {   \"data_source\": [     {       \"name\": \"ai4sh_fi-jokioinen\",       \"site\": [         {           \"name\": \"fi-jokioinen\",           \"point\": [             {               \"name\": \"22-r\",               \"latitude\": 60.817773051,               \"longitude\": 23.479190111,               \"setting\": \"uniform\",               \"sampling_log\": [                 {                   \"name\": \"fi-jokioinen-20241008\",                   \"date_stamp\": \"20241008\",                   \"person__email\": \"thomasg@xspectre.com\",                   \"sample\": [                     {                       \"name\": \"fi-jokioinen-20241008_22-r_0-20\",                       \"min_depth\": 0,                       \"max_depth\": 20,                       \"observation\": [                         {                           \"sample_preparation__name\": \"mixed-untreated-soil-in-lab\",                           \"person__email\": \"thomasg@xspectre.com\",                           \"subsample\": \"b\",                           \"replicate\": 0,                           \"n_repeats\": 6,                           \"date_stamp\": \"20241008\",                           \"logistic\": {                             \"sample_preservation__name\": null,                             \"sample_transport__name\": null,                             \"transport_duration_h\": 0,                             \"sample_storage__name\": null,                             \"storage_duration_h\": 0,                             \"person__email\": \"thomasg@xspectre.com\"                           },                           \"analysis_method\": {                             \"xspectre-penetrometer\": [                               {                                 \"value\": 16.45,                                 \"standard_deviation\": 0.05000000000000071,                                 \"unit__name\": \"C\",                                 \"indicator__name\": \"temperature\",                                 \"analysis_method__name\": \"penetrometer\",                                 \"instrument-brand__name\": \"comwintop\",                                 \"instrument-model__name\": \"npkphcth-s\",                                 \"instrument_id\": \"0003\"                               },                               {                                 \"value\": 22.183333333333334,                                 \"standard_deviation\": 0.24094720491334964,                                 \"unit__name\": \"vol*vol-1\",                                 \"indicator__name\": \"sm\",                                 \"analysis_method__name\": \"penetrometer\",                                 \"instrument-brand__name\": \"comwintop\",                                 \"instrument-model__name\": \"npkphcth-s\",                                 \"instrument_id\": \"0003\"                               },                               {                                 \"value\": 28.0,                                 \"standard_deviation\": 0.0,                                 \"unit__name\": \"us*cm-1\",                                 \"indicator__name\": \"ec\",                                 \"analysis_method__name\": \"penetrometer\",                                 \"instrument-brand__name\": \"comwintop\",                                 \"instrument-model__name\": \"npkphcth-s\",                                 \"instrument_id\": \"0003\"                               },                               {                                 \"value\": 6.366666666666667,                                 \"standard_deviation\": 0.18856180831641262,                                 \"unit__name\": \"pH\",                                 \"indicator__name\": \"ph\",                                 \"analysis_method__name\": \"penetrometer\",                                 \"instrument-brand__name\": \"comwintop\",                                 \"instrument-model__name\": \"npkphcth-s\",                                 \"instrument_id\": \"0003\"                               },                               {                                 \"value\": 88.0,                                 \"standard_deviation\": 0.0,                                 \"unit__name\": \"mg*l-1\",                                 \"indicator__name\": \"nitrogen\",                                 \"analysis_method__name\": \"penetrometer\",                                 \"instrument-brand__name\": \"comwintop\",                                 \"instrument-model__name\": \"npkphcth-s\",                                 \"instrument_id\": \"0003\"                               },                               {                                 \"value\": 21.0,                                 \"standard_deviation\": 0.0,                                 \"unit__name\": \"mg*l-1\",                                 \"indicator__name\": \"phosphorus\",                                 \"analysis_method__name\": \"penetrometer\",                                 \"instrument-brand__name\": \"comwintop\",                                 \"instrument-model__name\": \"npkphcth-s\",                                 \"instrument_id\": \"0003\"                               },                               {                                 \"value\": 96.0,                                 \"standard_deviation\": 0.0,                                 \"unit__name\": \"mg*l-1\",                                 \"indicator__name\": \"potassium\",                                 \"analysis_method__name\": \"penetrometer\",                                 \"instrument-brand__name\": \"comwintop\",                                 \"instrument-model__name\": \"npkphcth-s\",                                 \"instrument_id\": \"0003\"                               },                               {                                 \"value\": 158.0,                                 \"standard_deviation\": 0.0,                                 \"unit__name\": \"mg*l-1\",                                 \"indicator__name\": \"salinity\",                                 \"analysis_method__name\": \"penetrometer\",                                 \"instrument-brand__name\": \"comwintop\",                                 \"instrument-model__name\": \"npkphcth-s\",                                 \"instrument_id\": \"0003\"                               },                               {                                 \"value\": 144.0,                                 \"standard_deviation\": 0.0,                                 \"unit__name\": \"mg*l-1\",                                 \"indicator__name\": \"tds\",                                 \"analysis_method__name\": \"penetrometer\",                                 \"instrument-brand__name\": \"comwintop\",                                 \"instrument-model__name\": \"npkphcth-s\",                                 \"instrument_id\": \"0003\"                               }                             ]                           }                         }                       ]                     }                   ]                 }               ]             }           ]         }       ]     }   ] }   Data access and file naming   The data is provided under the following licenses:      Data License: Creative Commons Attribution license (CC-BY)   Code License: Massachusetts Institute of Technology License (MIT License)   recorded data from the Microbiometer analysis are available at      Flat json format https://github.com/AI4SH/in-situ_data/tree/main/flat/penetrometer   Nested json format: https://github.com/AI4SH/in-situ_data/tree/main/flat/penetrometer   The file naming is defined to separate all eventual renewed sampling or analysis efforts and is built up from the following metadata:     country   pilot site   sampling date   point id   min depth   max depth   subsample   replicate   setting   sample_preservation   procedure   instrument   analysis date   For the examples above, the file name thus have the following component values:      fi   jokioinen   20241008   22-r   0   20   b   0   uniform   mix-wet   npkphcth-s   0003   20241008   And the final file name becomes:   fi-jokioinen-20241008_22-r_0-20_b_0_uniform_mix-wet_npkphcth-s_0003_20241008   Acknowledgments and Funding   This work is part of the AI4SoilHealth project, funded by the European Union’s Horizon Europe Research and Innovation Programme under Grant Agreement No. 101086179.   Funded by the European Union. The views expressed are those of the authors and do not necessarily reflect those of the European Union or the European Research Executive Agency.   As noted above, the the peneteromer observations in Jokioinen, Finlad, three copies of the penterometer model (with id’s 0001, 0002 and 0003) where used and each penetrometer where tested on several subsampels (denoted a, b, c, etc).   ","categories": [],
        "tags": [],
        "url": "/in-situ_data_docs/docs/in-situ_methods/penetrometer/",
        "teaser": null
      },{
        "title": "Salinity",
        "excerpt":"Salinisation is recognised as one of the largest threats to soil health in Europe and elsewhere. A simple way to test soil salinity is to dissolve a small amount of a soil smaple is distilled water and estimate the salinity from the resistance between two (or more) electrodes.   Short method description   Dissolved salts decrease the resistance to electric currents, whether in (wet) soil or a solution. Measuring the resistance across two electrodes is a simple way to determine the resistance and by calibrating the observed resistance against standard solutions, the salinity of a smaple can be determined (table 1). In AI4SH a simple bi-pin electrode operated from a microcontroller built by the Swedish startup Xspectre is used as a citizen scientist approach for observing soil salnity. The procedure includes taking a small fraction of soil and dissolving it in distilled water at a ratio 1:5 and then observe the soil moisture. For calibration, two standard solutions, one with a high and one with low salainty are used. Note that the AI4SH project also measured salinity direcly in the field with the [penetrometer][penetrometer] method.   Table 1. Bi-pin electrode recorded total dissolved solids (TDS) in solutions of soil dissolved in distilled water at a ratio 1:5.                  Property       Indicator       AI4SH extended naming       Unit                       Total dissolved solids       total-dissolved-solids       xspectre-gx16-ec_total-dissolved-solids       ppm           Data format   Each observation with the bi-pin sensor is based on 6 replications and the recorded data report the mean value and the standard deviation. The observed data are available in 2 differently formatted json objects, a flat version more suitable for direct accessibility and a nested version intended for users who wants to convert the data to a relational sql database. The flat version does not include any arrays whereas the nested version is structured with nested arrays.   Flat json object   Below is an example of a flat json object for the bi-pin salinity electrode. Object keys with a double underscore denote a link to other entries in the database. For example “person__email” is a key for the person responsible, with more detailed contact information in a separate json document. The hashtag comments are not allowed when operating a json file, they are added here for explaining each json object.   {   \"campaign\": \"ai4sh_fi-jokioinen\", # 'project'+'_'+'country'+'-'+'pilot site'   \"sampling_log\": { # Assembly of samples collected in the same day at a particular pilot site     \"name\": \"fi-jokioinen-20241008\", # 'country'+'-'+'pilot site'+'-'+'sampling date'     \"date_stamp\": \"20241008\", # Date when the sample was collected in the field     \"person__email\": \"thomasg@xspectre.com\" # Email of person responsible for the sampling   },   \"sample\": \"fi-jokioinen-20241008_7-r_0-20\", # 'country'+'-'+'pilot site'+'-'+'sampling date'+'_'+'sample point id'+'_'+'min depth''+'-'+'max depth'   \"locus\": { # Assembly of the point location, setting and depth     \"position__name\": \"fi-jokioinen_7-r\", # 'country'+'-'+'pilot site'+'-'+'sample point id' used for checking internal consistency     \"point\": \"7-r\", # sample point id     \"setting\": \"uniform\", # The spatial setting in the landscape (uniform, alley, plantline, edge)     \"latitude\": 60.817951078, # Latitude in WGS84     \"longitude\": 23.493872667, # Longitude in WGS84     \"min_depth\": 0, # Sample upper limit in cm in soil profile     \"max_depth\": 20 # Sample lower limit in cm in soil profile   },   \"observation\": { # Assembly of metadata and data related to the observation     \"sample_preparation__name\": \"mixed-untreated-soil-in-lab\", # Explanatory name of soil treatment before observation     \"person__email\": \"thomasg@xspectre.com\", # Email of person responsible for the observation     \"subsample\": \"a\", # For registering if physically different soil subsamples were analysed separately, default for a single analysis = a     \"replicate\": 0, # If the analysis was replicated on the same subsample more than once, the first test is always set to 0     \"n_repeats\": 6, # The number of repetitions done by any instrument on a single subsample and as part of a single replicate, with results reported as mean or mean and standard deviation     \"date_stamp\": \"20241011\", # Date when the analysis was done     \"logistic\": { # Assembly of logistical metadata on the handling of the sample       \"sample_preservation__name\": null, # Explanatory name of soil preservation       \"sample_transport__name\": null, # Explanatory name of soil transportation if sensitive for analysis results       \"transport_duration_h\": 0, # duration of any sensitive transportation in hours       \"sample_storage__name\": null, # Explanatory name of soil storage if sensitive for analysis results       \"storage_duration_h\": 72,       \"person__email\": \"thomasg@xspectre.com\" # Email of person responsible for the logistics     },     \"analysis\": {  # Assembly of analysis results, procedures and instruments       \"xspectre-gx16-ec_total-dissolved-solids\": {  # Soil property reported as AI4SH extended naming (part after underscore = core indicator)         \"value\": 0.13142500000000001, # Recorded value (single value or mean of repeated observations)         \"standard_deviation\": 0.005514582939806052, # Recorded standard deviation of repeated measurements         \"unit__name\": \"ppm\", # Recorded unit         \"indicator__name\": \"xspectre-gx16-ec_total-dissolved-solids\", # AI4SH extended name of the soil indicator (part after underscore = core indicator)         \"procedure\": \"xspectre-gx16-ec\", # Procedure for the analysis         \"analysis_method__name\": \"tds-bipin-5xh2o\", # Name of the method         \"instrument_brand__name\": \"iduino\", # Instrument type or brand         \"instrument_model__name\": \"tc-9520260\", # Model or version of the instrument         \"instrument_id\": \"0\"  # Any individual instrument id, the default for unknown id is \"0\"       }     }   } }   Nested json object   The nested json object for bi-pin salinity electrode observations contains the same object keys and values as the flat version, but hierarchically organised with nested arrays. For explanations of the object keys please see the flat json object above.   {   \"data_source\": [     {       \"name\": \"ai4sh_fi-jokioinen\",       \"site\": [         {           \"name\": \"fi-jokioinen\",           \"point\": [             {               \"name\": \"7-r\",               \"latitude\": 60.817951078,               \"longitude\": 23.493872667,               \"setting\": \"uniform\",               \"sampling_log\": [                 {                   \"name\": \"fi-jokioinen-20241008\",                   \"date_stamp\": \"20241008\",                   \"person__email\": \"thomasg@xspectre.com\",                   \"sample\": [                     {                       \"name\": \"fi-jokioinen-20241008_7-r_0-20\",                       \"min_depth\": 0,                       \"max_depth\": 20,                       \"observation\": [                         {                           \"sample_preparation__name\": \"mixed-untreated-soil-in-lab\",                           \"person__email\": \"thomasg@xspectre.com\",                           \"subsample\": \"a\",                           \"replicate\": 0,                           \"n_repeats\": 6,                           \"date_stamp\": \"20241011\",                           \"logistic\": {                             \"sample_preservation__name\": null,                             \"sample_transport__name\": null,                             \"transport_duration_h\": 0,                             \"sample_storage__name\": null,                             \"storage_duration_h\": 72,                             \"person__email\": \"thomasg@xspectre.com\"                           },                           \"analysis_method\": {                             \"xspectre-gx16-ec\": [                               {                                 \"value\": 0.13142500000000001,                                 \"standard_deviation\": 0.005514582939806052,                                 \"unit__name\": \"ppm\",                                 \"indicator__name\": \"xspectre-gx16-ec_total-dissolved-solids\",                                 \"analysis_method__name\": \"tds-bipin-5xh2o\",                                 \"instrument_brand__name\": \"iduino\",                                 \"instrument_model__name\": \"tc-9520260\",                                 \"instrument_id\": \"0\"                               }                             ]                           }                         }                       ]                     }                   ]                 }               ]             }           ]         }       ]     }   ] }   Data access and file naming   The data is provided under the following licenses:      Data License: Creative Commons Attribution license (CC-BY)   Code License: Massachusetts Institute of Technology License (MIT License)   Recorded data from the bi-pin salinity electrode analysis are available at:      Flat json format https://github.com/AI4SH/in-situ_data/tree/main/flat/gx16-ec   Nested json format: https://github.com/AI4SH/in-situ_data/tree/main/nested/gx16-ec   The file naming is defined to separate all eventual renewed sampling or analysis efforts and is built up from the following metadata:     country   pilot site   sampling date   point id   min depth   max depth   subsample   replicate   setting   sample_preservation   procedure   instrument   analysis date   For the examples above, the file name thus have the following component values:      fi   jokioinen   20241008   7-r   0   20   a   0   uniform   mix-wet   tc-9520260   0   20241011   And the final file name becomes:   fi-jokioinen-20241008_7-r_0-20_a_0_uniform_mix-wet_tc-9520260_0_20241011.json  Acknowledgments and Funding   This work is part of the AI4SoilHealth project, funded by the European Union’s Horizon Europe Research and Innovation Programme under Grant Agreement No. 101086179.   Funded by the European Union. The views expressed are those of the authors and do not necessarily reflect those of the European Union or the European Research Executive Agency.   ","categories": [],
        "tags": [],
        "url": "/in-situ_data_docs/docs/in-situ_methods/salinity/",
        "teaser": null
      },{
        "title": "Spectroscopy",
        "excerpt":"Light is progressively used as an analytical tool for determining chemical and physical compositions in astronomy, industrial manufacturing, food and pharmaceuticals processing, and science - including for laboratory soil analysis. Recent developments in microelectronics, light sensing technology and artificial intelligence (AI), have led to both field spectrometers for in-situ soil analysis and to cheaper layperson spectrometers that are also possible to use for soil analysis. Soil spectroscopy is progressively becoming a mature method for determining soil properties.   Spectroscopy can not determine all properties that are traditionally analysed using wet laboratory methods, but many key attributes are already routinely determined by spectroscopy also in commercial soil laboratories.   Short method description   The AI4SH in-situ soil sampling and subsequent spectroscopic analysis looked at soil samples prepared in three different ways:      undisturbed soil directly in the field,   mixed and (naturally) wet soil samples, and   dried and sieved soil samples.   Existing soil spectral libraries, like the Open Soil Spectral Library (OSSL) are built from dried and sieved (2 mm) samples. The potential for determining the chemical and physical properties using soil spectroscopy is thus higher when using spectra from dried and sieved samples. Direct field scanning and spectral scanning of wet samples is more rapid and cost effective, but non-linearly affected by varying water contents, and were also tested as part of AI4SH.   Further, to compare the performance of high grade laboratory instruments with both commercial handheld soil spectrometers and cheap (layperson applicable) pocket-sized spectrometers, a suite of spectrometers is used. Most instruments where applied to all three preparation levels, but with restricted testing done with laboratory grade instruments directly in the field. The different instruments used in the project are listed in table 1.   Table 1. Spectrometers applied as part of the AI4SH in-situ soil sampling.                  spectrometer       spectral range (nm)       nr of bands       applied                       FOSS NIRS DS2500       400-2500       4200       ex-situ                 LabSpec ASD*       350–2500       2151       in-situ, ex-situ                 Neospectra/ProxiScout       1300-2550       257       in-situ, ex-situ                 Xspectre_c12880ma       340-780       288       in-situ, ex-situ                 Xspectre_c14384ma-01       650-1050       192       in-situ, ex-situ           *Data from the LabSpec instrument are under processing and not reported as of December 2025   Table 2 Spectral reflectance observation records of the 4 different spectrometers applied within AI4SH.                  Property       Indicator       AI4SH extended naming       Unit                       diffuse spectral reflectance       reflectance       foss-ds2500-scan_reflectance       reflectance                 diffuse spectral reflectance       reflectance       neospectra-scan_reflectance       reflectance                 diffuse spectral reflectance       reflectance       xspectre-c12880ma_reflectance       reflectance                 diffuse spectral reflectance       reflectance       xspectre-c14384ma-01_reflectance       reflectance           Data format   The spectral signals derived from the suite of spectrometer used in the project differs. But for the database all where transformed to reflectance (fraction of light of each wavelength that is reflected). Compared to all other methods applied, a spectral signal is not a single number but an array. In the database the recorded results (value, and for the Xspectre instruments also standard deviation) are thus reported as arrays for all spectral analysis. All other objects are the same as for other methods.   Flat json object   Below is an example of a flat json object for the spectral analysis derived from the NeoSpectra/ProxiScout spectrometer. Only a few actual reflectance values are included in the example. Object keys with a double underscore denote a link to other entries in the database. For example “person__email” is a key for the person responsible, with more detailed contact information in a separate json document. The hashtag comments are not allowed when operating a json file, they are added here for explaining each json object.   {   \"campaign\": \"ai4sh_fi-jokioinen\", # 'project'+'_'+'country'+'-'+'pilot site'   \"sampling_log\": { # Assembly of samples collected in the same day at a particular pilot site     \"name\": \"fi-jokioinen-20241008\", # 'country'+'-'+'pilot site'+'-'+'sampling date'     \"date_stamp\": \"20241008\", # Date when the sample was collected in the field     \"person__email\": \"thomasg@xspectre.com\" # Email of person responsible for the sampling   },   \"sample\": \"fi-jokioinen-20241008_7-r_0-20\", # 'country'+'-'+'pilot site'+'-'+'sampling date'+'_'+'sample point id'+'_'+'min depth''+'-'+'max depth'   \"locus\": { # Assembly of the point location, setting and depth     \"position__name\": \"fi-jokioinen_7-r\", # 'country'+'-'+'pilot site'+'-'+'sample point id' used for checking internal consistency     \"point\": \"7-r\", # sample point id     \"setting\": \"uniform\", # The spatial setting in the landscape (uniform, alley, plantline, edge)     \"latitude\": 60.817951078, # Latitude in WGS84     \"longitude\": 23.493872667, # Longitude in WGS84     \"min_depth\": 0, # Sample upper limit in cm in soil profile     \"max_depth\": 20 # Sample lower limit in cm in soil profile   },   \"observation\": { # Assembly of metadata and data related to the observation     \"sample_preparation__name\": \"dried-sieved-soil-in-lab\", # Explanatory name of soil treatment before observation     \"person__email\": \"thomasg@xspectre.com\", # Email of person responsible for the observation     \"subsample\": \"a\", # For registering if physically different soil subsamples were analysed separately, default for a single analysis = a     \"replicate\": 0, # If the analysis was replicated on the same subsample more than once, the first test is always set to 0     \"n_repeats\": 3, # The number of repetitions done by any instrument on a single subsample and as part of a single replicate, with results reported as mean or mean and standard deviation     \"date_stamp\": \"20241107\",     # Date when the analysis was done     \"logistic\": { # Assembly of logistical metadata on the handling of the sample       \"sample_preservation__name\": null, # Explanatory name of soil preservation       \"sample_transport__name\": null, # Explanatory name of soil transportation if sensitive for analysis results       \"transport_duration_h\": 0, # duration of any sensitive transportation in hours       \"sample_storage__name\": null, # Explanatory name of soil storage if sensitive for analysis results       \"storage_duration_h\": 720,       \"person__email\": \"thomasg@xspectre.com\" # Email of person responsible for the logistics     },     \"analysis\": {  # Assembly of analysis results, procedures and instruments       \"neospectra-scan_reflectance\": { # Soil property reported as AI4SH extended naming (part after underscore = core indicator)         \"value\": [  # Recorded value (array of single values or mean of repeated observations)           0.37557660975300666,           0.376968892702499,           ...           ...           0.40057577579749326,           0.40105743918323955         ],         \"unit__name\": \"reflectance\", # Recorded unit         \"indicator__name\": \"neospectra-scan_reflectance\", # AI4SH extended name of the soil indicator (part after underscore = core indicator)         \"procedure\": \"neospectra-scan\", # Procedure for the analysis         \"analysis_method__name\": \"diffuse-reflectance-spectroscopy\", # Name of the method         \"instrument_brand__name\": \"neospectra\", # Instrument type or brand         \"instrument_model__name\": \"proxiscout\", # Model or version of the instrument         \"instrument_id\": \"neoscanner_23040128\" # Any individual instrument id, the default for unknown id is \"0\"       }     }   } }   The nested json object for a sepctral analysis contains the same object keys and values as the flat version, but hierarchically organised with nested arrays. For explanations of the object keys please see the flat json object above.   Nested json object   {   \"data_source\": [     {       \"name\": \"ai4sh_fi-jokioinen\",       \"site\": [         {           \"name\": \"fi-jokioinen\",           \"point\": [             {               \"name\": \"7-r\",               \"latitude\": 60.817951078,               \"longitude\": 23.493872667,               \"setting\": \"uniform\",               \"sampling_log\": [                 {                   \"name\": \"fi-jokioinen-20241008\",                   \"date_stamp\": \"20241008\",                   \"person__email\": \"sampling@gmail.com\",                   \"sample\": [                     {                       \"name\": \"fi-jokioinen-20241008_7-r_0-20\",                       \"min_depth\": 0,                       \"max_depth\": 20,                       \"observation\": [                         {                           \"sample_preparation__name\": \"dried-sieved-soil-in-lab\",                           \"person__email\": \"analysis@gmail.com\",                           \"subsample\": \"a\",                           \"replicate\": 0,                           \"n_repeats\": 3,                           \"date_stamp\": \"20241107\",                           \"logistic\": {                             \"sample_preservation__name\": null,                             \"sample_transport__name\": null,                             \"transport_duration_h\": 0,                             \"sample_storage__name\": null,                             \"storage_duration_h\": 720,                             \"person__email\": \"logistic@gmail.com\"                           },                           \"analysis_method\": {                             \"neospectra-scan\": [                               {                                 \"value\": [                                   0.37557660975300666,                                   0.376968892702499,                                   ...                                   ...                                   0.40057577579749326,                                   0.40105743918323955                                 ],                                 \"unit__name\": \"reflectance\",                                 \"indicator__name\": \"neospectra-scan_reflectance\",                                 \"analysis_method__name\": \"diffuse-reflectance-spectroscopy\",                                 \"instrument_brand__name\": \"neospectra\",                                 \"instrument_model__name\": \"proxiscout\",                                 \"instrument_id\": \"neoscanner_23040128\"                               }                             ]                           }                         }                       ]                     }                   ]                 }               ]             }           ]         }       ]     }   ] }   Data access and file naming   The data is provided under the following licenses:      Data License: Creative Commons Attribution license (CC-BY)   Code License: Massachusetts Institute of Technology License (MIT License)   Recorded data from the spectrometer analysis are available divded in folders for each spectrometer brand.   FOSS NIRS DS2500      Flat json format https://github.com/AI4SH/in-situ_data/tree/main/flat/ds2500_spectra   Nested json format: https://github.com/AI4SH/in-situ_data/tree/main/nested/ds2500_spectra   Neospectra/ProxiScout      Flat json format https://github.com/AI4SH/in-situ_data/tree/main/flat/proxiscout_spectra   Nested json format: https://github.com/AI4SH/in-situ_data/tree/main/nested/proxiscout_spectra   Xspectre      Flat json format https://github.com/AI4SH/in-situ_data/tree/main/flat/xspectre_spectra   Nested json format: https://github.com/AI4SH/in-situ_data/tree/main/nested/xspectre_spectra   File naming   The file naming is defined to separate all eventual renewed sampling or analysis efforts and is built up from the following metadata:     country   pilot site   sampling date   point id   min depth   max depth   subsample   replicate   setting   sample_preservation   procedure   instrument   analysis date   For the examples above, the file name thus have the following component values:      fi   jokioinen   20241008   7-r   0   20   a   0   uniform   dried-sieved   proxiscout   23040128   20241107   And the final file name becomes:   fi-jokioinen-20241008_7-r_0-20_a_0_uniform_dried-sieved_proxiscout_neoscanner-23040128_20241107.json   Acknowledgments and Funding   This work is part of the AI4SoilHealth project, funded by the European Union’s Horizon Europe Research and Innovation Programme under Grant Agreement No. 101086179.   Funded by the European Union. The views expressed are those of the authors and do not necessarily reflect those of the European Union or the European Research Executive Agency.   ","categories": [],
        "tags": [],
        "url": "/in-situ_data_docs/docs/in-situ_methods/spectroscopy/",
        "teaser": null
      },{
        "title": "Wet laboratory",
        "excerpt":"Traditional chemicophysical soil properties have been determined by wet laboratory methods. While traditional methods are restricted to chemical and physical properties, disregarding biological compostions and precesse, they are still the benchmark against which novel physicochemical methods are tested and evaluated. They are also used for interpreting biological methods and their results, also in the light of the machnancial paradigm of agricultural practicies focusing on chemical physical management.   Method description and reference   For almost all collected sample within the AI4SH in-situ campaign, 15 soil parameters were analysed using traditional wet laboratory methods (table 1). The majority of the samples were analysed at German [Agrolab][agrolab]. The exception is the trial sampling at the Greek pilot site Ktima-Gerovassiliou.   Table 1.                  Property       Indicator       AI4SH extended naming       Unit                       pH       pH(water)       agrolab_ph(water)       pH                 Electrical conductivity       electrical-conductivity       agrolab_electrical-conductivity       us*cm^-1                 Total Organic Carbon       total-organic-carbon       agrolab_total-organic-carbon       percent                 Total Nitrogen       total-nitrogen       agrolab_total-nitrogen       percent                 Calcium       calcium       agrolab_calcium       cmol*kg^-1                 Magnesium       magnesium       agrolab_magnesium       cmol*kg^-1                 Potassium       potassium       agrolab_potassium       cmol*kg^-1                 Sodium       sodium       agrolab_sodium       cmol*kg^-1                 Cation exchange capacity       cation-exchange-capacity       agrolab_cation-exchange-capacity       cmol*kg^-1                 Olsen phosphorus       olsen-phosphorus       agrolab_olsen-phosphorus       mg*100g^-1                 clay(&lt;0.002mm)       clay(&lt;0.002mm)       agrolab-wet-clay(&lt;0.002mm)       g*100g^-1                 fine-silt(0.002-0.02mm)       fine-silt(0.002-0.02mm)       agrolab_fine-silt(0.002-0.02mm)       g*100g^-1                 coarse-silt(0.02-0.06mm)       coarse-silt(0.02-0.06mm)       agrolab_coarse-silt(0.02-0.06mm)       g*100g^-1                 fine-sand(0.06-0.2mm)       fine-sand(0.06-0.2mm)       agrolab_fine-sand(0.06-0.2mm)       g*100g^-1                 coarse-sand(0.2-2.0mm)       coarse-sand(0.2-2.0mm)       agrolab_coarse-sand(0.2-2.0mm)       g*100g^-1           Table 2. Detailed methods applied by Agrolab for analysing AI4SH soil sample chemicophysical properties.                  Property       abbreviation       method       ISO       USDA       FAO       url                       pH(water)       pH-h2o                                                         Electrical conductivity       EC                                                         Total Organic Carbon       TOC                                                         Total Nitrogen       tot-N                                                         Calcium       Ca2+                                                         Magnesium       Mg2+                                                         Potassium       K+                                                         Cation exchange capacity       CEC                                                         Olsen phosphorus       olsen-P                                                         clay(&lt;0.002mm)       clay                                                         fine-silt(0.002-0.02mm)       fine-silt                                                         coarse-silt(0.02-0.06mm)       coarse-silt                                                         fine-sand(0.06-0.2mm)       fine-sand                                                         coarse-sand(0.2-2.0mm)       coarse-sand                                                   Scientific reports   Data access   The data is provided under the following licenses:      Data License: Creative Commons Attribution license (CC-BY)   Code License: Massachusetts Institute of Technology License (MIT License)   Acknowledgments and Funding   This work is part of the AI4SoilHealth project, funded by the European Union’s Horizon Europe Research and Innovation Programme under Grant Agreement No. 101086179.   Funded by the European Union. The views expressed are those of the authors and do not necessarily reflect those of the European Union or the European Research Executive Agency.   Resources   Howebrew   RVM   Asdf   chruby   Frum   blog on installing Ruby on MacOS   ","categories": [],
        "tags": [],
        "url": "/in-situ_data_docs/docs/in-situ_methods/wetlab/",
        "teaser": null
      },{
        "title": "Earth Observation (EO) satellite data",
        "excerpt":"Earth Observation (EO) data is progressively becoming more important for understanding patterns and processes at the Earths’s surface. Properties related to soil health that can be derived from EO-data include landscape and landform scale patterns, vegetation cover, type and production and soil physicochemical properties.   As part of the AI4SoilHealth project, a pan-European Soil Health Data Cube with EO (satellite) data and derived static and dynamic properties related to the Earths’s terrestrial land surface has been developed under the leadership of OpenGeoHub. This page illustrates how static landscape scale properties derived from the Soil Health Data Cube can be added to the AI4SH in-situ database.   The satellite derived properties listed herein are retrieved from the Soil Health Data Cube using the Jupyter Notebook Extract_AI4SH_point_datacube, freely available on this site.   Short method description   The Soil Health Data Cube for pan-EU SHDC4EU is a compilation of pan-European spatial layers organized into a data cube and served through an API as analysis-ready and decision-ready data. OpenGeoHub, an independent not-for-profit research foundation and a partner of the AI4SH project, has developed this analysis-ready open environmental data cube for Europe.   It includes EU-wide Earth Observation (EO) products, terrain parameters, primary soil properties, land degradation indices, etc., at 30-m resolution covering the period 2000–2024+. It also provides climatic time-series datasets (such as rainfall, soil moisture, and surface temperature) and projected crop model outputs (typically at 1-km spatial resolution).   All soil health indicators collected at different pilot sites with their coordinates, can be combined with these SHDC layers to enable terrain and remote sensing analysis for furthur processes. Table 1 is an excerpt of available data layers. Single data points from these layers can be extracted with the AI4SH Jupyter notebook Extract_AI4SH_point_datacube. To combine the extracted data with the in-situ data is then done with the notebook AI4SH_csv_to_AI4SH_json.   Table 1. Static soil properties retrieved from the Soil Health Data Cube.                  Property       Indicator       AI4SH extended naming       Unit                       OpenLandMap Difference from mean elevation (GEDTM30)       dfme-edtm       eo-data_dfme-edtm       index                 OpenLandMap Geomorphons (GEDTM30)       geomorphons-edtm       eo-data_dfme-geomorphons       index                 OpenLandMap Hillshade (GEDTM30)       hillshade-edtm       eo-data_hillshade-edtm       index                 OpenLandMap LS factor (GEDTM30)       ls.factor-edtm_20060101-20151231       eo-data_ls.factor-edtm20060101-20151231       index                 OpenLandMap Maximal curvature (GEDTM30       maxic-edtm       eo-data_maxic-edtm       index                 OpenLandMap Minimal curvature (GEDTM30)       minic-edtm       eo-data_minic-edtm       index                 OpenLandMap Negative openness (GEDTM30)       neg.openness-edtm       eo-data_neg.openness-edtm       index                 OpenLandMap Positive openness (GEDTM30)       pos.openness-edtm       eo-data_pos.openness-edtm       index                 OpenLandMap Profile curvature (GEDTM30)       pro.curv-edtm       eo-data_pro.curv-edtm       index                 OpenLandMap Shape index (GEDTM30)       shpindx-edtm       eo-data_shpindx-edtm       index                 OpenLandMap Slope in degree (GEDTM30)       slope.in.degree-edtm       eo-data_slope.in.degree-edtm       degrees                 OpenLandMap Topographic Wetness Index (GEDTM30)       topographic wetness index       eo-data_twi-edtm       index                 EUCROPMAP Pan-EU year 2022       crop.type-eucropmap.v1       eo-data_crop.type-eucropmap.v1_2022       index           Data format   The EO derived data are available in 2 differently formatted json objects, a flat version more suitable for direct accessibility and a nested version intended for users who wants to convert the data to a relational sql database. The flat version does not include any arrays whereas the nested version is structured with nested arrays.   Flat json object   Below is an example of a flat json object for EO-data observations. Object keys with a double underscore denote a link to other entries in the database. For example “person__email” is a key for the person responsible, with more detailed contact information in a separate json document. The hashtag comments are not allowed when operating a json file, they are added here for explaining each json object.   {   \"campaign\": \"ai4sh_se-loennstorp\", # 'project'+'_'+'country'+'-'+'pilot site'   \"sampling_log\": { # Assembly of samples collected for a common source     \"name\": \"se-loennstorp-0\",  # 'country'+'-'+'pilot site'+'-'+'sampling date' where sampling date = 0 denotes a static variable     \"date_stamp\": 0, # Date when the sample was collected, 0 denotes a static variable     \"person__email\": \"na\"  # Email of person responsible for the sampling/retrieval   },   \"sample\": \"se-loennstorp-0_5-a_0-1\", # 'country'+'-'+'pilot site'+'-'+'sampling date'+'_'+'sample   \"locus\": { # Assembly of the point location, setting and sampling depth     \"position__name\": \"se-loennstorp_5-a\", # 'country'+'-'+'pilot site'+'-'+'sample point id' used for checking internal consistency     \"point\": \"5-a\", # sample point id     \"setting\": \"uniform\",  # The spatial setting in the landscape (uniform, alley, plantline, edge)     \"latitude\": 55.6689444444444, # Latitude in WGS84     \"longitude\": 13.1081388888888, # Longitude in WGS84     \"min_depth\": 0, # Sample upper limit in cm in soil profile     \"max_depth\": 1 # Sample lower limit in cm in soil profile   },   \"observation\": { # Assembly of metadata and data related to the observation     \"sample_preparation__name\": \"eo-data\", # Explanatory name of soil treatment before observation     \"person__email\": \"analysis@gmail.com\", # Email of person responsible for the observation/EO-data processing     \"subsample\": \"a\", # For registering if physically different soil subsamples were analysed separately, default for all EO-data = a     \"replicate\": 0,  # If the analysis was replicated on the same subsample more than once, EO-data is always set to 0     \"n_repeats\": 1,  # The number of repetitions done by any instrument on a single subsample, default for all EO-data = 1     \"date_stamp\": \"20251010\", # Date when the analysis was done     \"logistic\": { # Assembly of logistical metadata on the handling of the sample, not applicable for EO-data       \"sample_preservation__name\": \"none\",       \"sample_transport__name\": \"none\",       \"transport_duration_h\": 0,       \"sample_storage__name\": \"na\",       \"storage_duration_h\": 0,       \"person__email\": \"na\"     },     \"analysis\": { # Assembly of analysis results, procedures and instruments       \"eo-data_dfme-edtm\": { # Soil property reported from the analysis         \"value\": -2.0, # The recorded value         \"unit__name\": \"index\", # the recorded unit         \"indicator__name\": \"eo-data_dfme-edtm\",         \"procedure\": \"eo-data\", # The name of the indicator         \"analysis_method__name\": \"eo-data-dfme_edtm\", # The procedure used for the analysis         \"instrument_brand__name\": \"eo-data-dfme_edtm\",  # Instrument type or brand used for the analysis         \"instrument_model__name\": \"eo-data\", # Instrument model used for the analysis         \"instrument_id\": \"0\" # Any individual instrument id, the default for unknown id is \"0\"       },       \"eo-data_geomorphon-edtm\": { # Each property reported with the same instrument/metod etc is reported as individual objects         \"value\": 6.0,         \"unit__name\": \"index\",         \"indicator__name\": \"eo-data_geomorphon-edtm\",         \"procedure\": \"eo-data\",         \"analysis_method__name\": \"eo-data-geomorphon_edtm\",         \"instrument_brand__name\": \"eo-data-geomorphon_edtm\",         \"instrument_model__name\": \"eo-data\",         \"instrument_id\": \"0\"       },       \"eo-data_hillshade-edtm\": { # Each property reported with the same instrument/metod etc is reported as individual objects         \"value\": 18518.0,         \"unit__name\": \"index\",         \"indicator__name\": \"eo-data_hillshade-edtm\",         \"procedure\": \"eo-data\",         \"analysis_method__name\": \"eo-data-hillshade_edtm\",         \"instrument_brand__name\": \"eo-data-hillshade_edtm\",         \"instrument_model__name\": \"eo-data\",         \"instrument_id\": \"0\"       },       \"eo-data_ls.factor-edtm\": { # Each property reported with the same instrument/metod etc is reported as individual objects         \"value\": 86.0,         \"unit__name\": \"index\",         \"indicator__name\": \"eo-data_ls.factor-edtm\",         \"procedure\": \"eo-data\",         \"analysis_method__name\": \"eo-data-ls.factor_edtm\",         \"instrument_brand__name\": \"eo-data-ls.factor_edtm\",         \"instrument_model__name\": \"eo-data\",         \"instrument_id\": \"0\"       },       \"eo-data_maxic-edtm\": { # Each property reported with the same instrument/metod etc is reported as individual objects         \"value\": 2691.0,         \"unit__name\": \"index\",         \"indicator__name\": \"eo-data_maxic-edtm\",         \"procedure\": \"eo-data\",         \"analysis_method__name\": \"eo-data-maxic_edtm\",         \"instrument_brand__name\": \"eo-data-maxic_edtm\",         \"instrument_model__name\": \"eo-data\",         \"instrument_id\": \"0\"       },       \"eo-data_minic-edtm\": { # Each property reported with the same instrument/metod etc is reported as individual objects         \"value\": -2201.0,         \"unit__name\": \"index\",         \"indicator__name\": \"eo-data_minic-edtm\",         \"procedure\": \"eo-data\",         \"analysis_method__name\": \"eo-data-minic_edtm\",         \"instrument_brand__name\": \"eo-data-minic_edtm\",         \"instrument_model__name\": \"eo-data\",         \"instrument_id\": \"0\"       },       \"eo-data_neg.openness-edtm\": { # Each property reported with the same instrument/metod etc is reported as individual objects         \"value\": 8971.0,         \"unit__name\": \"index\",         \"indicator__name\": \"eo-data_neg.openness-edtm\",         \"procedure\": \"eo-data\",         \"analysis_method__name\": \"eo-data-neg.openness_edtm\",         \"instrument_brand__name\": \"eo-data-neg.openness_edtm\",         \"instrument_model__name\": \"eo-data\",         \"instrument_id\": \"0\"       },       \"eo-data_pos.openness-edtm\": { # Each property reported with the same instrument/metod etc is reported as individual objects         \"value\": 8987.0,         \"unit__name\": \"index\",         \"indicator__name\": \"eo-data_pos.openness-edtm\",         \"procedure\": \"eo-data\",         \"analysis_method__name\": \"eo-data-pos.openness_edtm\",         \"instrument_brand__name\": \"eo-data-pos.openness_edtm\",         \"instrument_model__name\": \"eo-data\",         \"instrument_id\": \"0\"       },       \"eo-data_pro.curv-edtm\": { # Each property reported with the same instrument/metod etc is reported as individual objects         \"value\": 2656.0,         \"unit__name\": \"index\",         \"indicator__name\": \"eo-data_pro.curv-edtm\",         \"procedure\": \"eo-data\",         \"analysis_method__name\": \"eo-data-pro.curv_edtm\",         \"instrument_brand__name\": \"eo-data-pro.curv_edtm\",         \"instrument_model__name\": \"eo-data\",         \"instrument_id\": \"0\"       },       \"eo-data_shpindx-edtm\": { # Each property reported with the same instrument/metod etc is reported as individual objects         \"value\": 146.0,         \"unit__name\": \"index\",         \"indicator__name\": \"eo-data_shpindx-edtm\",         \"procedure\": \"eo-data\",         \"analysis_method__name\": \"eo-data-shpindx_edtm\",         \"instrument_brand__name\": \"eo-data-shpindx_edtm\",         \"instrument_model__name\": \"eo-data\",         \"instrument_id\": \"0\"       },       \"eo-data_slope.in.degree-edtm\": { # Each property reported with the same instrument/metod etc is reported as individual objects         \"value\": 91.0,         \"unit__name\": \"index\",         \"indicator__name\": \"eo-data_slope.in.degree-edtm\",         \"procedure\": \"eo-data\",         \"analysis_method__name\": \"eo-data-slope.in.degree_edtm\",         \"instrument_brand__name\": \"eo-data-slope.in.degree_edtm\",         \"instrument_model__name\": \"eo-data\",         \"instrument_id\": \"0\"       },       \"eo-data_twi-edtm\": { # Each property reported with the same instrument/metod etc is reported as individual objects         \"value\": 593.0,         \"unit__name\": \"index\",         \"indicator__name\": \"eo-data_twi-edtm\",         \"procedure\": \"eo-data\",         \"analysis_method__name\": \"eo-data-twi_edtm\",         \"instrument_brand__name\": \"eo-data-twi_edtm\",         \"instrument_model__name\": \"eo-data\",         \"instrument_id\": \"0\"       },       \"eo-data_crop.type-eucropmap.v1\": { # Each property reported with the same instrument/metod etc is reported as individual objects         \"value\": 214.0,         \"unit__name\": \"index\",         \"indicator__name\": \"eo-data_crop.type-eucropmap.v1\",         \"procedure\": \"eo-data\",         \"analysis_method__name\": \"eo-data-crop.type_eucropmap.v1\",         \"instrument_brand__name\": \"eo-data-crop.type_eucropmap.v1\",         \"instrument_model__name\": \"eo-data\",         \"instrument_id\": \"0\"       }     }   } }   Nested json object   The nested json object for EO-data observations contains the same object keys and values as the flat version, but hierarchically organised with nested arrays. For explanations of the object keys please see the flat json object above.   {  \"data_source\": [    {      \"name\": \"ai4sh_se-loennstorp\",      \"site\": [        {          \"name\": \"se-loennstorp\",          \"point\": [            {              \"name\": \"5-a\",              \"latitude\": 55.6689444444444,              \"longitude\": 13.1081388888888,              \"setting\": \"uniform\",              \"sampling_log\": [                {                  \"name\": \"se-loennstorp-0\",                  \"date_stamp\": 0,                  \"person__email\": \"na\",                  \"sample\": [                    {                      \"name\": \"se-loennstorp-0_5-a_0-1\",                      \"min_depth\": 0,                      \"max_depth\": 1,                      \"observation\": [                        {                          \"sample_preparation__name\": \"eo-data\",                          \"person__email\": \"analysis@gmail.com\",                          \"subsample\": \"a\",                          \"replicate\": 0,                          \"n_repeats\": 1,                          \"date_stamp\": \"20251010\",                          \"logistic\": {                            \"sample_preservation__name\": \"none\",                            \"sample_transport__name\": \"none\",                            \"transport_duration_h\": 0,                            \"sample_storage__name\": \"na\",                            \"storage_duration_h\": 0,                            \"person__email\": \"na\"                          },                          \"analysis_method\": {                            \"eo-data\": [                              {                                \"eo-data\": [                                  {                                    \"value\": -2.0,                                    \"unit__name\": \"index\",                                    \"indicator__name\": \"eo-data_dfme-edtm\",                                    \"analysis_method__name\": \"eo-data-dfme_edtm\",                                    \"instrument_brand__name\": \"eo-data-dfme_edtm\",                                    \"instrument_model__name\": \"eo-data\",                                    \"instrument_id\": \"0\"                                  },                                  {                                    \"value\": 6.0,                                    \"unit__name\": \"index\",                                    \"indicator__name\": \"eo-data_geomorphon-edtm\",                                    \"procedure\": \"eo-data\",                                    \"analysis_method__name\": \"eo-data-geomorphon_edtm\",                                    \"instrument_brand__name\": \"eo-data-geomorphon_edtm\",                                    \"instrument_model__name\": \"eo-data\",                                    \"instrument_id\": \"0\"                                  },                                  {                                    \"value\": 18518.0,                                    \"unit__name\": \"index\",                                    \"indicator__name\": \"eo-data_hillshade-edtm\",                                    \"procedure\": \"eo-data\",                                    \"analysis_method__name\": \"eo-data-hillshade_edtm\",                                    \"instrument_brand__name\": \"eo-data-hillshade_edtm\",                                    \"instrument_model__name\": \"eo-data\",                                    \"instrument_id\": \"0\"                                  },                                  {                                    \"value\": 86.0,                                    \"unit__name\": \"index\",                                    \"indicator__name\": \"eo-data_ls.factor-edtm\",                                    \"procedure\": \"eo-data\",                                    \"analysis_method__name\": \"eo-data-ls.factor_edtm\",                                    \"instrument_brand__name\": \"eo-data-ls.factor_edtm\",                                    \"instrument_model__name\": \"eo-data\",                                    \"instrument_id\": \"0\"                                  },                                  {                                    \"value\": 2691.0,                                    \"unit__name\": \"index\",                                    \"indicator__name\": \"eo-data_maxic-edtm\",                                    \"procedure\": \"eo-data\",                                    \"analysis_method__name\": \"eo-data-maxic_edtm\",                                    \"instrument_brand__name\": \"eo-data-maxic_edtm\",                                    \"instrument_model__name\": \"eo-data\",                                    \"instrument_id\": \"0\"                                  },                                  {                                    \"value\": -2201.0,                                    \"unit__name\": \"index\",                                    \"indicator__name\": \"eo-data_minic-edtm\",                                    \"procedure\": \"eo-data\",                                    \"analysis_method__name\": \"eo-data-minic_edtm\",                                    \"instrument_brand__name\": \"eo-data-minic_edtm\",                                    \"instrument_model__name\": \"eo-data\",                                    \"instrument_id\": \"0\"                                  },                                  {                                    \"value\": 8971.0,                                    \"unit__name\": \"index\",                                    \"indicator__name\": \"eo-data_neg.openness-edtm\",                                    \"procedure\": \"eo-data\",                                    \"analysis_method__name\": \"eo-data-neg.openness_edtm\",                                    \"instrument_brand__name\": \"eo-data-neg.openness_edtm\",                                    \"instrument_model__name\": \"eo-data\",                                    \"instrument_id\": \"0\"                                  },                                  {                                    \"value\": 8987.0,                                    \"unit__name\": \"index\",                                    \"indicator__name\": \"eo-data_pos.openness-edtm\",                                    \"procedure\": \"eo-data\",                                    \"analysis_method__name\": \"eo-data-pos.openness_edtm\",                                    \"instrument_brand__name\": \"eo-data-pos.openness_edtm\",                                    \"instrument_model__name\": \"eo-data\",                                    \"instrument_id\": \"0\"                                  },                                  {                                    \"value\": 2656.0,                                    \"unit__name\": \"index\",                                    \"indicator__name\": \"eo-data_pro.curv-edtm\",                                    \"procedure\": \"eo-data\",                                    \"analysis_method__name\": \"eo-data-pro.curv_edtm\",                                    \"instrument_brand__name\": \"eo-data-pro.curv_edtm\",                                    \"instrument_model__name\": \"eo-data\",                                    \"instrument_id\": \"0\"                                  },                                  {                                    \"value\": 146.0,                                    \"unit__name\": \"index\",                                    \"indicator__name\": \"eo-data_shpindx-edtm\",                                    \"procedure\": \"eo-data\",                                    \"analysis_method__name\": \"eo-data-shpindx_edtm\",                                    \"instrument_brand__name\": \"eo-data-shpindx_edtm\",                                    \"instrument_model__name\": \"eo-data\",                                    \"instrument_id\": \"0\"                                  },                                  {                                    \"value\": 91.0,                                    \"unit__name\": \"index\",                                    \"indicator__name\": \"eo-data_slope.in.degree-edtm\",                                    \"procedure\": \"eo-data\",                                    \"analysis_method__name\": \"eo-data-slope.in.degree_edtm\",                                    \"instrument_brand__name\": \"eo-data-slope.in.degree_edtm\",                                    \"instrument_model__name\": \"eo-data\",                                    \"instrument_id\": \"0\"                                  },                                  {                                    \"value\": 593.0,                                    \"unit__name\": \"index\",                                    \"indicator__name\": \"eo-data_twi-edtm\",                                    \"procedure\": \"eo-data\",                                    \"analysis_method__name\": \"eo-data-twi_edtm\",                                    \"instrument_brand__name\": \"eo-data-twi_edtm\",                                    \"instrument_model__name\": \"eo-data\",                                    \"instrument_id\": \"0\"                                  },                                  {                                    \"value\": 214.0,                                    \"unit__name\": \"index\",                                    \"indicator__name\": \"eo-data_crop.type-eucropmap.v1\",                                    \"procedure\": \"eo-data\",                                    \"analysis_method__name\": \"eo-data-crop.type_eucropmap.v1\",                                    \"instrument_brand__name\": \"eo-data-crop.type_eucropmap.v1\",                                    \"instrument_model__name\": \"eo-data\",                                    \"instrument_id\": \"0\"                                  }                                ]                              }                            ]                          }                        }                      ]                    }                  ]                }              ]            }          ]        }      ]    }  ] }   Data access and file naming   The data is provided under the following licenses:      Data License: Creative Commons Attribution license (CC-BY)   Code License: Massachusetts Institute of Technology License (MIT License)   recorded data from the Microbiometer analysis are available at      Flat json format https://github.com/AI4SH/in-situ_data/tree/main/flat/eo-data   Nested json format: https://github.com/AI4SH/in-situ_data/tree/main/flat/eo-data   The file naming is defined to separate all eventual renewed sampling or analysis efforts and is built up from the following metadata:     country   pilot site   sampling date   point id   min depth   max depth   subsample   replicate   setting   sample_preservation   procedure   instrument   analysis date   For the examples above, the file name thus have the following component values:      se   loennstorp   0   5-a   0   1   a   0   uniform   eo-data   eo-data   0   20251010   And the final file name becomes:   se-loennstorp-0_5-a_0-1_a_0_uniform_eo-data_eo-data_0_20251010   Acknowledgments and Funding   This work is part of the AI4SoilHealth project, funded by the European Union’s Horizon Europe Research and Innovation Programme under Grant Agreement No. 101086179.   Funded by the European Union. The views expressed are those of the authors and do not necessarily reflect those of the European Union or the European Research Executive Agency.   ","categories": [],
        "tags": [],
        "url": "/in-situ_data_docs/docs/in-situ_methods/eo-data/",
        "teaser": null
      },,]
