var store = [{
        "title": "Aggregates",
        "excerpt":"Aggregates, clumps of smaller soil particles glued together by biological processes, are key components for e.g. soil hydraulic properties, nutrient recycling and pool sizes. Stable soil aggregates also contribute to resistance towards wind and water erosion, and are linked to improved water capture, infiltration, and storage.   The smartphone apps Slakes and Moulder (a renamed version of Slakes) uses image analysis of dried, pea sized soil aggregates before and after soaking. Aggregates that resist dispersion after rewetting and slake (or mould, i.e. disintegrate) less are more stable compared to those that break apart. A higher resistance indicate a healthier soil.   Short method description   From a dired soil sample three pea sized aggregates are selected and put in a shallow dish. A smartphone with the app installed is mounted above the dish and an image taken of the dish and the three aggregates. The dish is then gently filled with water, enough to just cover the aggregates. The user can adjust the app’s identification of the aggregates, and another images is captured. After 10 minutes a third image is taken and the app calculates an aggregate stability index.   Data format   The AI4SH observed aggregate stability data are available in 2 differently formatted json objects, a flat version more suitable for direct accessibility and a nested version intended for users who wants to convert the data to a relational sql database. The flat version does not include any arrays whereas the nested version is structured with nested arrays.   Flat json object   Below is an example of a flat json object for aggregates stability observations. Object keys with a double underscore denote a link to other entries in the database. For example “person__email” is a key for the person responsible, with more detailed contact information in a separate json document. The hashtag comments are not allowed when operating a json file, they are added here for explaining each json object.   {   \"data_source\": \"ai4sh_se-loennstorp\", # 'project'+'_'+'country'+'-'+'pilot site'   \"sampling_log\": { # Assembly of samples collected in the same day at a particular pilot site       \"name\": \"se-loennstorp-20240815\", # 'country'+'-'+'pilot site'+'-'+'sampling date'     \"date_stamp\": \"20240815\", # Date when the sample was collected in the field     \"person__email\": \"sampling@gmail.com\" # Email of person responsible for the sampling   },   \"sample\": \"se-loennstorp-20240815_55-c_0-20\", # 'country'+'-'+'pilot site'+'-'+'sampling date'+'_'+'sample point id'+'_'+'min depth''+'-'+'max depth'   \"locus\": { # Assembly of the point location, setting and depth     \"position__name\": \"se-loennstorp_55-c\", # 'country'+'-'+'pilot site'+'-'+'sample point id' used for checking internal consistency     \"point\": \"55-c\", # sample point id     \"setting\": \"uniform\", # The spatial setting in the landscape (uniform, alley, plantline, edge)     \"latitude\": 55.6671111111111, # Latitude in WGS84     \"longitude\": 13.1156944444444, # Longitude in WGS84     \"min_depth\": 0, # Sample upper limit in cm in soil profile     \"max_depth\": 20 # Sample lower limit in cm in soil profile   },   \"observation\": { # Assembly of metadata and data related to the obeservation     \"sample_preparation__name\": \"dried-aggregate-select+soaked\", # Explanatory name of soil treatment before observation     \"person__email\": \"analysis@gmail.com\", # Email of person responsible for the obeservation     \"subsample\": \"a\", # For registering if physically different soil subsamples were analysed separately, default for a single analysis = a     \"replicate\": 0, # If the analysis was replicated on the same subsample more than once, the first test is always set to 0     \"n_repeats\": 1, # The number of repetitions done by any instrument on a single subsample and as part of a single replicate, with results reported as mean or mean and standard deviation     \"date_stamp\": \"20241110\", # Date when the analysis was done     \"logistic\": { # Assembly of logistical metadata on the handling of the sample       \"sample_preservation__name\": null, # Explanatory name of soil preservation       \"sample_transport__name\": null, # Explanatory name of soil transportation if sensitive for analysis results       \"transport_duration_h\": 0, # duration of any sensitive transportation in hours       \"sample_storage__name\": null, # Explanatory name of soil storage if sensitive for analysis results       \"storage_duration_h\": 2008, # Storage duration in hours       \"person__email\": \"logistic@gmail.com\" # Email of person responsible for the logistics     },     \"analysis\": { # Assembly of analysis results, procedures and instruments       \"slakes_aggregate-stability-index\": { # Soil property reported as AI4SH extended naming (part after underscore = core indicator)         \"value\": 0.69, # Recorded value (single value or mean of repeated observations)         \"unit__name\": \"index\", # Recorded unit         \"indicator__name\": \"slakes_aggregate-stability-index\", # AI4SH extended name of the soil indicator (part after underscore = core indicator)         \"procedure\": \"slakes\", # Procedure for the analysis (not always similar to method)         \"analysis_method__name\": \"slakes\", # Name of the method         \"instrument_brand__name\": \"smartphone\", # Instrument type or brand         \"instrument_model__name\": \"slakes+samsung-a26\", # Model or version of the instrument         \"instrument_id\": \"0\" # Any individual instrument id, the default for unknown id is \"0\"       }     }   } }   Nested json object   The nested json object for aggregate stability observations contains the same object keys and values as the flat version, but hierarcically organised with nested arrays. For explanations of the object keys please see the flat json object above.   {   \"data_source\": [     {       \"name\": \"ai4sh_se-loennstorp\",       \"site\": [         {           \"name\": \"se-loennstorp\",           \"point\": [             {               \"name\": \"55-c\",               \"latitude\": 55.6671111111111,               \"longitude\": 13.1156944444444,               \"setting\": \"uniform\",               \"sampling_log\": [                 {                   \"name\": \"se-loennstorp-20240815\",                   \"date_stamp\": \"20240815\",                   \"person__email\": \"sampling@gmail.com\",                   \"sample\": [                     {                       \"name\": \"se-loennstorp-20240815_55-c_0-20\",                       \"min_depth\": 0,                       \"max_depth\": 20,                       \"observation\": [                         {                           \"sample_preparation__name\": \"dried-aggregate-select+soaked\",                           \"person__email\": \"analysis@gmail.com\",                           \"subsample\": \"a\",                           \"replicate\": 0,                           \"n_repeats\": 1,                           \"date_stamp\": \"20241110\",                           \"logistic\": {                             \"sample_preservation__name\": \"dried\",                             \"sample_transport__name\": null,                             \"transport_duration_h\": 0,                             \"sample_storage__name\": null,                             \"storage_duration_h\": 2088,                             \"person__email\": \"logistic@gmail.com\"                           },                           \"analysis_method\": {                             \"slakes\": [                               {                                 \"slakes app\": [                                   {                                     \"value\": 0.69,                                     \"unit__name\": \"index\",                                     \"indicator__name\": \"slakes_aggregate-stability-index\",                                     \"analysis_method__name\": \"slakes\",                                     \"instrument_brand__name\": \"smartphone\",                                     \"instrument_model__name\": \"slakes+samsung-a26\",                                     \"instrument_id\": \"0\"                                   }                                 ]                               }                             ]                           }                         }                       ]                     }                   ]                 }               ]             }           ]         }       ]     }   ] }   Data access and file naming   Recorded data from the aggregate stability analysis are available at:      Flat json format https://github.com/AI4SH/in-situ_data/tree/main/flat/aggregates   Nested json format: https://github.com/AI4SH/in-situ_data/tree/main/nested/aggregates   The file naming is defined to separate alll eventual renewed sampling or analysis efforts and is built up from the following metadata:     country   pilot site   sampling date   point id   min depth   max depth   subsample   replicate   setting   sample_preservation   procedure   instrument   analysis date   For the examples above, the file name thus have the following component values:      se   loennstorp   20240815   55-c   0   20   a   0   uniform   dry-pick-soak   slakes+samsung-a26   0   20241110   And the final file name becomes:   se-loennstorp-20240815_55-c_0-20_a_0_uniform_dry-pick-soak_slakes+samsung-a26_0_20241110.json   ","categories": [],
        "tags": [],
        "url": "/in-situ_methods/aggregates/",
        "teaser": null
      },{
        "title": "Bulk density and volumetric soil moisture content",
        "excerpt":"Soil compaction is recognised as one of the major threats to soil health. Compaction affects the infiltration, soil water holding capacity and other hydraulic properties. It also affects the macrofauna and indirectly also the microbiological community.   The bulk density of the soil upper layers is influenced by the soil mother material, soil natural processes and human soil management. To record the bulk density the classical method of sinking and withdrawing a soil cylinder into and out of underdisturbed soil is the superior method.   Short method description   A sturdy soil cylinder, usually with one end sharpened and with a precisely determined volumes is sunk (i.e. hammered) into the soil and taken out with soil extending out from both sides. A knife or similar tool is used to cut off any soil sticking out from the cylinder and the ends are sealed, e.g. with a plastic cap. The cylinder with its wet soil is then weighed, dried at 105o C and weighed again. Together with the weight of the cylinder (and caps if needed), the weight and volume of the wet and dry cylinder content are calculated. From this data the soil bulk density and volumetric soil moisture content are calculated.   Data format   The AI4SH observed bulk density and volumetric soil moisture content data are available in 2 differently formatted json objects, a flat version more suitable for direct accessibility and a nested version intended for users who wants to convert the data to a relational sql database. The flat version does not include any arrays whereas the nested version is structured with nested arrays.   Flat json object   Below is an example of a flat json object for bulk density and volumetric soil moisture content observations. Object keys with a double underscore denote a link to other entries in the database. For example “person__email” is a key for the person responsible, with more detailed contact information in a separate json document. The hashtag comments are not allowed when operating a json file, they are added here for explaining each json object.   {   \"data_source\": \"ai4sh_se-loennstorp\", # 'project'+'_'+'country'+'-'+'pilot site'   \"sampling_log\": { # Assembly of samples collected in the same day at a particular pilot site       \"name\": \"se-loennstorp-20240815\", # 'country'+'-'+'pilot site'+'-'+'sampling date'     \"date_stamp\": \"20240815\", # Date when the sample was collected in the field     \"person__email\": \"sampling@gmail.com\" # Email of person responsible for the sampling   },   \"sample\": \"se-loennstorp-20240815_55-c_0-20\", # 'country'+'-'+'pilot site'+'-'+'sampling date'+'_'+'sample point id'+'_'+'min depth''+'-'+'max depth'   \"locus\": { # Assembly of the point location, setting and depth     \"position__name\": \"se-loennstorp_55-c\", # 'country'+'-'+'pilot site'+'-'+'sample point id' used for checking internal consistency     \"point\": \"55-c\", # sample point id     \"setting\": \"uniform\", # The spatial setting in the landscape (uniform, alley, plantline, edge)     \"latitude\": 55.6671111111111, # Latitude in WGS84     \"longitude\": 13.1156944444444, # Longitude in WGS84     \"min_depth\": 0, # Sample upper limit in cm in soil profile     \"max_depth\": 20 # Sample lower limit in cm in soil profile   },   \"observation\": { # Assembly of metadata and data related to the obeservation     sample_preparation__name\": \"soil-undisturbed-in-situ\", # Explanatory name of soil treatment before observation     \"person__email\": \"analysis@gmail.com\", # Email of person responsible for the obeservation     \"subsample\": \"a\", # For registering if physically different soil subsamples were analysed separately, default for a single analysis = a     \"replicate\": 0, # If the analysis was replicated on the same subsample more than once, the first test is always set to 0     \"n_repeats\": 1, # The number of repetitions done by any instrument on a single subsample and as part of a single replicate, with results reported as mean or mean and standard deviation     \"date_stamp\": \"20240815\", # Date when the analysis was done     \"logistic\": { # Assembly of logistical metadata on the handling of the sample       \"sample_preservation__name\": null, # Explanatory name of soil preservation       \"sample_transport__name\": null, # Explanatory name of soil transportation if sensitive for analysis results       \"transport_duration_h\": 0, # duration of any sensitive transportation in hours       \"sample_storage__name\": null, # Explanatory name of soil storage if sensitive for analysis results       \"storage_duration_h\": 0, # Storage duration in hours       \"person__email\": \"logistic@gmail.com\" # Email of person responsible for the logistics     },     \"analysis\": { # Assembly of analysis results, procedures and instruments       \"soil-cylinder-drying@105c_soil-moisture-volumetric-content\": { # Soil property reported as AI4SH extended naming (part after underscore = core indicator)         \"value\": 0.162717138949006, # Recorded value (single value or mean of repeated observations)         \"unit__name\": \"vol*vol^-1\", # Recorded unit         \"indicator__name\": \"soil-cylinder-drying@105c_soil-moisture-volumetric-content\", # AI4SH extended name of the soil indicator (part after underscore = core indicator)         \"procedure\": \"soil-cylinder-drying@105c\", # Procedure for the analysis (not always similar to method)         \"analysis_method__name\": \"soil-cylinder-drying@105c\", # Name of the method         \"instrument_brand__name\": \"general-soil-cylinder\", # Instrument type or brand         \"instrument_model__name\": \"0\", # Model or version of the instrument         \"instrument_id\": \"0\" # Any individual instrument id, the default for unknown id is \"0\"           }       \"soil-cylinder-drying@105c_bulk-density\": {         \"value\": 1.28338286105099,         \"unit__name\": \"g*cm^-3\",         \"indicator__name\": \"soil-cylinder-drying@105c_bulk-density\",         \"procedure\": \"soil-cylinder-drying@105c\",         \"analysis_method__name\": \"soil-cylinder-drying@105c\",         \"instrument_brand__name\": \"general-soil-cylinder\",         \"instrument_model__name\": \"0\",         \"instrument_id\": \"0\"       }     }   } }   Nested json object   The nested json object for soil bulk density and volumetric soil moisture content observations contains the same object keys and values as the flat version, but hierarcically organised with nested arrays. For explanations of the object keys please see the flat json object above.   {   \"data_source\": [     {       \"name\": \"ai4sh_se-loennstorp\",       \"site\": [         {           \"name\": \"se-loennstorp\",           \"point\": [             {               \"name\": \"55-c\",               \"latitude\": 55.6671111111111,               \"longitude\": 13.1156944444444,               \"setting\": \"uniform\",               \"sampling_log\": [                 {                   \"name\": \"se-loennstorp-20240815\",                   \"date_stamp\": \"20240815\",                   \"person__email\": \"sampling@gmail.com\",                   \"sample\": [                     {                       \"name\": \"se-loennstorp-20240815_55-c_0-20\",                       \"min_depth\": 0,                       \"max_depth\": 20,                       \"observation\": [                         {                           \"sample_preparation__name\": \"soil-undisturbed-in-situ\",                           \"person__email\": \"analysis@gmail.com\",                           \"subsample\": \"a\",                           \"replicate\": 0,                           \"n_repeats\": 1,                           \"date_stamp\": \"20240815\",                           \"logistic\": {                             \"sample_preservation__name\": null,                             \"sample_transport__name\": null,                             \"transport_duration_h\": 0,                             \"sample_storage__name\": null,                             \"storage_duration_h\": 0,                             \"person__email\": \"logistic@gmail.com\"                           },                           \"analysis_method\": {                             \"soil-cylinder-drying@105c\": [                               {                                 \"soil-cylinder\": [                                   {                                     \"value\": 0.162717138949006,                                     \"unit__name\": \"vol*vol^-1\",                                     \"indicator__name\": \"soil-cylinder-drying@105c_soil-moisture-volumetric-content\",                                     \"analysis_method__name\": \"soil-cylinder-drying@105c\",                                     \"instrument_brand__name\": \"general-soil-cylinder\",                                     \"instrument_model__name\": \"0\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 1.28338286105099,                                     \"unit__name\": \"g*cm^-3\",                                     \"indicator__name\": \"soil-cylinder-drying@105c_bulk-density\",                                     \"procedure\": \"soil-cylinder-drying@105c\",                                     \"analysis_method__name\": \"soil-cylinder-drying@105c\",                                     \"instrument_brand__name\": \"general-soil-cylinder\",                                     \"instrument_model__name\": \"0\",                                     \"instrument_id\": \"0\"                                   }                                 ]                               }                             ]                           }                         }                       ]                     }                   ]                 }               ]             }           ]         }       ]     }   ] }   Data access and file naming   Recorded data from the bulk density and volumetric soil moisture content analysis are available at:      Flat json format https://github.com/AI4SH/in-situ_data/tree/main/flat/bulk_density   Nested json format: https://github.com/AI4SH/in-situ_data/tree/main/nested/bulk_density   The file naming is defined to separate alll eventual renewed sampling or analysis efforts and is built up from the following metadata:     country   pilot site   sampling date   point id   min depth   max depth   subsample   replicate   setting   sample_preservation   procedure   instrument   analysis date   For the examples above, the file name thus have the following component values:      se   loennstorp   20240815   55-c   0   20   a   0   uniform   no-prep   soil-cylinder-drying@105c   0   20240815   And the final file name becomes:   se-loennstorp-20240815_55-c_0-20_a_0_uniform_no-prep_soil-cylinder-drying@105c_0_20240815.json   ","categories": [],
        "tags": [],
        "url": "/in-situ_methods/bulkdensity/",
        "teaser": null
      },{
        "title": "Enzymatic activity",
        "excerpt":"All biolgical processes and functions are related to enzymes and their activities. Assessing enzymatic metabolic activities and their rates have traditionally been a specialist bound and expensive analysis. The Swiss start-up Digit Soil, a partner of the AI4SH project, has developed a rapid method for observing five key enzymatic activities. The instrument developed by Digit Soil is a stand alone unit that can be used in an ordinary home or office and is applicable for use by for instance citizen scientists.   Short method description   The Soil Enzymatic Activity Reader (SEAR) developed by Digit Soil uses a combination of reaction plates and fluorescence spectroscopy. The method requires fresh samples, either directly from the field or kept at a few degrees C when stored and transported and then analysed. Soil samples are placed on a reaction plate with 5*5 cells (i.e. 5 replicates for each enzyme) and inserted into a spectrometer that uses UV light to excite the enzymes and a digital camera that registers the enzmatic activity and its change over a 40 minute period. The enzymatic activity rates of five key enzymes are then reported. The enzymatic activities observed are summarised in table 1.   Table 1. Enzymatic activities recorded with each observation with the Soil Enzymatic Activity Reader (SEAR).                  Property       Indicator       AI4SH extended naming       Unit                       Chitinase/β-glucosaminidase       GLA       digit-soil-sear_GLA       pmol*min^-1                 β-Glucosidase       GLU       digit-soil-sear_GLU       pmol*min^-1                 phosphatases (Phosphomonesterases)       PHO       digit-soil-sear_PHO       pmol*min^-1                 β-Xylosidase       XYL       digit-soil-sear_XYL       pmol*min^-1                 Leucine-aminopeptidase       LEU       digit-soil-sear_LEU       pmol*min^-1           Data format   Each SEAR measurement is based on 5 replications of each enzymatic activity and the recorded data reports the mean value. The observed enzymatic activity data are available in 2 differently formatted json objects, a flat version more suitable for direct accessibility and a nested version intended for users who wants to convert the data to a relational sql database. The flat version does not include any arrays whereas the nested version is structured with nested arrays.   Flat json object   Below is an example of a flat json object for enzymatic observations. Object keys with a double underscore denote a link to other entries in the database. For example “person__email” is a key for the person responsible, with more detailed contact information in a separate json document. The hashtag comments are not allowed when operating a json file, they are added here for explaining each json object.   {   \"data_source\": \"ai4sh_se-loennstorp\", # 'project'+'_'+'country'+'-'+'pilot site'   \"sampling_log\": { # Assembly of samples collected in the same day at a particular pilot site       \"name\": \"se-loennstorp-20240815\", # 'country'+'-'+'pilot site'+'-'+'sampling date'     \"date_stamp\": \"20240815\", # Date when the sample was collected in the field     \"person__email\": \"sampling@gmail.com\" # Email of person responsible for the sampling   },   \"sample\": \"se-loennstorp-20240815_55-c_0-20\", # 'country'+'-'+'pilot site'+'-'+'sampling date'+'_'+'sample point id'+'_'+'min depth''+'-'+'max depth'   \"locus\": { # Assembly of the point location, setting and depth     \"position__name\": \"se-loennstorp_55-c\", # 'country'+'-'+'pilot site'+'-'+'sample point id' used for checking internal consistency     \"point\": \"55-c\", # sample point id     \"setting\": \"uniform\", # The spatial setting in the landscape (uniform, alley, plantline, edge)     \"latitude\": 55.6671111111111, # Latitude in WGS84     \"longitude\": 13.1156944444444, # Longitude in WGS84     \"min_depth\": 0, # Sample upper limit in cm in soil profile     \"max_depth\": 20 # Sample lower limit in cm in soil profile   },   \"observation\": { # Assembly of metadata and data related to the obeservation     \"sample_preparation__name\": \"mixed-untreated-soil-in-lab\", # Explanatory name of soil treatment before observation     \"person__email\": \"analysis@gmail.com\", # Email of person responsible for the obeservation     \"subsample\": \"a\", # For registering if physically different soil subsamples were analysed separately, default for a single analysis = a     \"replicate\": 0, # If the analysis was replicated on the same subsample more than once, the first test is always set to 0     \"n_repeats\": 5, # The number of repetitions done by any instrument on a single subsample and as part of a single replicate, with results reported as mean or mean and standard deviation     \"date_stamp\": \"20240815\", # Date when the analysis was done     \"logistic\": { # Assembly of logistical metadata on the handling of the sample       \"sample_preservation__name\": null, # Explanatory name of soil preservation       \"sample_transport__name\": null, # Explanatory name of soil transportation if sensitive for analysis results       \"transport_duration_h\": 0, # duration of any sensitive transportation in hours       \"sample_storage__name\": null, # Explanatory name of soil storage if sensitive for analysis results       \"storage_duration_h\": 0, # Storage duration in hours       \"person__email\": \"logistic@gmail.com\" # Email of person responsible for the logistics     },     \"analysis\": { # Assembly of analysis results, procedures and instruments       \"digit-soil-sear_LEU\": { # Soil property reported as AI4SH extended naming (part after underscore = core indicator)         \"value\": 3.5, # Recorded value (single value or mean of repeated observations)         \"unit__name\": \"pmol*min^-1\", # Recorded unit         \"indicator__name\": \"digit-soil-sear_LEU\", # AI4SH extended name of the soil indicator (part after underscore = core indicator)         \"procedure\": \"digit-soil-sear\", # Procedure for the analysis (not always similar to method)         \"analysis_method__name\": \"digit-soil-sear\", # Name of the method         \"instrument_brand__name\": \"digit-soil\", # Instrument type or brand         \"instrument_model__name\": \"bob\", # Model or version of the instrument         \"instrument_id\": \"0\" # Any individual instrument id, the default for unknown id is \"0\"       },       \"digit-soil-sear_GLA\": {         \"value\": 8.2905,         \"unit__name\": \"pmol*min^-1\",         \"indicator__name\": \"digit-soil-sear_GLA\",         \"procedure\": \"digit-soil-sear\",         \"analysis_method__name\": \"digit-soil-sear\",         \"instrument_brand__name\": \"digit-soil\",         \"instrument_model__name\": \"bob\",         \"instrument_id\": \"0\"       },       \"digit-soil-sear_GLU\": {         \"value\": 25.273,         \"unit__name\": \"pmol*min^-1\",         \"indicator__name\": \"digit-soil-sear_GLU\",         \"procedure\": \"digit-soil-sear\",         \"analysis_method__name\": \"digit-soil-sear\",         \"instrument_brand__name\": \"digit-soil\",         \"instrument_model__name\": \"bob\",         \"instrument_id\": \"0\"       },       \"digit-soil-sear_PHO\": {         \"value\": 60.2077,         \"unit__name\": \"pmol*min^-1\",         \"indicator__name\": \"digit-soil-sear_PHO\",         \"procedure\": \"digit-soil-sear\",         \"analysis_method__name\": \"digit-soil-sear\",         \"instrument_brand__name\": \"digit-soil\",         \"instrument_model__name\": \"bob\",         \"instrument_id\": \"0\"       },       \"digit-soil-sear_XYL\": {         \"value\": 8.1644,         \"unit__name\": \"pmol*min^-1\",         \"indicator__name\": \"digit-soil-sear_XYL\",         \"procedure\": \"digit-soil-sear\",         \"analysis_method__name\": \"digit-soil-sear\",         \"instrument_brand__name\": \"digit-soil\",         \"instrument_model__name\": \"bob\",         \"instrument_id\": \"0\"       }     }   } }   Nested json object   The nested json object for soil enzymatic activity observations contains the same object keys and values as the flat version, but hierarcically organised with nested arrays. For explanations of the object keys please see the flat json object above.   {   \"data_source\": [     {       \"name\": \"ai4sh_se-loennstorp\",       \"site\": [         {           \"name\": \"se-loennstorp\",           \"point\": [             {               \"name\": \"55-c\",               \"latitude\": 55.6671111111111,               \"longitude\": 13.1156944444444,               \"setting\": \"uniform\",               \"sampling_log\": [                 {                   \"name\": \"se-loennstorp-20240815\",                   \"date_stamp\": \"20240815\",                   \"person__email\": \"sampling@gmail.com\",                   \"sample\": [                     {                       \"name\": \"se-loennstorp-20240815_55-c_0-20\",                       \"min_depth\": 0,                       \"max_depth\": 20,                       \"observation\": [                         {                           \"sample_preparation__name\": \"mixed-untreated-soil-in-lab\",                           \"person__email\": \"analysis@gmail.com\",                           \"subsample\": \"a\",                           \"replicate\": 0,                           \"n_repeats\": 1,                           \"date_stamp\": \"20240815\",                           \"logistic\": {                             \"sample_preservation__name\": null,                             \"sample_transport__name\": null,                             \"transport_duration_h\": 0,                             \"sample_storage__name\": null,                             \"storage_duration_h\": 0,                             \"person__email\": \"logistic@gmail.com\"                           },                           \"analysis_method\": {                             \"digit-soil-sear\": [                               {                                 \"digitsoil\": [                                   {                                     \"value\": 3.5,                                     \"unit__name\": \"pmol*min^-1\",                                     \"indicator__name\": \"digit-soil-sear_LEU\",                                     \"analysis_method__name\": \"digit-soil-sear\",                                     \"instrument_brand__name\": \"digit-soil\",                                     \"instrument_model__name\": \"bob\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 3.8288,                                     \"unit__name\": \"pmol*min^-1\",                                     \"indicator__name\": \"digit-soil-sear_GLA\",                                     \"procedure\": \"digit-soil-sear\",                                     \"analysis_method__name\": \"digit-soil-sear\",                                     \"instrument_brand__name\": \"digit-soil\",                                     \"instrument_model__name\": \"bob\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 6.5383,                                     \"unit__name\": \"pmol*min^-1\",                                     \"indicator__name\": \"digit-soil-sear_GLU\",                                     \"procedure\": \"digit-soil-sear\",                                     \"analysis_method__name\": \"digit-soil-sear\",                                     \"instrument_brand__name\": \"digit-soil\",                                     \"instrument_model__name\": \"bob\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 7.9978,                                     \"unit__name\": \"pmol*min^-1\",                                     \"indicator__name\": \"digit-soil-sear_PHO\",                                     \"procedure\": \"digit-soil-sear\",                                     \"analysis_method__name\": \"digit-soil-sear\",                                     \"instrument_brand__name\": \"digit-soil\",                                     \"instrument_model__name\": \"bob\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.5,                                     \"unit__name\": \"pmol*min^-1\",                                     \"indicator__name\": \"digit-soil-sear_XYL\",                                     \"procedure\": \"digit-soil-sear\",                                     \"analysis_method__name\": \"digit-soil-sear\",                                     \"instrument_brand__name\": \"digit-soil\",                                     \"instrument_model__name\": \"bob\",                                     \"instrument_id\": \"0\"                                   }                                 ]                               }                             ]                           }                         }                       ]                     }                   ]                 }               ]             }           ]         }       ]     }   ] }    Data access and file naming   Recorded data from the enzymatic activity analysis are available at:      Flat json format https://github.com/AI4SH/in-situ_data/tree/main/flat/digit-soil_sear   Nested json format: https://github.com/AI4SH/in-situ_data/tree/main/nested/digit-soil_sear   The file naming is defined to separate alll eventual renewed sampling or analysis efforts and is built up from the following metadata:     country   pilot site   sampling date   point id   min depth   max depth   subsample   replicate   setting   sample_preservation   procedure   instrument   analysis date   For the examples above, the file name thus have the following component values:      se   loennstorp   20240815   55-c   0   20   a   0   uniform   mix-wet   bob   0   20240815   And the final file name becomes:   se-loennstorp-20240815_55-c_0-20_a_0_uniform_mix-wet_bob_0_20240815.json   ","categories": [],
        "tags": [],
        "url": "/in-situ_methods/digitsoil/",
        "teaser": null
      },{
        "title": "eDNA",
        "excerpt":"environmental DNA (eDNA) analysis is a complex laboratory bound chain of processes called metabarcoding. Also the sampling in the field requires special equipment and the sample has to be preserved with a shielding liquid preserving the DNA during storage and transport. The raw eDNA result is a dataset of nucleotid sequences, that is then compared to database libraries (bioinformatic treatment) for determining both the organisms (or orgamism groups) and the processes that occur in the soil.   With growing DNA sequence libraries and reduced costs for the metabarcoding, progressively more and more information can be obtained from eDNA analysis. eDNA metabarcoding is thus increasingly becoming an important method for soil health characterisation.   Short method description   The field sampling requires care to avoid DNA contamination from e.g. human DNA. Once extracted only a very samll sample (milligram) is required. The sample must be preserved to prevent any deterioration or metabolic changes in the DNA nuceoltides and that requires a DNA sheild to be added to the sample. Once the DNA shield is added, the sample can be stored and transported.   The pipeline of processes that compose eDNA metabarcoding in the laboratry include:     extraction,   amplification,   purification,   sequencing, and   bioinformatic treatment (database library mining).   The eDNA analysed as part of AI4SH focused on three taxonomic domains:     Bacteria,   Archaea, and   Eukarya.   For the Eukarya the analyses were restricted to fungi.   For each group the richness (total number of species - recorded as alpha and chao1 indexes), diversity (recorded as alpha Shannon and alpha Simpson indexes) and evenness (pielo index) are recorded. Group related soil functions included for example chemoheterotrophy, human pathogens, plant pathogens and nitrogen fixation. Table 1 is a complete list of eDNA derived eDNA derived organism groups and functions.   Table 1. eDNA derived organism groups and functions recorded with each observation with the applied metabarcoding pipelines.                  Property       Indicator       AI4SH extended naming       Unit                       Prokaryotes alpha observed richness       Prokaryotes alpha richness       metabarcoding_Prokaryotes-alpha-observed-richness       unitless                 Prokaryotes alpha chao1 estimated richness       Prokaryotes chao1 richness       metabarcoding_Prokaryotes-alpha-chao1-estimated-richness       unitless                 Prokaryotes alpha dominance       Prokaryotes alpha dominance       metabarcoding_Prokaryotes-alpha-dominance       unitless                 Prokaryotes alpha pielou e       Prokaryotes Pielou evenness       metabarcoding_Prokaryotes-alpha-pielou-e       unitless                 Prokaryotes alpha shannon       Prokaryotes alpha Shannon diversity       metabarcoding_Prokaryotes-alpha-shannon       unitless                 Prokaryotes alpha shannon       Prokaryotes alpha Simpson diversity       metabarcoding_Prokaryotes-alpha-simpson       unitless                 Prokaryotes functional prediction chemoheterotrophy       Prokaryotes chemoheterotrophy       metabarcoding_Prokaryotes-functional-prediction-chemoheterotrophy       unitless                 Prokaryotes functional prediction human pathogens all       Prokaryotes human pathogens       metabarcoding_Prokaryotes-functional-prediction-human-pathogens-all       unitless                 Prokaryotes functional prediction nitrogen fixation       Nitrogen fixation       metabarcoding_Prokaryotes-functional-prediction-nitrogen-fixation       unitless                 Fungi alpha observed richness       Fungi alpha richness       metabarcoding_Fungi-alpha-observed-richness       unitless                 Fungi alpha chao1 estimated richness       Fungi chao1 richness       metabarcoding_Fungi-alpha-chao1-estimated-richness       unitless                 Fungi alpha dominance       Fungi alpha dominance       metabarcoding_Fungi-alpha-dominance       uitless                 Fungi alpha pielou e       Fungi Pielou evenness       metabarcoding_Fungi-alpha-pielou-e       unitless                 Fungi alpha shannon       Fungi alpha shannon diversity       metabarcoding_Fungi-alpha-shannon       unitless                 Fungi alpha simpson       Fungi alpha simpson diversity       metabarcoding_Fungi-alpha-simpson       unitless                 Fungi funtional prediction Ectomycorrhizal fungi       Ectomycorrhizal fungi function       metabarcoding_Fungi-funtional-prediction-Ectomycorrhizal-fungi       unitless                 Fungi funtional prediction Arbuscular mycorrhizal fungi       Arbuscular mycorrhizal fungi function       metabarcoding_Fungi-funtional-prediction-Arbuscular-mycorrhizal-fungi       unitless                 Fungi funtional prediction fungal saprotrophs       Saprotrophic fungi function       metabarcoding_Fungi-funtional-prediction-fungal-saprotrophs       unitless                 Fungi funtional prediction fungal plant pathogens       Plant pathogenic fungi function       metabarcoding_Fungi-funtional-prediction-fungal-plant-pathogens       unitless           Data format   The AI4SH observed eDNA derived organism groups and functions data are available in 2 differently formatted json objects, a flat version more suitable for direct accessibility and a nested version intended for users who wants to convert the data to a relational sql database. The flat version does not include any arrays whereas the nested version is structured with nested arrays.   Flat json object   Below is an example of a flat json object for eDNA derived organism groups and functions observations. Object keys with a double underscore denote a link to other entries in the database. For example “person__email” is a key for the person responsible, with more detailed contact information in a separate json document. The hashtag comments are not allowed when operating a json file, they are added here for explaining each json object.   {   \"data_source\": \"ai4sh_se-loennstorp\", # 'project'+'_'+'country'+'-'+'pilot site'   \"sampling_log\": { # Assembly of samples collected in the same day at a particular pilot site       \"name\": \"se-loennstorp-20240815\", # 'country'+'-'+'pilot site'+'-'+'sampling date'     \"date_stamp\": \"20240815\", # Date when the sample was collected in the field     \"person__email\": \"sampling@gmail.com\" # Email of person responsible for the sampling   },   \"sample\": \"se-loennstorp-20240815_55-c_0-20\", # 'country'+'-'+'pilot site'+'-'+'sampling date'+'_'+'sample point id'+'_'+'min depth''+'-'+'max depth'   \"locus\": { # Assembly of the point location, setting and depth     \"position__name\": \"se-loennstorp_55-c\", # 'country'+'-'+'pilot site'+'-'+'sample point id' used for checking internal consistency     \"point\": \"55-c\", # sample point id     \"setting\": \"uniform\", # The spatial setting in the landscape (uniform, alley, plantline, edge)     \"latitude\": 55.6671111111111, # Latitude in WGS84     \"longitude\": 13.1156944444444, # Longitude in WGS84     \"min_depth\": 0, # Sample upper limit in cm in soil profile     \"max_depth\": 20 # Sample lower limit in cm in soil profile   },   \"observation\": { # Assembly of metadata and data related to the obeservation     sample_preparation__name\": \"mixed-untreated-soil-in-lab\", # Explanatory name of soil treatment before observation     \"person__email\": \"analysis@gmail.com\", # Email of person responsible for the obeservation     \"subsample\": \"a\", # For registering if physically different soil subsamples were analysed separately, default for a single analysis = a     \"replicate\": 0, # If the analysis was replicated on the same subsample more than once, the first test is always set to 0     \"n_repeats\": 1, # The number of repetitions done by any instrument on a single subsample and as part of a single replicate, with results reported as mean or mean and standard deviation     \"date_stamp\": \"20241110\", # Date when the analysis was done     \"logistic\": { # Assembly of logistical metadata on the handling of the sample       \"sample_preservation__name\": \"dna shield\", # Explanatory name of soil preservation       \"sample_transport__name\": \"cold\", # Explanatory name of soil transportation if sensitive for analysis results       \"transport_duration_h\": 14, # duration of any sensitive transportation in hours       \"sample_storage__name\": \"cold\",  # Explanatory name of soil storage if sensitive for analysis results       \"storage_duration_h\": 2088, # Storage duration in hours       \"person__email\": \"logistic@gmail.com\" # Email of person responsible for the logistics     },     \"analysis\": { # Assembly of analysis results, procedures and instruments       \"metabarcoding_Prokaryotes-alpha-observed-richness\": { # Soil property reported as AI4SH extended naming (part after underscore = core indicator)         \"value\": 1219.0, # Recorded value (single value or mean of repeated observations)         \"unit__name\": \"unitless\", # Recorded unit         \"indicator__name\": \"metabarcoding_Prokaryotes-alpha-observed-richness\", # AI4SH extended name of the soil indicator (part after underscore = core indicator)         \"procedure\": \"metabarcoding\", # Procedure for the analysis         \"analysis_method__name\": \"edna-prokaryotes-alpha-observed-richness\", # Name of the method         \"instrument_brand__name\": \"edna-prokaryotes-alpha-observed-richness\", # Instrument type or brand         \"instrument_model__name\": \"metabarcoding-chain\",  # Model or version of the instrument         \"instrument_id\": \"0\" # Any individual instrument id, the default for unknown id is       },       \"metabarcoding_Prokaryotes-alpha-chao1-estimated-richness\": {         \"value\": 1247.879,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Prokaryotes-alpha-chao1-estimated-richness\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-prokaryotes-alpha-chao1-estimated-richness\",         \"instrument_brand__name\": \"edna-prokaryotes-alpha-chao1-estimated-richness\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Prokaryotes-alpha-dominance\": {         \"value\": 0.004,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Prokaryotes-alpha-dominance\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-prokaryotes-alpha-dominance\",         \"instrument_brand__name\": \"edna-prokaryotes-alpha-dominance\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Prokaryotes-alpha-pielou-e\": {         \"value\": 0.887,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Prokaryotes-alpha-pielou-e\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-prokaryotes-alpha-pielou-e\",         \"instrument_brand__name\": \"edna-prokaryotes-alpha-pielou-e\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Prokaryotes-alpha-shannon\": {         \"value\": 9.094,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Prokaryotes-alpha-shannon\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-prokaryotes-alpha-shannon\",         \"instrument_brand__name\": \"edna-prokaryotes-alpha-shannon\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Prokaryotes-alpha-simpson\": {         \"value\": 0.996,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Prokaryotes-alpha-simpson\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-prokaryotes-alpha-simpson\",         \"instrument_brand__name\": \"edna-prokaryotes-alpha-simpson\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Prokaryotes-functional-prediction-chemoheterotrophy\": {         \"value\": 0.110063849194,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Prokaryotes-functional-prediction-chemoheterotrophy\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-prokaryotes-functional-prediction-chemoheterotrophy\",         \"instrument_brand__name\": \"edna-prokaryotes-functional-prediction-chemoheterotrophy\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Prokaryotes-functional-prediction-human-pathogens-all\": {         \"value\": 0.000337826424783,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Prokaryotes-functional-prediction-human-pathogens-all\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-prokaryotes-functional-prediction-human-pathogens-all\",         \"instrument_brand__name\": \"edna-prokaryotes-functional-prediction-human-pathogens-all\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Prokaryotes-functional-prediction-nitrogen-fixation\": {         \"value\": 0.0,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Prokaryotes-functional-prediction-nitrogen-fixation\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-prokaryotes-functional-prediction-nitrogen-fixation\",         \"instrument_brand__name\": \"edna-prokaryotes-functional-prediction-nitrogen-fixation\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Fungi-alpha-observed-richness\": {         \"value\": 537.0,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Fungi-alpha-observed-richness\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-fungi-alpha-observed-richness\",         \"instrument_brand__name\": \"edna-fungi-alpha-observed-richness\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Fungi-alpha-chao1-estimated-richness\": {         \"value\": 543.6,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Fungi-alpha-chao1-estimated-richness\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-fungi-alpha-chao1-estimated-richness\",         \"instrument_brand__name\": \"edna-fungi-alpha-chao1-estimated-richness\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Fungi-alpha-dominance\": {         \"value\": 0.044,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Fungi-alpha-dominance\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-fungi-alpha-dominance\",         \"instrument_brand__name\": \"edna-fungi-alpha-dominance\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Fungi-alpha-pielou-e\": {         \"value\": 0.671,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Fungi-alpha-pielou-e\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-fungi-alpha-pielou-e\",         \"instrument_brand__name\": \"edna-fungi-alpha-pielou-e\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Fungi-alpha-shannon\": {         \"value\": 6.082,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Fungi-alpha-shannon\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-fungi-alpha-shannon\",         \"instrument_brand__name\": \"edna-fungi-alpha-shannon\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Fungi-alpha-simpson\": {         \"value\": 0.956,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Fungi-alpha-simpson\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-fungi-alpha-simpson\",         \"instrument_brand__name\": \"edna-fungi-alpha-simpson\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Fungi-funtional-prediction-Ectomycorrhizal-fungi\": {         \"value\": 0.00284074439098,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Fungi-funtional-prediction-Ectomycorrhizal-fungi\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-fungi-funtional-prediction-ectomycorrhizal-fungi\",         \"instrument_brand__name\": \"edna-fungi-funtional-prediction-ectomycorrhizal-fungi\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Fungi-funtional-prediction-Arbuscular-mycorrhizal-fungi\": {         \"value\": 0.000135273542428,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Fungi-funtional-prediction-Arbuscular-mycorrhizal-fungi\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-fungi-funtional-prediction-arbuscular-mycorrhizal-fungi\",         \"instrument_brand__name\": \"edna-fungi-funtional-prediction-arbuscular-mycorrhizal-fungi\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Fungi-funtional-prediction-fungal-saprotrophs\": {         \"value\": 0.381471389645888,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Fungi-funtional-prediction-fungal-saprotrophs\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-fungi-funtional-prediction-fungal-saprotrophs\",         \"instrument_brand__name\": \"edna-fungi-funtional-prediction-fungal-saprotrophs\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       },       \"metabarcoding_Fungi-funtional-prediction-fungal-plant-pathogens\": {         \"value\": 0.395404564515375,         \"unit__name\": \"unitless\",         \"indicator__name\": \"metabarcoding_Fungi-funtional-prediction-fungal-plant-pathogens\",         \"procedure\": \"metabarcoding\",         \"analysis_method__name\": \"edna-fungi-funtional-prediction-fungal-plant-pathogens\",         \"instrument_brand__name\": \"edna-fungi-funtional-prediction-fungal-plant-pathogens\",         \"instrument_model__name\": \"metabarcoding-chain\",         \"instrument_id\": \"0\"       }     }   } }   Nested json object   The nested json object for the eDNA derived organism groups and functions observations contains the same object keys and values as the flat version, but hierarcically organised with nested arrays. For explanations of the object keys please see the flat json object above.   {   \"data_source\": [     {       \"name\": \"ai4sh_se-loennstorp\",       \"site\": [         {           \"name\": \"se-loennstorp\",           \"point\": [             {               \"name\": \"55-c\",               \"latitude\": 55.6671111111111,               \"longitude\": 13.1156944444444,               \"setting\": \"uniform\",               \"sampling_log\": [                 {                   \"name\": \"se-loennstorp-20240815\",                   \"date_stamp\": \"20240815\",                   \"person__email\": \"sampling@gmail.com\",                   \"sample\": [                     {                       \"name\": \"se-loennstorp-20240815_55-c_0-20\",                       \"min_depth\": 0,                       \"max_depth\": 20,                       \"observation\": [                         {                           \"sample_preparation__name\": \"mixed-untreated-soil-in-lab\",                           \"person__email\": \"analysis@gmail.com\",                           \"subsample\": \"a\",                           \"replicate\": 0,                           \"n_repeats\": 1,                           \"date_stamp\": \"20241110\",                           \"logistic\": {                             \"sample_preservation__name\": \"dna shield\",                             \"sample_transport__name\": \"cold\",                             \"transport_duration_h\": 14,                             \"sample_storage__name\": \"cold\",                             \"storage_duration_h\": 2088,                             \"person__email\": \"logistic@gmail.com\"                           },                           \"analysis_method\": {                             \"metabarcoding\": [                               {                                 \"edna\": [                                   {                                     \"value\": 1219.0,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Prokaryotes-alpha-observed-richness\",                                     \"analysis_method__name\": \"edna-prokaryotes-alpha-observed-richness\",                                     \"instrument_brand__name\": \"edna-prokaryotes-alpha-observed-richness\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 1247.879,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Prokaryotes-alpha-chao1-estimated-richness\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-prokaryotes-alpha-chao1-estimated-richness\",                                     \"instrument_brand__name\": \"edna-prokaryotes-alpha-chao1-estimated-richness\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.004,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Prokaryotes-alpha-dominance\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-prokaryotes-alpha-dominance\",                                     \"instrument_brand__name\": \"edna-prokaryotes-alpha-dominance\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.887,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Prokaryotes-alpha-pielou-e\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-prokaryotes-alpha-pielou-e\",                                     \"instrument_brand__name\": \"edna-prokaryotes-alpha-pielou-e\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 9.094,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Prokaryotes-alpha-shannon\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-prokaryotes-alpha-shannon\",                                     \"instrument_brand__name\": \"edna-prokaryotes-alpha-shannon\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.996,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Prokaryotes-alpha-simpson\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-prokaryotes-alpha-simpson\",                                     \"instrument_brand__name\": \"edna-prokaryotes-alpha-simpson\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.110063849194,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Prokaryotes-functional-prediction-chemoheterotrophy\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-prokaryotes-functional-prediction-chemoheterotrophy\",                                     \"instrument_brand__name\": \"edna-prokaryotes-functional-prediction-chemoheterotrophy\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.000337826424783,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Prokaryotes-functional-prediction-human-pathogens-all\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-prokaryotes-functional-prediction-human-pathogens-all\",                                     \"instrument_brand__name\": \"edna-prokaryotes-functional-prediction-human-pathogens-all\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.0,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Prokaryotes-functional-prediction-nitrogen-fixation\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-prokaryotes-functional-prediction-nitrogen-fixation\",                                     \"instrument_brand__name\": \"edna-prokaryotes-functional-prediction-nitrogen-fixation\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 537.0,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Fungi-alpha-observed-richness\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-fungi-alpha-observed-richness\",                                     \"instrument_brand__name\": \"edna-fungi-alpha-observed-richness\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 543.6,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Fungi-alpha-chao1-estimated-richness\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-fungi-alpha-chao1-estimated-richness\",                                     \"instrument_brand__name\": \"edna-fungi-alpha-chao1-estimated-richness\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.044,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Fungi-alpha-dominance\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-fungi-alpha-dominance\",                                     \"instrument_brand__name\": \"edna-fungi-alpha-dominance\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.671,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Fungi-alpha-pielou-e\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-fungi-alpha-pielou-e\",                                     \"instrument_brand__name\": \"edna-fungi-alpha-pielou-e\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 6.082,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Fungi-alpha-shannon\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-fungi-alpha-shannon\",                                     \"instrument_brand__name\": \"edna-fungi-alpha-shannon\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.956,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Fungi-alpha-simpson\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-fungi-alpha-simpson\",                                     \"instrument_brand__name\": \"edna-fungi-alpha-simpson\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.00284074439098,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Fungi-funtional-prediction-Ectomycorrhizal-fungi\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-fungi-funtional-prediction-ectomycorrhizal-fungi\",                                     \"instrument_brand__name\": \"edna-fungi-funtional-prediction-ectomycorrhizal-fungi\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.000135273542428,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Fungi-funtional-prediction-Arbuscular-mycorrhizal-fungi\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-fungi-funtional-prediction-arbuscular-mycorrhizal-fungi\",                                     \"instrument_brand__name\": \"edna-fungi-funtional-prediction-arbuscular-mycorrhizal-fungi\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.381471389645888,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Fungi-funtional-prediction-fungal-saprotrophs\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-fungi-funtional-prediction-fungal-saprotrophs\",                                     \"instrument_brand__name\": \"edna-fungi-funtional-prediction-fungal-saprotrophs\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.395404564515375,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"metabarcoding_Fungi-funtional-prediction-fungal-plant-pathogens\",                                     \"procedure\": \"metabarcoding\",                                     \"analysis_method__name\": \"edna-fungi-funtional-prediction-fungal-plant-pathogens\",                                     \"instrument_brand__name\": \"edna-fungi-funtional-prediction-fungal-plant-pathogens\",                                     \"instrument_model__name\": \"metabarcoding-chain\",                                     \"instrument_id\": \"0\"                                   }                                 ]                               }                             ]                           }                         }                       ]                     }                   ]                 }               ]             }           ]         }       ]     }   ] }   Data access and file naming   Recorded data from the eDNA metabarcoding analysis are available at:      Flat json format https://github.com/AI4SH/in-situ_data/tree/main/flat/edna   Nested json format: https://github.com/AI4SH/in-situ_data/tree/main/nested/edna   The file naming is defined to separate alll eventual renewed sampling or analysis efforts and is built up from the following metadata:     country   pilot site   sampling date   point id   min depth   max depth   subsample   replicate   setting   sample_preservation   procedure   instrument   analysis date   For the examples above, the file name thus have the following component values:      se   loennstorp   20240815   55-c   0   20   a   0   uniform   mix-wet   metabarcoding-chain   0   20241110   And the final file name becomes:   se-loennstorp-20240815_55-c_0-20_a_0_uniform_mix-wet_metabarcoding-chain_0_20241110.json   ","categories": [],
        "tags": [],
        "url": "/in-situ_methods/edna/",
        "teaser": null
      },{
        "title": "Infiltration",
        "excerpt":"Infiltration, the capacity of soil to swallow water, is a prerequisit for soils and their ecosystems to function. Reduced infiltration capacity, due e.g. to compaction or water repellant conditions arising from drought, makes soils more prone to erosion and less resilient to droughts and storms. Low infiltrations capacity also exacerbate flash floods and downstream flooding. As part of AI4SH a fairly simple singe-ring infiltration method is used.   Short method description   Single ring infiltration is a published scientific method for estimating soil hydraulic properties. The ring is easily constructed from a food tin can (or other tin can) and inserted a few cm into the soil. Small, fixed volumes of water (typically 50 to 100 ml) are sequentially poured into the ring and the time it takes for the water to infiltrate is recorded. The pouring of water contiuous until the time it takes for it to infiltration is stable. To calculate the hydraulic properties (table 1), data on bulk denisty and soil mositure prior to the infiltration test are needed - e.g. obtained using the soil cylinder bulk density and volumetric soil moisture content method.   Table 1. Hydarulic soil properties recorded with each single ring infiltration observation.                  Property       Indicator       AI4SH extended naming       Unit                       infiltration rate       infiltration       single-ring-infiltration_final-infiltration-rate       m*day^-1                 porosity       porosity       single-ring-infiltration_porosity       vol*vol^-1                 sorptivity       sorptivity       single-ring-infiltration_sorptivity       m*day^-1/2                 saturated hydraulic conductivity       hydraulic conductivity       single-ring-infiltration_saturated-hydraulic-conductivity       m*day^-1                 field capacity       field capacity       single-ring-infiltration_field-capacity       vol*vol^-1                 plant avaiable water       plant available water       single-ring-infiltration_plant-available-water       vol*vol^-1                 saturated water content       saturated water content       single-ring-infiltration_saturated-water-content       vol*vol^-1                 hg water pressure head scale parameter       hg water pressure head       single-ring-infiltration_hg-water-pressure-head-scale-parameter       mm                 n shape parameter of retention curve       retention curve n       single-ring-infiltration_n-shape-parameter-of-retention-curve       unitless                 m shape parameter of retention curve       retention curve m       single-ring-infiltration_n-shape-parameter-of-retention-curve       unitless           Data format   The AI4SH observed infiltration derived data are available in 2 differently formatted json objects, a flat version more suitable for direct accessibility and a nested version intended for users who wants to convert the data to a relational sql database. The flat version does not include any arrays whereas the nested version is structured with nested arrays.   Flat json object   Below is an example of a flat json object for infiltration derived observations. Object keys with a double underscore denote a link to other entries in the database. For example “person__email” is a key for the person responsible, with more detailed contact information in a separate json document. The hashtag comments are not allowed when operating a json file, they are added here for explaining each json object.   {   \"data_source\": \"ai4sh_se-loennstorp\", # 'project'+'_'+'country'+'-'+'pilot site'   \"sampling_log\": { # Assembly of samples collected in the same day at a particular pilot site       \"name\": \"se-loennstorp-20240815\", # 'country'+'-'+'pilot site'+'-'+'sampling date'     \"date_stamp\": \"20240815\", # Date when the sample was collected in the field     \"person__email\": \"sampling@gmail.com\" # Email of person responsible for the sampling   },   \"sample\": \"se-loennstorp-20240815_55-c_0-20\", # 'country'+'-'+'pilot site'+'-'+'sampling date'+'_'+'sample point id'+'_'+'min depth''+'-'+'max depth'   \"locus\": { # Assembly of the point location, setting and depth     \"position__name\": \"se-loennstorp_55-c\", # 'country'+'-'+'pilot site'+'-'+'sample point id' used for checking internal consistency     \"point\": \"55-c\", # sample point id     \"setting\": \"uniform\", # The spatial setting in the landscape (uniform, alley, plantline, edge)     \"latitude\": 55.6671111111111, # Latitude in WGS84     \"longitude\": 13.1156944444444, # Longitude in WGS84     \"min_depth\": 0, # Sample upper limit in cm in soil profile     \"max_depth\": 20 # Sample lower limit in cm in soil profile   },   \"observation\": { # Assembly of metadata and data related to the obeservation     sample_preparation__name\": \"soil-undisturbed-in-situ\", # Explanatory name of soil treatment before observation     \"person__email\": \"analysis@gmail.com\", # Email of person responsible for the obeservation     \"subsample\": \"a\", # For registering if physically different soil subsamples were analysed separately, default for a single analysis = a     \"replicate\": 0, # If the analysis was replicated on the same subsample more than once, the first test is always set to 0     \"n_repeats\": 1, # The number of repetitions done by any instrument on a single subsample and as part of a single replicate, with results reported as mean or mean and standard deviation     \"date_stamp\": \"20240815\", # Date when the analysis was done     \"logistic\": { # Assembly of logistical metadata on the handling of the sample       \"sample_preservation__name\": null, # Explanatory name of soil preservation       \"sample_transport__name\": null, # Explanatory name of soil transportation if sensitive for analysis results       \"transport_duration_h\": 0, # duration of any sensitive transportation in hours       \"sample_storage__name\": null, # Explanatory name of soil storage if sensitive for analysis results       \"storage_duration_h\": 0, # Storage duration in hours       \"person__email\": \"logistic@gmail.com\" # Email of person responsible for the logistics     },     \"analysis\": { # Assembly of analysis results, procedures and instruments       \"single-ring-infiltration_porosity\": { # Soil property reported as AI4SH extended naming (part after underscore = core indicator)         \"value\": 0.53,  # Recorded value (single value or mean of repeated observations)         \"unit__name\": \"vol*vol^-1\",  # Recorded unit         \"indicator__name\": \"single-ring-infiltration_porosity\",  # AI4SH extended name of the soil indicator (part after underscore = core indicator)         \"procedure\": \"single-ring-infiltration\", # Procedure for the analysis (not always similar to method)         \"analysis_method__name\": \"single-ring-infiltration\", # Name of the method         \"instrument_brand__name\": \"single-ring\", # Instrument type or brand         \"instrument_model__name\": \"tin-can\", # Model or version of the instrument         \"instrument_id\": \"0\" # Any individual instrument id, the default for unknown id is \"0\"       },       \"single-ring-infiltration_final-infiltration-rate\": {         \"value\": 10.8524,         \"unit__name\": \"m*day^-1\",         \"indicator__name\": \"single-ring-infiltration_final-infiltration-rate\",         \"procedure\": \"single-ring-infiltration\",         \"analysis_method__name\": \"single-ring-infiltration\",         \"instrument_brand__name\": \"single-ring\",         \"instrument_model__name\": \"tin-can\",         \"instrument_id\": \"0\"       },       \"single-ring-infiltration_sorptivity\": {         \"value\": 0.291019,         \"unit__name\": \"m*day^-1/2\",         \"indicator__name\": \"single-ring-infiltration_sorptivity\",         \"procedure\": \"single-ring-infiltration\",         \"analysis_method__name\": \"single-ring-infiltration\",         \"instrument_brand__name\": \"single-ring\",         \"instrument_model__name\": \"tin-can\",         \"instrument_id\": \"0\"       },       \"single-ring-infiltration_saturated-hydraulic-conductivity\": {         \"value\": 7.61166,         \"unit__name\": \"m*day^-1\",         \"indicator__name\": \"single-ring-infiltration_saturated-hydraulic-conductivity\",         \"procedure\": \"single-ring-infiltration\",         \"analysis_method__name\": \"single-ring-infiltration\",         \"instrument_brand__name\": \"single-ring\",         \"instrument_model__name\": \"tin-can\",         \"instrument_id\": \"0\"       },       \"single-ring-infiltration_field-capacity\": {         \"value\": 0.206606,         \"unit__name\": \"vol*vol^-1\",         \"indicator__name\": \"single-ring-infiltration_field-capacity\",         \"procedure\": \"single-ring-infiltration\",         \"analysis_method__name\": \"single-ring-infiltration\",         \"instrument_brand__name\": \"single-ring\",         \"instrument_model__name\": \"tin-can\",         \"instrument_id\": \"0\"       },       \"single-ring-infiltration_plant-available-water\": {         \"value\": 0.0981831,         \"unit__name\": \"vol*vol^-1\",         \"indicator__name\": \"single-ring-infiltration_plant-available-water\",         \"procedure\": \"single-ring-infiltration\",         \"analysis_method__name\": \"single-ring-infiltration\",         \"instrument_brand__name\": \"single-ring\",         \"instrument_model__name\": \"tin-can\",         \"instrument_id\": \"0\"       },       \"single-ring-infiltration_saturated-water-content\": {         \"value\": 0.53,         \"unit__name\": \"vol*vol^-1\",         \"indicator__name\": \"single-ring-infiltration_saturated-water-content\",         \"procedure\": \"single-ring-infiltration\",         \"analysis_method__name\": \"single-ring-infiltration\",         \"instrument_brand__name\": \"single-ring\",         \"instrument_model__name\": \"tin-can\",         \"instrument_id\": \"0\"       },       \"single-ring-infiltration_hg-water-pressure-head-scale-parameter\": {         \"value\": 0.0124932,         \"unit__name\": \"mm\",         \"indicator__name\": \"single-ring-infiltration_hg-water-pressure-head-scale-parameter\",         \"procedure\": \"single-ring-infiltration\",         \"analysis_method__name\": \"single-ring-infiltration\",         \"instrument_brand__name\": \"single-ring\",         \"instrument_model__name\": \"tin-can\",         \"instrument_id\": \"0\"       },       \"single-ring-infiltration_n-shape-parameter-of-retention-curve\": {         \"value\": 2.16893,         \"unit__name\": \"unitless\",         \"indicator__name\": \"single-ring-infiltration_n-shape-parameter-of-retention-curve\",         \"procedure\": \"single-ring-infiltration\",         \"analysis_method__name\": \"single-ring-infiltration\",         \"instrument_brand__name\": \"single-ring\",         \"instrument_model__name\": \"tin-can\",         \"instrument_id\": \"0\"       },       \"single-ring-infiltration_m-shape-parameter-of-retention-curve\": {         \"value\": 0.0778884,         \"unit__name\": \"unitless\",         \"indicator__name\": \"single-ring-infiltration_m-shape-parameter-of-retention-curve\",         \"procedure\": \"single-ring-infiltration\",         \"analysis_method__name\": \"single-ring-infiltration\",         \"instrument_brand__name\": \"single-ring\",         \"instrument_model__name\": \"tin-can\",         \"instrument_id\": \"0\"       }     }   } }   Nested json object   The nested json object for soil infiltration derived observations contains the same object keys and values as the flat version, but hierarcically organised with nested arrays. For explanations of the object keys please see the flat json object above.   {   \"data_source\": [     {       \"name\": \"ai4sh_se-loennstorp\",       \"site\": [         {           \"name\": \"se-loennstorp\",           \"point\": [             {               \"name\": \"55-c\",               \"latitude\": 55.6671111111111,               \"longitude\": 13.1156944444444,               \"setting\": \"uniform\",               \"sampling_log\": [                 {                   \"name\": \"se-loennstorp-20240815\",                   \"date_stamp\": \"20240815\",                   \"person__email\": \"sampling@gmail.com\",                   \"sample\": [                     {                       \"name\": \"se-loennstorp-20240815_55-c_0-20\",                       \"min_depth\": 0,                       \"max_depth\": 20,                       \"observation\": [                         {                           \"sample_preparation__name\": \"soil-undisturbed-in-situ\",                           \"person__email\": \"analysis@gmail.com\",                           \"subsample\": \"a\",                           \"replicate\": 0,                           \"n_repeats\": 1,                           \"date_stamp\": \"20240815\",                           \"logistic\": {                             \"sample_preservation__name\": null,                             \"sample_transport__name\": null,                             \"transport_duration_h\": 0,                             \"sample_storage__name\": null,                             \"storage_duration_h\": 0,                             \"person__email\": \"logistic@gmail.com\"                           },                           \"analysis_method\": {                             \"single-ring-infiltration\": [                               {                                 \"single-ring infiltration\": [                                   {                                     \"value\": 0.53,                                     \"unit__name\": \"vol*vol^-1\",                                     \"indicator__name\": \"single-ring-infiltration_porosity\",                                     \"analysis_method__name\": \"single-ring-infiltration\",                                     \"instrument_brand__name\": \"single-ring\",                                     \"instrument_model__name\": \"tin-can\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 10.8524,                                     \"unit__name\": \"m*day^-1\",                                     \"indicator__name\": \"single-ring-infiltration_final-infiltration-rate\",                                     \"procedure\": \"single-ring-infiltration\",                                     \"analysis_method__name\": \"single-ring-infiltration\",                                     \"instrument_brand__name\": \"single-ring\",                                     \"instrument_model__name\": \"tin-can\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.291019,                                     \"unit__name\": \"m*day^-1/2\",                                     \"indicator__name\": \"single-ring-infiltration_sorptivity\",                                     \"procedure\": \"single-ring-infiltration\",                                     \"analysis_method__name\": \"single-ring-infiltration\",                                     \"instrument_brand__name\": \"single-ring\",                                     \"instrument_model__name\": \"tin-can\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 7.61166,                                     \"unit__name\": \"m*day^-1\",                                     \"indicator__name\": \"single-ring-infiltration_saturated-hydraulic-conductivity\",                                     \"procedure\": \"single-ring-infiltration\",                                     \"analysis_method__name\": \"single-ring-infiltration\",                                     \"instrument_brand__name\": \"single-ring\",                                     \"instrument_model__name\": \"tin-can\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.206606,                                     \"unit__name\": \"vol*vol^-1\",                                     \"indicator__name\": \"single-ring-infiltration_field-capacity\",                                     \"procedure\": \"single-ring-infiltration\",                                     \"analysis_method__name\": \"single-ring-infiltration\",                                     \"instrument_brand__name\": \"single-ring\",                                     \"instrument_model__name\": \"tin-can\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.0981831,                                     \"unit__name\": \"vol*vol^-1\",                                     \"indicator__name\": \"single-ring-infiltration_plant-available-water\",                                     \"procedure\": \"single-ring-infiltration\",                                     \"analysis_method__name\": \"single-ring-infiltration\",                                     \"instrument_brand__name\": \"single-ring\",                                     \"instrument_model__name\": \"tin-can\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.53,                                     \"unit__name\": \"vol*vol^-1\",                                     \"indicator__name\": \"single-ring-infiltration_saturated-water-content\",                                     \"procedure\": \"single-ring-infiltration\",                                     \"analysis_method__name\": \"single-ring-infiltration\",                                     \"instrument_brand__name\": \"single-ring\",                                     \"instrument_model__name\": \"tin-can\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.0124932,                                     \"unit__name\": \"mm\",                                     \"indicator__name\": \"single-ring-infiltration_hg-water-pressure-head-scale-parameter\",                                     \"procedure\": \"single-ring-infiltration\",                                     \"analysis_method__name\": \"single-ring-infiltration\",                                     \"instrument_brand__name\": \"single-ring\",                                     \"instrument_model__name\": \"tin-can\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 2.16893,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"single-ring-infiltration_n-shape-parameter-of-retention-curve\",                                     \"procedure\": \"single-ring-infiltration\",                                     \"analysis_method__name\": \"single-ring-infiltration\",                                     \"instrument_brand__name\": \"single-ring\",                                     \"instrument_model__name\": \"tin-can\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 0.0778884,                                     \"unit__name\": \"unitless\",                                     \"indicator__name\": \"single-ring-infiltration_m-shape-parameter-of-retention-curve\",                                     \"procedure\": \"single-ring-infiltration\",                                     \"analysis_method__name\": \"single-ring-infiltration\",                                     \"instrument_brand__name\": \"single-ring\",                                     \"instrument_model__name\": \"tin-can\",                                     \"instrument_id\": \"0\"                                   }                                 ]                               }                             ]                           }                         }                       ]                     }                   ]                 }               ]             }           ]         }       ]     }   ] }   Data access and file naming   Recorded data from the infiltration derived analysis are available at:      Flat json format https://github.com/AI4SH/in-situ_data/tree/main/flat/infiltration   Nested json format: https://github.com/AI4SH/in-situ_data/tree/main/nested/infiltration   The file naming is defined to separate alll eventual renewed sampling or analysis efforts and is built up from the following metadata:     country   pilot site   sampling date   point id   min depth   max depth   subsample   replicate   setting   sample_preservation   procedure   instrument   analysis date   For the examples above, the file name thus have the following component values:      se   loennstorp   20240815   55-c   0   20   a   0   uniform   no-prep   tin-can   0   20240815   And the final file name becomes:   se-loennstorp-20240815_55-c_0-20_a_0_uniform_no-prep_tin-can_0_20240815.json   ","categories": [],
        "tags": [],
        "url": "/in-situ_methods/infiltration/",
        "teaser": null
      },{
        "title": "Ion Selective Electrode",
        "excerpt":"Short method description   Two or more electrodes are inserted into the soil to a specific depth. A voltage is applied across the electrodes. The resistance or conductivity of the soil to the current is measured. This measurement of electrical conductivity (EC) is used to estimate the total dissolved salts, which is a measure of soil salinity.   Scientific reports   Data access   Resources   Howebrew   RVM   Asdf   chruby   Frum   blog on installing Ruby on MacOS   ","categories": [],
        "tags": [],
        "url": "/in-situ_methods/ise/",
        "teaser": null
      },{
        "title": "Microbiometer",
        "excerpt":"Microbiometer is a commercial kit for quick estimation of soil microbial carbon biomass and the ratio between bacteria and fungi.   Short method description   The commercial Microbiometer test mixes a small (1 ml) soil sample with a reagent solution whereafter the mixture is rested for 20 minutes while the reaction is completed and the solution settles. Three drops of the supernatant are extracetd using a provided pipette and applied to a likewise provided test card. The test card is then scanned with a smartphone using the Microbiometer app. If the color change of the drop area on the test card is too weak, additional three drops are added and the test card re-scanned. From the color intensity of the drop area, the app reports the soil’s microbial biomass carbon content and fungal-to-bacterial ratio. The test takes approximately 30 minutes in total. The app and the test card were updated during 2025 and thus the AI4SH reported results relate to both the classic and pro versions of the Microbiometer.   Data format   The AI4SH observed microbiometer data are available in 2 differently formatted json objects, a flat version more suitable for direct accessibility and a nested version intended for users who wants to convert the data to a relational sql database. The flat version does not include any arrays whereas the nested version is structured with nested arrays.   Flat json object   Below is an example of a flat json object for Microbiometer observations. Object keys with a double underscore denote a link to other entries in the database. For example “person__email” is a key for the person responsible, with more detailed contact information in a separate json document. The hashtag comments are not allowed when operating a json file, they are added here for explaining each json object.   {   \"data_source\": \"ai4sh_se-loennstorp\", # 'project'+'_'+'country'+'-'+'pilot site'   \"sampling_log\": { # Assembly of samples collected in the same day at a particular pilot site       \"name\": \"se-loennstorp-20240815\", # 'country'+'-'+'pilot site'+'-'+'sampling date'     \"date_stamp\": \"20240815\", # Date when the sample was collected in the field     \"person__email\": \"sampling@gmail.com\" # Email of person responsible for the sampling   },   \"sample\": \"se-loennstorp-20240815_55-c_0-20\", # 'country'+'-'+'pilot site'+'-'+'sampling date'+'_'+'sample point id'+'_'+'min depth''+'-'+'max depth'   \"locus\": { # Assembly of the point location, setting and depth     \"position__name\": \"se-loennstorp_55-c\", # 'country'+'-'+'pilot site'+'-'+'sample point id' used for checking internal consistency     \"point\": \"55-c\", # sample point id     \"setting\": \"uniform\", # The spatial setting in the landscape (uniform, alley, plantline, edge)     \"latitude\": 55.6671111111111, # Latitude in WGS84     \"longitude\": 13.1156944444444, # Longitude in WGS84     \"min_depth\": 0, # Sample upper limit in cm in soil profile     \"max_depth\": 20 # Sample lower limit in cm in soil profile   },   \"observation\": { # Assembly of metadata and data related to the obeservation     \"sample_preparation__name\": \"mixed-untreated-soil-in-lab\", # Explanatory name of soil treatment before observation     \"person__email\": \"analysis@gmail.com\", # Email of person responsible for the obeservation     \"subsample\": \"a\", # For registering if physically different soil subsamples were analysed separately, default for a single analysis = a     \"replicate\": 0, # If the analysis was replicated on the same subsample more than once, the first test is always set to 0     \"n_repeats\": 1, # The number of repetitions done by any instrument on a single subsample and as part of a single replicate, with results reported as mean or mean and standard deviation     \"date_stamp\": \"20240815\", # Date when the analysis was done     \"logistic\": { # Assembly of logistical metadata on the handling of the sample       \"sample_preservation__name\": null, # Explanatory name of soil preservation       \"sample_transport__name\": null, # Explanatory name of soil transportation if sensitive for analysis results       \"transport_duration_h\": 0, # duration of any sensitive transportation in hours       \"sample_storage__name\": null, # Explanatory name of soil storage if sensitive for analysis results       \"storage_duration_h\": 0, # Storage duration in hours       \"person__email\": \"logistic@gmail.com\" # Email of person responsible for the logistics     },     \"analysis\": { # Assembly of analysis results, procedures and instruments       \"microbiometer_Microbial-C\": { # Soil property reported as AI4SH extended naming (part after underscore = core indicator)         \"value\": 385.0, # Recorded value (single value or mean of repeated observations)         \"unit__name\": \"ug*g^-1\", # Recorded unit         \"indicator__name\": \"microbiometer_Microbial-C\", # AI4SH extended name of the soil indicator (part after underscore = core indicator)         \"procedure\": \"microbiometer\", # Procedure for the analysis (not always similar to method)         \"analysis_method__name\": \"microbiometer\", # Name of the method         \"instrument_brand__name\": \"microbiometer\", # Instrument type or brand         \"instrument_model__name\": \"classic\", # Model or version of the instrument         \"instrument_id\": \"0\" # Any individual instrument id, the default for unknown id is \"0\"       },       \"microbiometer_fungi-fraction\": {         \"value\": 42.0,         \"unit__name\": \"%\",         \"indicator__name\": \"microbiometer_fungi-fraction\",         \"procedure\": \"microbiometer\",         \"analysis_method__name\": \"microbiometer\",         \"instrument_brand__name\": \"microbiometer\",         \"instrument_model__name\": \"classic\",         \"instrument_id\": \"0\"       },       \"microbiometer_bacteria-fraction\": {         \"value\": 58.0,         \"unit__name\": \"%\",         \"indicator__name\": \"microbiometer_bacteria-fraction\",         \"procedure\": \"microbiometer\",         \"analysis_method__name\": \"microbiometer\",         \"instrument_brand__name\": \"microbiometer\",         \"instrument_model__name\": \"classic\",         \"instrument_id\": \"0\"       }     }   } }   Nested json object   The nested json object for Microbiometer observations contains the same object keys and values as the flat version, but hierarcically organised with nested arrays. For explanations of the object keys please see the flat json object above.   {   \"data_source\": [     {       \"name\": \"ai4sh_se-loennstorp\",       \"site\": [         {           \"name\": \"se-loennstorp\",           \"point\": [             {               \"name\": \"55-c\",               \"latitude\": 55.6671111111111,               \"longitude\": 13.1156944444444,               \"setting\": \"uniform\",               \"sampling_log\": [                 {                   \"name\": \"se-loennstorp-20240815\",                   \"date_stamp\": \"20240815\",                   \"person__email\": \"sampling@gmail.com\",                   \"sample\": [                     {                       \"name\": \"se-loennstorp-20240815_55-c_0-20\",                       \"min_depth\": 0,                       \"max_depth\": 20,                       \"observation\": [                         {                           \"sample_preparation__name\": \"mixed-untreated-soil-in-lab\",                           \"person__email\": \"analysis@gmail.com\",                           \"subsample\": \"a\",                           \"replicate\": 0,                           \"n_repeats\": 1,                           \"date_stamp\": \"20240815\",                           \"logistic\": {                             \"sample_preservation__name\": null,                             \"sample_transport__name\": null,                             \"transport_duration_h\": 0,                             \"sample_storage__name\": null,                             \"storage_duration_h\": 0,                             \"person__email\": \"logistic@gmail.com\"                           },                           \"analysis_method\": {                             \"microbiometer\": [                               {                                 \"microbiometer\": [                                   {                                     \"value\": 385.0,                                     \"unit__name\": \"ug*g^-1\",                                     \"indicator__name\": \"tot-c\",                                     \"analysis_method__name\": \"microbiometer-tot-c\",                                     \"instrument-brand__name\": \"microbiometer-tot-c\",                                     \"instrument-model__name\": \"classic\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 42.0,                                     \"unit__name\": \"%\",                                     \"indicator__name\": \"fungi\",                                     \"procedure\": \"microbiometer-fungi\",                                     \"analysis_method__name\": \"microbiometer-fungi\",                                     \"instrument-brand__name\": \"microbiometer-fungi\",                                     \"instrument-model__name\": \"classic\",                                     \"instrument_id\": \"0\"                                   },                                   {                                     \"value\": 58.0,                                     \"unit__name\": \"%\",                                     \"indicator__name\": \"bacteria\",                                     \"procedure\": \"microbiometer-bacteria\",                                     \"analysis_method__name\": \"microbiometer-bacteria\",                                     \"instrument-brand__name\": \"microbiometer-bacteria\",                                     \"instrument-model__name\": \"classic\",                                     \"instrument_id\": \"0\"                                   }                                 ]                               }                             ]                           }                         }                       ]                     }                   ]                 }               ]             }           ]         }       ]     }   ] }   Data access and file naming   Recorded data from the Microbiometer analysis are available at:      Flat json format https://github.com/AI4SH/in-situ_data/tree/main/flat/microbiometer   Nested json format: https://github.com/AI4SH/in-situ_data/tree/main/nested/microbiometer   The file naming is defined to separate alll eventual renewed sampling or analysis efforts and is built up from the following metadata:     country   pilot site   sampling date   point id   min depth   max depth   subsample   replicate   setting   sample_preservation   procedure   instrument   analysis date   For the examples above, the file name thus have the following component values:      se   loennstorp   20240815   55-c   0   20   a   0   uniform   mix-wet   classic   0   20240815   And the final file name becomes:   se-loennstorp-20240815_55-c_0-20_a_0_uniform_mix-wet_classic_0_20240815.json   ","categories": [],
        "tags": [],
        "url": "/in-situ_methods/microbiometer/",
        "teaser": null
      },{
        "title": "Penetrometer",
        "excerpt":"Physically robust and microchip controlled pentrometers for analysing physicochemical soil properties directly in-situ has become available over the past few years. Using multiple steel pins, developments in microelectronics and signal interpretation these simple instrumetns can now separate water and salt content and also detect specific ions, like hydrogen (pH) and those derived from nitrogen, phosphorus and potassium (NPK) – the key soil macronutrients. Soil penetrometers that can operate at the power levels of mobile phones have emerged only recently and are as yet scientifically unproven. The AI4SH project tested the 5-pin penetrometer model NPKPHCTH-S from ComWinTop store on AliExpress operating at 5 volt and mounted with a microcontroller and usb/bluetooth connection, developed by the startup Xsepctre.   Note the following issues related to the penetrometer data when applying it for further analysis:     the penetrometer data is only available for a subset of pilot sites,   in most cases each sample was anaysed using more than 1 subsample (thus there are more than one record per sample), and   for the Finnish site (Jokionen, used as examples below), 3 different copies of the same penetrometer model where used to allow evaluating bias and consistency.   Difference between subsample, replicate and repetition   The difference between a subsample and a replicate is that if the same physical volume of soil is used for the analysis it is a replicate, but if the analysis is used on a separate physical volume it is a subsample. Thus all destructive analysis methods can per definition not be replicates. For the soil penetrometers to apply a replicate would mean letting it stay in the exact same postion for two sequantial observaion. As each penetrometer observation is already based on 6 repetion and the recorded data is stated as mean value and standard deviation (see example below), one more observation in the exact same position renders no new information. Thus, for the penetrometer analysis done as part of AI4SH, the position was typically shifted to different sides in the dug central pit, and each observation recorded as a subsample.   Short method description   The penetrometer is simply pushed into the soil, either from the top or vertically in a pit. The observation is started from the device (computer or smartphone) connected to the microcontroller unit. In the setup used in AI4SH, each observation is repeated 6 times and the user must approve of the results displayed as mean and standard deviation before the observation is recorded. The penetrometer used in the AI4SH project simultaneously records 9 soil parameters with each observation, table 1.   Table 1. Soil properties recorded with each observation with the ComWinTop NPKPHCTH-S 5 steel pin penetrometer.                  Property       Abbreviation       AI4SH Indicator       Unit                       temperature       temp       temp       C                 Soil moisture       sm       sm       vol*vol^-1                 pH       pH       pH       ph                 Nitrogen       N       n       mg*l-1                 Phosphorus       P       p       mg*l-1                 Potassium       K       k       mg*l-1                 Electric conductivity       ec       ec       us*cm-1                 Salinity       Sal       sal       mg*l-1                 Total Dissolved Solid       TDS       tds       mg*l-1           Data format   Each measurement with the NPKPHCTH-S penetrometer is based on 6 repetions and the recorded data contains the mean value and the standard deviation of these 6 repetitions. The observed penetrometer data are available in 2 differently formatted json objects, a flat version more suitable for direct accessibility and a nested version intended for users who wants to convert the data to a relational sql database. The flat version does not include any arrays whereas the nested version is structured with nested arrays.   Flat json object   Below is an example of a flat json object for penetrometer observations. Object keys with a double underscore denote a link to other entries in the database. For example “person__email” is a key for the person responsible, with more detailed contact information in a separate json document. The hashtag comments are not allowed when operating a json file, they are added here for explaining each json object.   {   \"campaign\": \"ai4sh_fi-jokioinen\", # 'project'+'_'+'country'+'-'+'pilot site'   \"sampling_log\": { # Assembly of samples collected in the same day at a particular pilot site     \"name\": \"fi-jokioinen-20241008\", # 'country'+'-'+'pilot site'+'-'+'sampling date'     \"date_stamp\": \"20241008\", # Date when the sample was collected in the field     \"person__email\": \"thomasg@xspectre.com\" # Email of person responsible for the sampling   },   \"sample\": \"fi-jokioinen-20241008_22-r_0-20\", # 'country'+'-'+'pilot site'+'-'+'sampling date'+'_'+'sample   \"locus\": { # Assembly of the point location, setting and depth     \"position__name\": \"fi-jokioinen_22-r\",  # 'country'+'-'+'pilot site'+'-'+'sample point id' used for checking internal consistency     \"point\": \"22-r\", # sample point id     \"setting\": \"uniform\", # The spatial setting in the landscape (uniform, alley, plantline, edge)     \"latitude\": 60.817773051, # Latitude in WGS84     \"longitude\": 23.479190111, # Longitude in WGS84     \"min_depth\": 0, # Sample upper limit in cm in soil profile     \"max_depth\": 20 # Sample lower limit in cm in soil profile   },   \"observation\": { # Assembly of metadata and data related to the obeservation     \"sample_preparation__name\": \"mixed-untreated-soil-in-lab\", # Explanatory name of soil treatment before observation     \"person__email\": \"thomasg@xspectre.com\", # Email of person responsible for the obeservation     \"subsample\": \"b\", # For registering if physically different soil subsamples were analysed separately, default for a single analysis = a     \"replicate\": 0, # If the analysis was replicated on the same subsample more than once, the first test is always set to 0     \"n_repeats\": 6, # The number of repetitions done by any instrument on a single subsample and as part of a single replicate, with results reported as mean or mean and standard deviation     \"date_stamp\": \"20241008\", # Date when the analysis was done     \"logistic\": { # Assembly of logistical metadata on the handling of the sample       \"sample_preservation__name\": null, # Explanatory name of soil preservation       \"sample_transport__name\": null, # Explanatory name of soil transportation if sensitive for analysis results       \"transport_duration_h\": 0, # duration of any sensitive transportation in hours       \"sample_storage__name\": null, # Explanatory name of soil storage if sensitive for analysis results       \"storage_duration_h\": 0, # Storage duration in hours       \"person__email\": \"thomasg@gxspectre.com\" # Email of person responsible for the logistics     },     \"analysis\": { # Assembly of analysis results, procedures and instruments       \"temperature\": { # Soil property reported from the analysis         \"value\": 16.45, # The recorded value (single value or mean or repeated observations)         \"standard_deviation\": 0.05000000000000071, # The recorded satandard deviation, if applicable         \"unit__name\": \"C\", # the recorded unit         \"indicator__name\": \"temperature\", # The name of the soil indicator         \"procedure\": \"xspectre-penetrometer\", # The procedure used for the analysis         \"analysis_method__name\": \"penetrometer\", # The instrument type or brand used for the analysis         \"instrument-brand__name\": \"comwintop\", # # Name of the method used for the analysis         \"instrument-model__name\": \"npkphcth-s\", # The instrument type or brand used for the analysis         \"instrument_id\": \"0003\" # Any individual instrument id, the default for unknown id is \"0\"       },       \"sm\": { # Each soil property reported with the same instrument are reported as individual objects         \"value\": 22.183333333333334,         \"standard_deviation\": 0.24094720491334964,         \"unit__name\": \"vol*vol-1\",         \"indicator__name\": \"sm\",         \"procedure\": \"xspectre-penetrometer\",         \"analysis_method__name\": \"penetrometer\",         \"instrument-brand__name\": \"comwintop\",         \"instrument-model__name\": \"npkphcth-s\",         \"instrument_id\": \"0003\"       },       \"ec\": { # Each soil property reported with the same instrument are reported as individual objects         \"value\": 28.0,         \"standard_deviation\": 0.0,         \"unit__name\": \"us*cm-1\",         \"indicator__name\": \"ec\",         \"procedure\": \"xspectre-penetrometer\",         \"analysis_method__name\": \"penetrometer\",         \"instrument-brand__name\": \"comwintop\",         \"instrument-model__name\": \"npkphcth-s\",         \"instrument_id\": \"0003\"       },       \"ph\": { # Each soil property reported with the same instrument are reported as individual objects         \"value\": 6.366666666666667,         \"standard_deviation\": 0.18856180831641262,         \"unit__name\": \"pH\",         \"indicator__name\": \"ph\",         \"procedure\": \"xspectre-penetrometer\",         \"analysis_method__name\": \"penetrometer\",         \"instrument-brand__name\": \"comwintop\",         \"instrument-model__name\": \"npkphcth-s\",         \"instrument_id\": \"0003\"       },       \"nitrogen\": { # Each soil property reported with the same instrument are reported as individual objects         \"value\": 88.0,         \"standard_deviation\": 0.0,         \"unit__name\": \"mg*l-1\",         \"indicator__name\": \"nitrogen\",         \"procedure\": \"xspectre-penetrometer\",         \"analysis_method__name\": \"penetrometer\",         \"instrument-brand__name\": \"comwintop\",         \"instrument-model__name\": \"npkphcth-s\",         \"instrument_id\": \"0003\"       },       \"phosphorus\": { # Each soil property reported with the same instrument are reported as individual objects         \"value\": 21.0,         \"standard_deviation\": 0.0,         \"unit__name\": \"mg*l-1\",         \"indicator__name\": \"phosphorus\",         \"procedure\": \"xspectre-penetrometer\",         \"analysis_method__name\": \"penetrometer\",         \"instrument-brand__name\": \"comwintop\",         \"instrument-model__name\": \"npkphcth-s\",         \"instrument_id\": \"0003\"       },       \"potassium\": { # Each soil property reported with the same instrument are reported as individual objects         \"value\": 96.0,         \"standard_deviation\": 0.0,         \"unit__name\": \"mg*l-1\",         \"indicator__name\": \"potassium\",         \"procedure\": \"xspectre-penetrometer\",         \"analysis_method__name\": \"penetrometer\",         \"instrument-brand__name\": \"comwintop\",         \"instrument-model__name\": \"npkphcth-s\",         \"instrument_id\": \"0003\"       },       \"salinity\": { # Each soil property reported with the same instrument are reported as individual objects         \"value\": 158.0,         \"standard_deviation\": 0.0,         \"unit__name\": \"mg*l-1\",         \"indicator__name\": \"salinity\",         \"procedure\": \"xspectre-penetrometer\",         \"analysis_method__name\": \"penetrometer\",         \"instrument-brand__name\": \"comwintop\",         \"instrument-model__name\": \"npkphcth-s\",         \"instrument_id\": \"0003\"       },       \"tds\": { # Each soil property reported with the same instrument are reported as individual objects         \"value\": 144.0,         \"standard_deviation\": 0.0,         \"unit__name\": \"mg*l-1\",         \"indicator__name\": \"tds\",         \"procedure\": \"xspectre-penetrometer\",         \"analysis_method__name\": \"penetrometer\",         \"instrument-brand__name\": \"comwintop\",         \"instrument-model__name\": \"npkphcth-s\",         \"instrument_id\": \"0003\"       }     }   } }   Nested json object   The nested json object for penetrometer observations contains the same object keys and values as the flat version, but hierarcically organised with nested arrays. For explanations of the object keys please see the flat json object above.   {   \"data_source\": [     {       \"name\": \"ai4sh_fi-jokioinen\",       \"site\": [         {           \"name\": \"fi-jokioinen\",           \"point\": [             {               \"name\": \"22-r\",               \"latitude\": 60.817773051,               \"longitude\": 23.479190111,               \"setting\": \"uniform\",               \"sampling_log\": [                 {                   \"name\": \"fi-jokioinen-20241008\",                   \"date_stamp\": \"20241008\",                   \"person__email\": \"thomasg@xspectre.com\",                   \"sample\": [                     {                       \"name\": \"fi-jokioinen-20241008_22-r_0-20\",                       \"min_depth\": 0,                       \"max_depth\": 20,                       \"observation\": [                         {                           \"sample_preparation__name\": \"mixed-untreated-soil-in-lab\",                           \"person__email\": \"thomasg@xspectre.com\",                           \"subsample\": \"b\",                           \"replicate\": 0,                           \"n_repeats\": 6,                           \"date_stamp\": \"20241008\",                           \"logistic\": {                             \"sample_preservation__name\": null,                             \"sample_transport__name\": null,                             \"transport_duration_h\": 0,                             \"sample_storage__name\": null,                             \"storage_duration_h\": 0,                             \"person__email\": \"thomasg@xspectre.com\"                           },                           \"analysis_method\": {                             \"xspectre-penetrometer\": [                               {                                 \"value\": 16.45,                                 \"standard_deviation\": 0.05000000000000071,                                 \"unit__name\": \"C\",                                 \"indicator__name\": \"temperature\",                                 \"analysis_method__name\": \"penetrometer\",                                 \"instrument-brand__name\": \"comwintop\",                                 \"instrument-model__name\": \"npkphcth-s\",                                 \"instrument_id\": \"0003\"                               },                               {                                 \"value\": 22.183333333333334,                                 \"standard_deviation\": 0.24094720491334964,                                 \"unit__name\": \"vol*vol-1\",                                 \"indicator__name\": \"sm\",                                 \"analysis_method__name\": \"penetrometer\",                                 \"instrument-brand__name\": \"comwintop\",                                 \"instrument-model__name\": \"npkphcth-s\",                                 \"instrument_id\": \"0003\"                               },                               {                                 \"value\": 28.0,                                 \"standard_deviation\": 0.0,                                 \"unit__name\": \"us*cm-1\",                                 \"indicator__name\": \"ec\",                                 \"analysis_method__name\": \"penetrometer\",                                 \"instrument-brand__name\": \"comwintop\",                                 \"instrument-model__name\": \"npkphcth-s\",                                 \"instrument_id\": \"0003\"                               },                               {                                 \"value\": 6.366666666666667,                                 \"standard_deviation\": 0.18856180831641262,                                 \"unit__name\": \"pH\",                                 \"indicator__name\": \"ph\",                                 \"analysis_method__name\": \"penetrometer\",                                 \"instrument-brand__name\": \"comwintop\",                                 \"instrument-model__name\": \"npkphcth-s\",                                 \"instrument_id\": \"0003\"                               },                               {                                 \"value\": 88.0,                                 \"standard_deviation\": 0.0,                                 \"unit__name\": \"mg*l-1\",                                 \"indicator__name\": \"nitrogen\",                                 \"analysis_method__name\": \"penetrometer\",                                 \"instrument-brand__name\": \"comwintop\",                                 \"instrument-model__name\": \"npkphcth-s\",                                 \"instrument_id\": \"0003\"                               },                               {                                 \"value\": 21.0,                                 \"standard_deviation\": 0.0,                                 \"unit__name\": \"mg*l-1\",                                 \"indicator__name\": \"phosphorus\",                                 \"analysis_method__name\": \"penetrometer\",                                 \"instrument-brand__name\": \"comwintop\",                                 \"instrument-model__name\": \"npkphcth-s\",                                 \"instrument_id\": \"0003\"                               },                               {                                 \"value\": 96.0,                                 \"standard_deviation\": 0.0,                                 \"unit__name\": \"mg*l-1\",                                 \"indicator__name\": \"potassium\",                                 \"analysis_method__name\": \"penetrometer\",                                 \"instrument-brand__name\": \"comwintop\",                                 \"instrument-model__name\": \"npkphcth-s\",                                 \"instrument_id\": \"0003\"                               },                               {                                 \"value\": 158.0,                                 \"standard_deviation\": 0.0,                                 \"unit__name\": \"mg*l-1\",                                 \"indicator__name\": \"salinity\",                                 \"analysis_method__name\": \"penetrometer\",                                 \"instrument-brand__name\": \"comwintop\",                                 \"instrument-model__name\": \"npkphcth-s\",                                 \"instrument_id\": \"0003\"                               },                               {                                 \"value\": 144.0,                                 \"standard_deviation\": 0.0,                                 \"unit__name\": \"mg*l-1\",                                 \"indicator__name\": \"tds\",                                 \"analysis_method__name\": \"penetrometer\",                                 \"instrument-brand__name\": \"comwintop\",                                 \"instrument-model__name\": \"npkphcth-s\",                                 \"instrument_id\": \"0003\"                               }                             ]                           }                         }                       ]                     }                   ]                 }               ]             }           ]         }       ]     }   ] }   Data access and file naming   recorded data from the Microbiometer analysis are available at      Flat json format https://github.com/AI4SH/in-situ_data/tree/main/flat/penetrometer   Nested json format: https://github.com/AI4SH/in-situ_data/tree/main/flat/penetrometer   The file naming is defined to separate all eventual renewed sampling or analysis efforts and is built up from the following metadata:     country   pilot site   sampling date   point id   min depth   max depth   subsample   replicate   setting   sample_preservation   procedure   instrument   analysis date   For the examples above, the file name thus have the following component values:      fi   jokioinen   20241008   22-r   0   20   b   0   uniform   mix-wet   npkphcth-s   0003   20241008   And the final file name becomes:   fi-jokioinen-20241008_22-r_0-20_b_0_uniform_mix-wet_npkphcth-s_0003_20241008   As noted above, the the peneteromer observations in Jokioinen, Finlad, three copies of the penterometer model (with id’s 0001, 0002 and 0003) where used and each penetroemter where tested on several subsampels (denoted a, b, c, etc).   ","categories": [],
        "tags": [],
        "url": "/in-situ_methods/penetrometer/",
        "teaser": null
      },{
        "title": "Salinity",
        "excerpt":"In the Netherhlands we used a soil penetrometer for determining what?   which can be a simple two-probe device or a more complex four-electrode probe, passes an electrical current through the soil, and the resistance is measured   Short method description   Two or more electrodes are inserted into the soil to a specific depth. A voltage is applied across the electrodes. The resistance or conductivity of the soil to the current is measured. This measurement of electrical conductivity (EC) is used to estimate the total dissolved salts, which is a measure of soil salinity.   Scientific reports   Data access   Resources   Howebrew   RVM   Asdf   chruby   Frum   blog on installing Ruby on MacOS   ","categories": [],
        "tags": [],
        "url": "/in-situ_methods/salinity/",
        "teaser": null
      },{
        "title": "Spectroscopy",
        "excerpt":"Soil spectroscopy is progressively becoming a mature method for determining soil properties. Spectroscopy can not determine all properties that are traditioally analysed using wet laboratory methods, but many key attributes are today routinely determined by spectroscopy also in commercial soil laboratories.   Short method description   FOSS instrument   The ladoratury FOSS DS spectrometer can report the spectra as aborbance or diffuse reflectamce. In the AI4SH all spectral data is stored as diffuse reflectance values. The conversion from absorbance to reflectance is integrated into the [Data management package][management_package].   Scientific reports   Data access   Resources   Howebrew   RVM   Asdf   chruby   Frum   blog on installing Ruby on MacOS   ","categories": [],
        "tags": [],
        "url": "/in-situ_methods/spectroscopy/",
        "teaser": null
      },{
        "title": "Wet laboratory",
        "excerpt":"Traditionally soil properties have been determined by wet laboratory methods.   Method description and reference   15 soil parameters were analysed using traditional wet laboratory methods (table 1). The majority of the samples were analysed at German [Agrolab][agrolab], using the methods listed in table 1.                  Indicator       abbreviation       unit       Method       ISO       url                       pH       pH       ph-h2o                                         Electric conductivity       EC       uS*cm^-1                                         Total Organic Carbon       TOC       g*100g^-1                                         Total Nitrogen       tot-N       g*100g^-1                                         Calcium       (Ca2+)       cmol*kg^-1                                         Magnesium       (Mg2+)       cmol*kg^-1                                         Potassium       (K+)       cmol*kg^-1                                         Sodium       (Na+)       cmol*kg^-1                                         Cation exchange capacity       CEC       cmol*kg^-1                                         Phosphorus       P       mg*100g^-1       Olsen-P                                 Clay (&lt;0,002 mm)       Clay       g*100g^-1                                         Fine silt (0.002-0.02 mm)       Fine silt       g*100g^-1                                         Coarse silt (0.02-0.06 mm)       Coarse silt       g*100g^-1                                         Fine sand (0.06-0.2 mm)       Fine sand       g*100g^-1                                         Coarse sand (0.2-2.0 mm)       Coarse sand       g*100g^-1                                   Scientific reports   Data access   Resources   Howebrew   RVM   Asdf   chruby   Frum   blog on installing Ruby on MacOS   ","categories": [],
        "tags": [],
        "url": "/in-situ_methods/wetlab/",
        "teaser": null
      },,]
